{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Inspiration Text Mining v2.0.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "authorship_tag": "ABX9TyPPe3pkbSn3XzddJK9WhJA2",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/cazdemun/colab-notebooks/blob/main/Inspiration_Text_Mining.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "uxYrqfthG8GB"
      },
      "source": [
        "# Web scrapping"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YEfdJMQHyMgY"
      },
      "source": [
        "## Selenium Installation"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7HqIZtMermVY"
      },
      "source": [
        "import subprocess\n",
        "\n",
        "def setup_selenium():\n",
        "  subprocess.call(['wget', 'https://bit.ly/3fk4vlc', '-O', 'bash.sh'])\n",
        "  subprocess.call(['sh', 'bash.sh'])\n",
        "\n",
        "setup_selenium()"
      ],
      "execution_count": 2,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "kDWVd-BoyRGS"
      },
      "source": [
        "## Selenium configuration"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SVJr2-BrHAJu"
      },
      "source": [
        "import sys\n",
        "sys.path.insert(0,'/usr/lib/chromium-browser/chromedriver')\n",
        "from selenium import webdriver\n",
        "from selenium.webdriver.common.by import By\n",
        "from selenium.webdriver.common.keys import Keys\n",
        "from selenium.webdriver.support.ui import WebDriverWait\n",
        "from selenium.webdriver.support import expected_conditions as EC\n",
        "import pprint\n",
        "\n",
        "# Standar webdriver options\n",
        "# https://stackoverflow.com/questions/51046454/how-can-we-use-selenium-webdriver-in-colab-research-google-com\n",
        "chrome_options = webdriver.ChromeOptions()\n",
        "chrome_options.add_argument('--headless')\n",
        "chrome_options.add_argument('--no-sandbox')\n",
        "chrome_options.add_argument('--disable-dev-shm-usage')\n",
        "\n",
        "# Cloudflare bypass (incomplete or doesn't work)\n",
        "# https://blog.m157q.tw/posts/2020/09/11/bypass-cloudflare-detection-while-using-selenium-with-chromedriver/\n",
        "chrome_options.add_experimental_option(\"excludeSwitches\", [\"enable-automation\"])\n",
        "chrome_options.add_experimental_option('useAutomationExtension', False)\n",
        "chrome_options.add_argument(\"--disable-blink-features=AutomationControlled\")\n",
        "\n",
        "# Headless anti-bot detection\n",
        "# https://intoli.com/blog/making-chrome-headless-undetectable/\n",
        "user_agent = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.50 Safari/537.36'\n",
        "chrome_options.add_argument(f'user-agent={user_agent}')\n",
        "\n",
        "pp = pprint.PrettyPrinter(indent=4)\n",
        "\n",
        "def print_body(wd):\n",
        "  \"\"\"\n",
        "  f: webdriver -> ()\n",
        "  \"\"\"\n",
        "  body_text = wd.find_element(By.TAG_NAME ,\"body\").text\n",
        "  pp.pprint(body_text)\n",
        "\n",
        "def scopus_signin(wd):\n",
        "  \"\"\"\n",
        "  f: webdriver -> webdriver\n",
        "  Returns a scopus homepage signed webdriver\n",
        "  Note: it seems that the web driver keep the credentials after login, this can be useful in a UI\n",
        "  \"\"\"\n",
        "  # home page\n",
        "  wd.get(\"https://www.scopus.com\")\n",
        "  signin_button = wd.find_element(By.ID ,\"signin_link_move\")\n",
        "  signin_button.click()\n",
        "\n",
        "  # sign in\n",
        "  # screen 1 - email\n",
        "  email_input = wd.find_element(By.ID ,\"bdd-email\")\n",
        "  email_input.send_keys(\"u201512699@upc.edu.pe\")\n",
        "\n",
        "  continue_button = wd.find_element(By.ID ,\"bdd-elsPrimaryBtn\")\n",
        "  continue_button.click()\n",
        "\n",
        "  # screen 2 - password\n",
        "  password_input = wd.find_element(By.ID ,\"bdd-password\")\n",
        "  password_input.send_keys(\"4De#4Dmf\")\n",
        "\n",
        "  submit_button = wd.find_element(By.ID ,\"bdd-elsPrimaryBtn\")\n",
        "  submit_button.click()\n",
        "\n",
        "  return wd"
      ],
      "execution_count": 3,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SOc1g_sUWZF4"
      },
      "source": [
        "wd = scopus_signin(webdriver.Chrome('chromedriver',options=chrome_options))\n",
        "wd.current_url"
      ],
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "35CdCdlNTOJZ"
      },
      "source": [
        "# OOP WebDriver"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yDzclcaa9fiS",
        "outputId": "3656cf68-792d-41b5-d56c-bc1b0582cb67"
      },
      "source": [
        "from typing import List\n",
        "import requests\n",
        "\n",
        "class SearchResult:\n",
        "  def __init__(self, title, abstract):\n",
        "    self.title: str = title\n",
        "    self.abstract: str = abstract\n",
        "  def __str__(self):\n",
        "    return f\"\"\"\n",
        "    Title: {self.title}\n",
        "    Abstract: {self.abstract}\n",
        "    \"\"\"\n",
        "\n",
        "class ScopusWebDriver:\n",
        "  def __init__(self, wd):\n",
        "    # State Machine: search.page | search.results.page\n",
        "    self.value: str = \"search.page\" if wd.current_url.startswith('https://www.scopus.com/search/form.uri') else \"search.results.page\"\n",
        "    self.wd = wd\n",
        "    # Search\n",
        "    self.current_query: str = \"\" if wd.current_url.startswith('https://www.scopus.com/search/form.uri') else wd.find_element(By.CLASS_NAME ,\"queryContainer\").text\n",
        "    self.current_display: int= 20 # 20 | 50 | 100 | 200\n",
        "    self.current_page: int= 1\n",
        "    self.total_results: int= 0\n",
        "    self.search_results: List[SearchResult] = []\n",
        "    # self.show_all_abstracts:Bool = False, deprecated?\n",
        "  \n",
        "  def __str__(self):\n",
        "      return f\"\"\"\n",
        "ScopusWebDriver State:\n",
        "  Machine State: \"{self.value}\"\n",
        "  WebDriver object: {self.wd}\n",
        "  Query: {self.current_query}\n",
        "  Display: {self.current_display}\n",
        "  Page: {self.current_page}\n",
        "  Total Results: {self.total_results}\n",
        "  Results: {self.search_results}\n",
        "      \"\"\"\n",
        "  def _search_query(self, query: str) -> str:\n",
        "    if not self.value == \"search.page\":\n",
        "      print(\"Navigation failed, WebDriver should be on the \\\"search.page\\\" state\")\n",
        "      return \"\"\n",
        "\n",
        "    search_input = self.wd.find_element(By.ID ,\"searchterm1\")\n",
        "    \n",
        "    search_input.send_keys(query)\n",
        "\n",
        "    search_button_row = self.wd.find_element(By.ID ,\"searchBtnRow\")\n",
        "    search_button = search_button_row.find_element(By.XPATH, '//button[@title=\"Search\"]')\n",
        "    search_button.click()\n",
        "\n",
        "    self.value = \"search.results.page\"\n",
        "    return self.wd.find_element(By.CLASS_NAME ,\"queryContainer\").text\n",
        "\n",
        "  def return_to_search_page(self) -> bool:\n",
        "    if not self.value == \"search.results.page\":\n",
        "      print(\"Navigation failed, WebDriver should be on the \\\"search.results.page\\\" state\")\n",
        "      return False\n",
        "\n",
        "    search_tab = self.wd.find_element(By.ID ,\"gh-Search\")\n",
        "    search_tab.click()\n",
        "\n",
        "    search_input = self.wd.find_element(By.ID ,\"searchterm1\")\n",
        "    search_input.send_keys(Keys.CONTROL, 'a')\n",
        "    search_input.send_keys(Keys.BACKSPACE)\n",
        "\n",
        "    self.value = \"search.page\" \n",
        "    self.current_query = \"\" \n",
        "    return True\n",
        "\n",
        "  def search(self, query: str, max: int) -> List[SearchResult]:\n",
        "    if not self.value == \"search.page\":\n",
        "      print(\"Search failed, WebDriver should be on the \\\"search.page\\\" state\")\n",
        "      return []\n",
        "\n",
        "    self.current_query = self._search_query(query)\n",
        "    self.total_results = (int)(wd.find_element(By.CLASS_NAME ,'resultsCount').text.replace(',', ''))\n",
        "    \n",
        "    # build abstract request\n",
        "    # pp.pprint(self.wd.get_cookies())\n",
        "    cookies_string = \"; \".join([f\"{c['name']}={c['value']}\" for c in self.wd.get_cookies()])\n",
        "    print(cookies_string)\n",
        "    \n",
        "    headers={\n",
        "        \"Cookie\": cookies_string,\n",
        "        \"Referer\": wd.current_url\n",
        "    }\n",
        "    \n",
        "    # print(\"https://www.scopus.com/results/showallabstractspreview.uri\")\n",
        "\n",
        "    data = {\n",
        "      \"count\": \"200\",\n",
        "      \"offset\": \"0\",\n",
        "      \"searchString\": \"TITLE-ABS-KEY(proteins)\",\n",
        "      \"sortType\": \"r-f\",\n",
        "      \"origin\": \"resultslist\",\n",
        "      \"originationType\": \"b\",\n",
        "      \"clusterRowDisplayCount\": \"10\",\n",
        "      \"resultsPerPage\": \"200\",\n",
        "      \"resultsCount\": \"6,193,247\",\n",
        "      \"eid\": \"null\",\n",
        "      \"listId\": \"\",\n",
        "      \"cite\": \"\",\n",
        "      \"citedAuthorId\": \"\",\n",
        "      \"citingId\": \"\",\n",
        "      \"refeid\": \"\",\n",
        "      \"ref\": \"\",\n",
        "      \"recent\": \"\",\n",
        "      \"recordIndex\": \"null\"\n",
        "    }\n",
        "\n",
        "    r = requests.post(\"https://www.scopus.com/results/showallabstractspreview.uri\", headers=headers, json=data)\n",
        "    print(r.headers)\n",
        "    print(r.text)\n",
        "    return []\n",
        "\n",
        "scopusWD = ScopusWebDriver(wd)\n",
        "print(scopusWD.wd.current_url, scopusWD)"
      ],
      "execution_count": 30,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "https://www.scopus.com/results/results.uri?numberOfFields=0&src=s&clickedLink=&edit=&editSaveSearch=&origin=searchbasic&authorTab=&affiliationTab=&advancedTab=&scint=1&menu=search&tablin=&searchterm1=protein&field1=TITLE_ABS_KEY&dateType=Publication_Date_Type&yearFrom=Before+1960&yearTo=Present&loadDate=7&documenttype=All&accessTypes=All&resetFormLink=&st1=protein&st2=&sot=b&sdt=b&sl=22&s=TITLE-ABS-KEY%28protein%29&sid=0db68610bac03448545772800be5a6c0&searchId=0db68610bac03448545772800be5a6c0&txGid=5f28e97e8a3c7c1be422774218458dd5&sort=r-f&originationType=b&rr= \n",
            "ScopusWebDriver State:\n",
            "  Machine State: \"search.results.page\"\n",
            "  WebDriver object: <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b99ddf7c60830b1e00a67f839121472b\")>\n",
            "  Query: TITLE-ABS-KEY ( protein ) \n",
            "  Display: 20\n",
            "  Page: 1\n",
            "  Total Results: 0\n",
            "  Results: []\n",
            "      \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "eK2qC5AnfSFc",
        "outputId": "f0cc5354-1776-4091-fe1d-ac16c3105341"
      },
      "source": [
        "scopusWD.search(\"protein\", 1000)\n",
        "print(scopusWD.wd.current_url, scopusWD)\n",
        "\n",
        "scopusWD.return_to_search_page()\n",
        "print(scopusWD.wd.current_url, scopusWD)"
      ],
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "s_sess=%20s_cpc%3D0%3B%20s_ppvl%3Dsc%25253Asearch%25253Adocument%252520searchform%252C40%252C40%252C600%252C800%252C600%252C800%252C600%252C1%252CP%3B%20s_ppv%3Dsc%25253Asearch%25253Adocument%252520results%252C2%252C2%252C600%252C800%252C600%252C800%252C600%252C1%252CP%3B%20c21%3Dtitle-abs-key%2528protein%2529%3B%20e13%3Dtitle-abs-key%2528protein%2529%253A1%3B%20c13%3Drelevance%3B%20e41%3D1%3B%20s_sq%3D%3B%20s_cc%3Dtrue%3B; AWSELB=CB9317D502BF07938DE10C841E762B7A33C19AADB1328F3C3DB5264F4CFEF1A02C9C063CCB3841E23B31F6C6F833247060CD6C008CA31AAC5A6BDE3E4B4DACF34F3854CEEB689045CC638AEB49B8B87EE8F8FC1CB9; AMCV_4D6368F454EC41940A4C98A6%40AdobeOrg=359503849%7CMCIDTS%7C18599%7CMCMID%7C88508890635440160581675557641314175166%7CMCAID%7CNONE%7CMCOPTOUT-1606917923s%7CNONE%7CMCAAMLH-1607515523%7C7%7CMCAAMB-1607515523%7Cj8Odv6LonN4r3an7LhD3WZrU1bUpAkFkkiY1ncBR96t2PTI%7CMCSYNCSOP%7C411-18606%7CMCCIDH%7C1863782%7CvVersion%7C5.0.1; s_pers=%20v8%3D1606913547116%7C1701521547116%3B%20v8_s%3DFirst%2520Visit%7C1606915347116%3B%20c19%3Dsc%253Asearch%253Adocument%2520results%7C1606915347123%3B%20v68%3D1606913535908%7C1606915347156%3B; AMCVS_4D6368F454EC41940A4C98A6%40AdobeOrg=1; __cfruid=f1a15f0024648689c6f2c7441ea3e0590c16afe7-1606910836; scopus.machineID=53D4562F04DF12730E3A85152555DE28.i-05ca17589e4896954; mbox=PC#024628240a7a4e7794446670742f037a.34_0#1670158337|session#242154a12ca640eab46c8cb0aeb58051#1606914840; scopusSessionUUID=900b9c66-55bc-49bf-9; SCSessionID=53D4562F04DF12730E3A85152555DE28.i-05ca17589e4896954; at_check=true; __cfduid=dbc7939ab6e8adab98b9f3080c0b7db201606910716\n",
            "{'Date': 'Wed, 02 Dec 2020 12:52:27 GMT', 'Content-Type': 'text/html; charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Frame-Options': 'SAMEORIGIN', 'Cache-Control': 'private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0', 'Expires': 'Thu, 01 Jan 1970 00:00:01 GMT', 'cf-request-id': '06c51bf519000073f9b7a25000000001', 'Expect-CT': 'max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"', 'Vary': 'Accept-Encoding', 'Server': 'cloudflare', 'CF-RAY': '5fb52f682fc973f9-IAD', 'Content-Encoding': 'gzip'}\n",
            "<!DOCTYPE html>\n",
            "<!--[if lt IE 7]> <html class=\"no-js ie6 oldie\" lang=\"en-US\"> <![endif]-->\n",
            "<!--[if IE 7]>    <html class=\"no-js ie7 oldie\" lang=\"en-US\"> <![endif]-->\n",
            "<!--[if IE 8]>    <html class=\"no-js ie8 oldie\" lang=\"en-US\"> <![endif]-->\n",
            "<!--[if gt IE 8]><!--> <html class=\"no-js\" lang=\"en-US\"> <!--<![endif]-->\n",
            "<head>\n",
            "<title>Access denied | www.scopus.com used Cloudflare to restrict access</title>\n",
            "<meta charset=\"UTF-8\" />\n",
            "<meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\" />\n",
            "<meta http-equiv=\"X-UA-Compatible\" content=\"IE=Edge,chrome=1\" />\n",
            "<meta name=\"robots\" content=\"noindex, nofollow\" />\n",
            "<meta name=\"viewport\" content=\"width=device-width,initial-scale=1\" />\n",
            "<link rel=\"stylesheet\" id=\"cf_styles-css\" href=\"/cdn-cgi/styles/main.css\" type=\"text/css\" media=\"screen,projection\" />\n",
            "\n",
            "\n",
            "</head>\n",
            "<body>\n",
            "  <div id=\"cf-wrapper\">\n",
            "    <div class=\"cf-alert cf-alert-error cf-cookie-error hidden\" id=\"cookie-alert\" data-translate=\"enable_cookies\">Please enable cookies.</div>\n",
            "    <div id=\"cf-error-details\" class=\"p-0\">\n",
            "      <header class=\"mx-auto pt-10 lg:pt-6 lg:px-8 w-240 lg:w-full mb-15 antialiased\">\n",
            "         <h1 class=\"inline-block md:block mr-2 md:mb-2 font-light text-60 md:text-3xl text-black-dark leading-tight\">\n",
            "           <span data-translate=\"error\">Error</span>\n",
            "           <span>1020</span>\n",
            "         </h1>\n",
            "         <span class=\"inline-block md:block heading-ray-id font-mono text-15 lg:text-sm lg:leading-relaxed\">Ray ID: 5fb52f682fc973f9 &bull;</span>\n",
            "         <span class=\"inline-block md:block heading-ray-id font-mono text-15 lg:text-sm lg:leading-relaxed\">2020-12-02 12:52:27 UTC</span>\n",
            "        <h2 class=\"text-gray-600 leading-1.3 text-3xl lg:text-2xl font-light\">Access denied</h2>\n",
            "      </header>\n",
            "\n",
            "      <section class=\"w-240 lg:w-full mx-auto mb-8 lg:px-8\">\n",
            "          <div id=\"what-happened-section\" class=\"w-1/2 md:w-full\">\n",
            "            <h2 class=\"text-3xl leading-tight font-normal mb-4 text-black-dark antialiased\" data-translate=\"what_happened\">What happened?</h2>\n",
            "            <p>This website is using a security service to protect itself from online attacks.</p>\n",
            "            \n",
            "          </div>\n",
            "\n",
            "          \n",
            "      </section>\n",
            "\n",
            "      <div class=\"cf-error-footer cf-wrapper w-240 lg:w-full py-10 sm:py-4 sm:px-8 mx-auto text-center sm:text-left border-solid border-0 border-t border-gray-300\">\n",
            "  <p class=\"text-13\">\n",
            "    <span class=\"cf-footer-item sm:block sm:mb-1\">Cloudflare Ray ID: <strong class=\"font-semibold\">5fb52f682fc973f9</strong></span>\n",
            "    <span class=\"cf-footer-separator sm:hidden\">&bull;</span>\n",
            "    <span class=\"cf-footer-item sm:block sm:mb-1\"><span>Your IP</span>: 35.196.215.149</span>\n",
            "    <span class=\"cf-footer-separator sm:hidden\">&bull;</span>\n",
            "    <span class=\"cf-footer-item sm:block sm:mb-1\"><span>Performance &amp; security by</span> <a rel=\"noopener noreferrer\" href=\"https://www.cloudflare.com/5xx-error-landing\" id=\"brand_link\" target=\"_blank\">Cloudflare</a></span>\n",
            "    \n",
            "  </p>\n",
            "</div><!-- /.error-footer -->\n",
            "\n",
            "\n",
            "    </div><!-- /#cf-error-details -->\n",
            "  </div><!-- /#cf-wrapper -->\n",
            "\n",
            "  <script type=\"text/javascript\">\n",
            "  window._cf_translation = {};\n",
            "  \n",
            "  \n",
            "</script>\n",
            "\n",
            "</body>\n",
            "</html>\n",
            "\n",
            "https://www.scopus.com/results/results.uri?numberOfFields=0&src=s&clickedLink=&edit=&editSaveSearch=&origin=searchbasic&authorTab=&affiliationTab=&advancedTab=&scint=1&menu=search&tablin=&searchterm1=protein&field1=TITLE_ABS_KEY&dateType=Publication_Date_Type&yearFrom=Before+1960&yearTo=Present&loadDate=7&documenttype=All&accessTypes=All&resetFormLink=&st1=protein&st2=&sot=b&sdt=b&sl=22&s=TITLE-ABS-KEY%28protein%29&sid=df83ec2ddfe0d6e86e71e1085f677ec8&searchId=df83ec2ddfe0d6e86e71e1085f677ec8&txGid=5be965e855a4770b0c20b1ff467cace9&sort=r-f&originationType=b&rr= \n",
            "ScopusWebDriver State:\n",
            "  Machine State: \"search.results.page\"\n",
            "  WebDriver object: <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b99ddf7c60830b1e00a67f839121472b\")>\n",
            "  Query: TITLE-ABS-KEY ( protein ) \n",
            "  Display: 20\n",
            "  Page: 1\n",
            "  Total Results: 6193247\n",
            "  Results: []\n",
            "      \n",
            "https://www.scopus.com/search/form.uri?zone=TopNavBar&origin=resultslist&display=basic \n",
            "ScopusWebDriver State:\n",
            "  Machine State: \"search.page\"\n",
            "  WebDriver object: <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b99ddf7c60830b1e00a67f839121472b\")>\n",
            "  Query: \n",
            "  Display: 20\n",
            "  Page: 1\n",
            "  Total Results: 6193247\n",
            "  Results: []\n",
            "      \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Be2zob0_mZFI",
        "outputId": "2d2b96a4-a780-42ab-edc6-d9095147eb81"
      },
      "source": [
        "scopusWD.return_to_search_page()\n",
        "print(scopusWD.wd.current_url, scopusWD)"
      ],
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "https://www.scopus.com/search/form.uri?zone=TopNavBar&origin=resultslist&display=basic \n",
            "ScopusWebDriver State:\n",
            "  Machine State: \"search.page\"\n",
            "  WebDriver object: <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b99ddf7c60830b1e00a67f839121472b\")>\n",
            "  Query: \n",
            "  Display: 20\n",
            "  Page: 1\n",
            "  Total Results: 6193247\n",
            "  Results: []\n",
            "      \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "I2h1JbV71Gtn"
      },
      "source": [
        "## Declarative bot behavior\n",
        "\n",
        "This machine just describes the bot data-collecting behaviour, it does not process nor analize data."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "dZ3a0xJPgsNu"
      },
      "source": [
        "def scopusMachine(wd=None):\n",
        "  return {\n",
        "    \"initial\": \"search.page\",\n",
        "    \"context\": {\n",
        "        \"wb\": wd,\n",
        "        \"current_query\": \"\",\n",
        "        \"current_display\": \"20\",\n",
        "        \"search_results\": [] # raw html lement\n",
        "        # show_abstracts: false\n",
        "        # page: 1\n",
        "        # total_pages: total results element / current_display\n",
        "    },\n",
        "    \"states\": {\n",
        "      \"search.page\": {\n",
        "          \"on\" : {\n",
        "            \"SEARCH\": {\n",
        "                \"target\": \"search.results.page\",\n",
        "                \"actions\": [\"wd_search_query(context.wd)\", \"update_current_query(context.current_query)\"]\n",
        "            }    \n",
        "          }\n",
        "      },\n",
        "      \"search.results.page\": {\n",
        "          \"on\": {\n",
        "              \"RETURN.TO.SEARCH\": {\n",
        "                \"target\": \"search.page\",\n",
        "                \"actions\": [\"wd_return_to_search(context.wd)\", \"delete_current_query(context.current_query), \"wd_extract_raw_results(context.wd)\"]\n",
        "              },\n",
        "              \"EXTRACT.RESULTS\": {\n",
        "                \"target\": \"search.page\",\n",
        "                \"actions\": [\"wd_extract_raw_results(context.wd)\"]\n",
        "              },\n",
        "          }\n",
        "      }\n",
        "    }\n",
        "  }\n",
        "\n",
        "# send(machine, event, data)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yFYrWfE90DB3",
        "outputId": "ea2b79f6-5cd7-4651-8606-acc6f5024b62"
      },
      "source": [
        "wd = webdriver.Chrome('chromedriver',options=chrome_options)\n",
        "signed_wd = scopus_signin(wd)\n",
        "\n",
        "# # home_signed_text\n",
        "# print_body(signed_wd)\n",
        "\n",
        "state = {\n",
        "  \"value\": \"search.page\", # search.results.page\n",
        "  \"wd\": signed_wd,\n",
        "  # search state\n",
        "  \"current_query\": \"\",\n",
        "  \"search_results\": [],\n",
        "  \"results_count\": 0,\n",
        "  \"show_all_abstracts\": False,\n",
        "  \"current_display\": 20, # 50, 100, 200\n",
        "}\n",
        "\n",
        "state"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'current_display': 20,\n",
              " 'current_query': '',\n",
              " 'results_count': 0,\n",
              " 'search_results': [],\n",
              " 'show_all_abstracts': False,\n",
              " 'value': 'search.page',\n",
              " 'wd': <selenium.webdriver.chrome.webdriver.WebDriver (session=\"50eaead24443124c0da5e767add03d1f\")>}"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 3
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "mS1Cymwzgagp"
      },
      "source": [
        "def wd_return_to_search(wd):\n",
        "  \"\"\"\n",
        "  transition: search.results.page -> search.page\n",
        "  \"\"\"\n",
        "  search_tab = wd.find_element(By.ID ,\"gh-Search\")\n",
        "  search_tab.click()\n",
        "\n",
        "  # erase current input\n",
        "  search_input = wd.find_element(By.ID ,\"searchterm1\")\n",
        "  search_input.send_keys(Keys.CONTROL, 'a')\n",
        "  search_input.send_keys(Keys.BACKSPACE)\n",
        "  return wd\n",
        "\n",
        "def wd_search_query(query, wd):\n",
        "  \"\"\"\n",
        "  transition: search.page -> search.results.page\n",
        "  \"\"\"\n",
        "  search_input = wd.find_element(By.ID ,\"searchterm1\")\n",
        "  \n",
        "  search_input.send_keys(query)\n",
        "\n",
        "  search_button_row = wd.find_element(By.ID ,\"searchBtnRow\")\n",
        "  search_button = search_button_row.find_element(By.XPATH, '//button[@title=\"Search\"]')\n",
        "  search_button.click()\n",
        "\n",
        "  return wd\n",
        "\n",
        "################################################################################ \n",
        "\n",
        "def wd_extract_results_count(wd):\n",
        "  \"\"\"\n",
        "  transition: search.results.page -> search.results.page\n",
        "  \"\"\"\n",
        "  results_count = (int)(wd.find_element(By.CLASS_NAME ,'resultsCount').text.replace(',', ''))\n",
        "  return results_count\n",
        "\n",
        "def wd_show_all_abstracts(wd):\n",
        "  \"\"\"\n",
        "  transition: search.results.page -> search.results.page\n",
        "  \"\"\"\n",
        "  show_abstracts_link = wd.find_element(By.ID ,\"previewAllAbstractsLinkText\")\n",
        "  show_abstracts_link.click()\n",
        "\n",
        "  return True\n",
        "\n",
        "def wd_extract_raw_results(results_count, current_display, wd):\n",
        "  \"\"\"\n",
        "  transition: search.results.page -> search.results.page\n",
        "  \"\"\"\n",
        "\n",
        "  # ws_change_display(200)\n",
        "  select_display = wd.find_element(By.ID ,\"resultsPerPage-button\")\n",
        "  select_display.click()\n",
        "\n",
        "  pages_200 = wd.find_element(By.ID ,\"ui-id-4\")\n",
        "  pages_200.click()\n",
        "\n",
        "  WebDriverWait(wd, 10).until(EC.presence_of_element_located((By.ID, \"previewAllAbstractsLinkText\")))\n",
        "\n",
        "  # ws_show_all_abstracts\n",
        "  show_abstracts_link = wd.find_element(By.ID ,\"previewAllAbstractsLinkText\")\n",
        "  if \"Hide\" in show_abstracts_link.text:\n",
        "    print(\"Don't click!\")\n",
        "    print(show_abstracts_link.text)\n",
        "  if \"Show\" in show_abstracts_link.text:\n",
        "    print(\"Click!\")\n",
        "    print(show_abstracts_link.text)\n",
        "    show_abstracts_link.click()\n",
        "\n",
        "  # WebDriverWait(wd, 10).until(EC.presence_of_element_located((By.ID, \"previewAbstract1\")))\n",
        "  WebDriverWait(wd, 10).until(EC.presence_of_element_located((By.ID, \"previewAbstract200\")))\n",
        "\n",
        "  #this does not have support for pages yet\n",
        "  results = [] \n",
        "  # for i in range(0, min(results_count, current_display)):\n",
        "  for i in range(0, min(results_count, 200)):\n",
        "    row_title = wd.find_element(By.ID , f'resultDataRow{i}').text\n",
        "    row_abstract = wd.find_element(By.ID , f'previewAbstract{i + 1}').text\n",
        "    results.append({\n",
        "        \"title\": row_title,\n",
        "        \"abstract\": row_abstract\n",
        "    })\n",
        "\n",
        "  return results\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "cXZHYssThgdF",
        "outputId": "1f6a0196-13a2-4dee-8532-836e1d18f122"
      },
      "source": [
        "state[\"current_query\"] = \"protein\"\n",
        "state[\"wd\"] = wd_search_query(state[\"current_query\"], state[\"wd\"])\n",
        "state[\"value\"] = \"search.results.page\"\n",
        "state[\"show_all_abstracts\"] = wd_show_all_abstracts(state[\"wd\"])\n",
        "state[\"results_count\"] = wd_extract_results_count(state[\"wd\"])\n",
        "state[\"search_results\"] = wd_extract_raw_results(state[\"results_count\"], state[\"current_display\"], state[\"wd\"])\n",
        "# display(state)\n",
        "\n",
        "# functional properties\n",
        "# viscosity\n",
        "# milk\n",
        "# soy\n",
        "# SEARCH RESULTS SIZE - 200\n",
        "# NEXT PAGE\n",
        "\n",
        "state[\"current_query\"] = \"\"\n",
        "state[\"wd\"] = wd_return_to_search(state[\"wd\"])\n",
        "state[\"value\"] = \"search.page\"\n",
        "display(state)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Don't click!\n",
            "Hide all abstracts\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "{'current_display': 20,\n",
              " 'current_query': '',\n",
              " 'results_count': 6189842,\n",
              " 'search_results': [{'abstract': 'Copyright © 2015 by The American Society for Pharmacology and Experimental Therapeutics. G protein-coupled receptors (GPCRs) contribute to the regulation of every aspect of human physiology and are therapeutic targets for the treatment of numerous diseases. As a consequence, understanding the myriad of mechanisms controlling GPCR signaling and trafficking is essential for the development of new pharmacological strategies for the treatment of human pathologies. Of the many GPCR-interacting proteins, postsynaptic density protein of 95 kilodaltons, disc large, zona occludens-1 (PDZ) domain-containing proteins appear most abundant and have similarly been implicated in disease mechanisms. PDZ proteins play an important role in regulating receptor and channel protein localization within synapses and tight junctions and function to scaffold intracellular signaling protein complexes. In the current study, we review the known functional interactions between PDZ domain-containing proteins and GPCRs and provide insight into the potential mechanisms of action. These PDZ domain-containing proteins include the membraneassociated guanylate-like kinases [postsynaptic density protein of 95 kilodaltons; synapse-associated protein of 97 kilodaltons; postsynaptic density protein of 93 kilodaltons; synapse-associated protein of 102 kilodaltons; discs, large homolog 5; caspase activation and recruitment domain and membrane-associated guanylate-like kinase domain-containing protein 3; membrane protein, palmitoylated 3; calcium/calmodulin-dependent serine protein kinase; membrane-associated guanylate kinase protein (MAGI)-1,MAGI-2, and MAGI-3], Na+/H+ exchanger regulatory factor proteins (NHERFs) (NHERF1, NHERF2, PDZ domain- containing kidney protein 1, and PDZ domain-containing kidney protein 2), Golgi-associated PDZ proteins (Gα-binding protein interacting protein, C-terminus and CFTR-associated ligand), PDZ domain-containing guanine nucleotide exchange factors (GEFs) 1 and 2, regulator of G protein signaling (RGS)- homology-RhoGEFs (PDZ domain-containing RhoGEF and leukemia-associated RhoGEF), RGS3 and RGS12, spinophilin and neurabin-1, SRC homology 3 domain and multiple ankyrin repeat domain (Shank) proteins (Shank1, Shank2, and Shank3), partitioning defective proteins 3 and 6, multiple PDZ protein 1, Tamalin, neuronal nitric oxide synthase, syntrophins, protein interacting with protein kinase C α 1, syntenin-1, and sorting nexin 27.',\n",
              "   'title': '1\\nPDZ protein regulation of g protein-coupled receptor trafficking and signaling pathways\\nOpen Access\\nDunn, H.A., Ferguson, S.S.G. 2015 Molecular Pharmacology\\n88(4), pp. 624-629\\n54'},\n",
              "  {'abstract': 'Aim: This study identified protein-protein interactions among the biosynthetic machinery responsible for exopolysaccharide (EPS) production in Streptococcus thermophilus MR-1C. Methods and Results: Protein-protein interactions were investigated using the yeast two-hybrid system. A strong protein-protein interaction was detected between the transmembrane activation protein Wzd and the protein tyrosine kinase Wze. Weaker protein-protein interactions were detected between two duplicate Wze proteins and between Wze and the phosphotyrosine phosphatase Wzh. Protein-protein interactions involving a Wzd/Wze fusion protein and Wzd and Wze may indicate that these proteins form multi-protein complexes. All combinations of the Wzh, Wzd, Wze, Wzg (regulation), CpsE (glycosyl-1-phosphate transferase), CpsS (polymerization), CpsL (unknown), CpsW (regulation) and CpsU (membrane translocation) were analysed for protein-protein interactions but no additional interactions were discovered using the yeast two-hybrid system. Conclusions: Interactions among the phosphotyrosine phosphatase, tyrosine kinase, and transmembrane activation protein are important in the regulation of capsule biosynthesis in Strep. thermophilus MR-1C. Significance and Impact of the Study: This study provides some valuable insight into the organization and interactions between the many proteins involved in EPS production. A better understanding of this process may facilitate the genetic manipulation of capsule production to impart desirable properties to dairy starter cultures. © 2011 The Authors. Journal of Applied Microbiology © 2011 The Society for Applied Microbiology.',\n",
              "   'title': '2\\nProtein-protein interactions among the components of the biosynthetic machinery responsible for exopolysaccharide production in Streptococcus thermophilus MR-1C\\nOpen Access\\nCefalo, A.D., Broadbent, J.R., Welker, D.L. 2011 Journal of Applied Microbiology\\n110(3), pp. 801-812\\n11'},\n",
              "  {'abstract': 'Severe acute respiratory syndrome coronavirus (SCoV) 7a protein is one of the viral accessory proteins. In expressing cells, 7a protein exhibits a variety of biological activities, including induction of apoptosis, activation of the mitogen-activated protein kinase signaling pathway, inhibition of host protein translation, and suppression of cell growth progression. Analysis of SCoV particles that were purified by either sucrose gradient equilibrium centrifugation or a virus capture assay, in which intact SCoV particles were specifically immunoprecipitated by anti-S protein monoclonal antibody, demonstrated that 7a protein was associated with purified SCoV particles. Coexpression of 7a protein with SCoV S, M, N, and E proteins resulted in production of virus-like particles (VLPs) carrying 7a protein, while 7a protein was not released from cells expressing 7a protein alone. Although interaction between 7a protein and another SCoV accessory protein, 3a, has been reported, 3a protein was dispensable for assembly of 7a protein into VLPs. S protein was not required for the 7a protein incorporation into VLPs, and yet 7a protein interacted with S protein in coexpressing cells. These data established that, in addition to 3a protein, 7a protein was a SCoV accessory protein identified as a SCoV structural protein. Copyright © 2006, American Society for Microbiology. All Rights Reserved.',\n",
              "   'title': '3\\nSevere acute respiratory syndrome coronavirus 7a accessory protein is a viral structural protein\\nOpen Access\\nHuang, C., Ito, N., Tseng, C.-T.K., Makino, S. 2006 Journal of Virology\\n80(15), pp. 7287-7294\\n53'},\n",
              "  {'abstract': '© 2016 American Chemical Society. The split SNAP-tag protein-fragment complementation assay (PCA) is a useful tool for imaging protein-protein interactions (PPIs) in living cells. In contrast to conventional methods employed for imaging PPIs, the split SNAP-tag PCA enables tracking of proteins following dissociation from protein-protein complexes. A limitation of this system, however, is that it only allows for labeling and tracking of one of the proteins forming the protein-protein complex. To track both proteins forming a protein-protein complex, each protein needs to be appropriately labeled. In this study, a split SNAP-CLIP double labeling system is developed and applied for tracking of each protein forming a protein-protein complex. As a proof-of concept, FM protein for PPIs and protein kinase C alpha (PKCα) for translocation are introduced to a split SNAP-CLIP double labeling system. The results show a split SNAP-CLIP double labeling system enables labeling of both proteins in a protein-protein complex and subsequent tracking of each of the proteins following dissociation from the protein-protein complexes in living cells.',\n",
              "   'title': '4\\nDevelopment of a split SNAP-CLIP double labeling system for tracking proteins following dissociation from protein-protein complexes in living cells Mie, M., Naoki, T., Kobatake, E. 2016 Analytical Chemistry\\n88(16), pp. 8166-8171\\n6'},\n",
              "  {'abstract': 'We present biochemical evidence for the functional similarity of Escherichia coli RecO protein and bacteriophage T4 UvsY protein to eukaryotic Rad52 protein. Although Rad52 protein is conserved in eukaryotes, no sequence homologue has been found in prokaryotes or archeabacteria. Rad52 protein has two unique activities: facilitation of replication protein-A (RPA) displacement by Rad51 protein and annealing of RPA-single-stranded DNA (ssDNA) complexes. Both activities require species-specific interaction between Rad52 protein and RPA. Both RecO and UvsY proteins also possess the former property with regard to their cognate ssDNA-binding protein. Here, we report that RecO protein anneals ssDNA that is complexed with only its cognate ssDNA-binding protein, suggesting the involvement of species-specific interactions. Optimal activity for RecO protein occurs after formation of a 1:1 complex with SSB protein. RecR protein, which is known to stimulate RecO protein to facilitate SSB protein displacement by RecA protein, inhibits annealing by RecO protein, suggesting that RecR protein may regulate the choice between the DNA strand invasion versus annealing pathways. In addition, we show that UvsY protein anneals ssDNA; furthermore, ssDNA, which is complexed only with its cognate ssDNA-binding protein, is annealed in the presence of UvsY protein. These results indicate that RecO and possibly UvsY proteins are functional counterparts of Rad52 protein. Based on the conservation of these functions, we propose a modified double-strand break repair model that includes DNA annealing as an important intermediate step.',\n",
              "   'title': '5\\nEscherichia coli RecO protein anneals ssDNA complexed with its cognate ssDNA-binding protein: A common step in genetic recombination\\nOpen Access\\nKantake, N., Madiraju, M.V.V.M., Sugiyama, T., Kowalczykowski, S.C. 2002 Proceedings of the National Academy of Sciences of the United States of America\\n99(24), pp. 15327-15332\\n109'},\n",
              "  {'abstract': 'Microtubules (MTs) play crucial roles in a variety of cell functions, such as mitosis, vesicle transport and cell motility. MTs also compose specialized structures, such as centrosomes, spindles and cilia. However, molecular mechanisms of these MT-based functions and structures are not fully understood. Here, we analyzed MT co-sedimented proteins from rat brain by tandem mass spectrometry (MS) upon ion exchange column chromatography. We identified a total of 391 proteins. These proteins were grouped into 12 categories: 57 MT cytoskeletal proteins, including MT-associated proteins (MAPs) and motor proteins; 66 other cytoskeletal proteins; 4 centrosomal proteins; 10 chaperons; 5 Golgi proteins; 7 mitochondrial proteins; 62 nucleic acid-binding proteins; 14 nuclear proteins; 13 ribosomal proteins; 28 vesicle transport proteins; 83 proteins with diverse function and/or localization; and 42 uncharacterized proteins. Of these uncharacterized proteins, six proteins were expressed in cultured cells, resulting in the identification of three novel components of centrosomes and cilia. Our present method is not specific for MAPs, but is useful for identifying low abundant novel MAPs and components of MT-based structures. Our analysis provides an extensive list of potential candidates for future study of the molecular mechanisms of MT-based functions and structures. Journal compilation © 2008 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.',\n",
              "   'title': '6\\nMass spectrometric analysis of microtubule co-sedimented proteins from rat brain\\nOpen Access\\nSakamoto, T., Uezu, A., Kawauchi, S., (...), Baba, H., Nakanishi, H. 2008 Genes to Cells\\n13(4), pp. 295-312\\n15'},\n",
              "  {'abstract': \"Protein-protein interactions are crucial to biological functions. Consequently, designing drugs to control protein-protein interactions is receiving increasing attention. Protein structures can associate in different ways. Analysis of the structures of protein-protein complexes using amino acid sequence order-independent multiple structural comparison algorithms, led us to conclude that the amino acids Trp, Met, and Phe are important for protein-protein interactions. Hence, in principle, drug design targeting the Trp/Met/Phe should modulate protein functions effectively. Several clusters of the Trp/Met/Phe residues are involved in the p53 protein-protein interactions. The best example in this regard is the Phe19/Trp23 of p53, which binds to transcriptional factors and to the MDM2 protein. In the HIV related proteins, the Trp/Met/Phe residues have roles in the dimerization of the transcriptase (p51/p66) and in cell-fusion processes, including the gp120-CD4 interaction and the gp41 six-helix bundle formation. Trp/Met/Phe residues are preferred in 'normal' functional protein-protein interactions and they also appear to be exploited in amyloid formation, especially the phenylalanine. Comparison of binding propensity and amyloid formation preference reveals that apart from Lysine, Isoleucine is the least structurally conserved in protein binding sites and has a high propensity in sequences forming amyloids. Thus, this may suggest that nature tends to avoid Ile conservation in protein-protein interaction to avoid amyloid formation. In this regards, Trp/Met/Phe as well as Ile may be targeted to modulate protein-protein interaction. © 2007 Bentham Science Publishers Ltd.\",\n",
              "   'title': '7\\nTrp/Met/Phe hot spots in protein-protein interactions: Potential targets in drug design Ma, B., Nussinov, R. 2007 Current Topics in Medicinal Chemistry\\n7(10), pp. 999-1005\\n74'},\n",
              "  {'abstract': 'Regulators of G-protein-signaling (RGS) proteins are a category of intracellular proteins that have an inhibitory effect on the intracellular signaling produced by G-protein-coupled receptors (GPCRs). RGS along with RGS-like proteins switch on through direct contact G-alpha subunits providing a variety of intracellular functions through intracellular signaling. RGS proteins have a common RGS domain that binds to G alpha. RGS proteins accelerate GTPase and thus enhance guanosine triphosphate hydrolysis through the alpha subunit of heterotrimeric G proteins. As a result, they inactivate the G protein and quickly turn off GPCR signaling thus terminating the resulting downstream signals. Activity and subcellular localization of RGS proteins can be changed through covalent molecular changes to the enzyme, differential gene splicing, and processing of the protein. Other roles of RGS proteins have shown them to not be solely committed to being inhibitors but behave more as modulators and integrators of signaling. RGS proteins modulate the duration and kinetics of slow calcium oscillations and rapid phototransduction and ion signaling events. In other cases, RGS proteins integrate G proteins with signaling pathways linked to such diverse cellular responses as cell growth and differentiation, cell motility, and intracellular trafficking. Human and animal studies have revealed that RGS proteins play a vital role in physiology and can be ideal targets for diseases such as those related to addiction where receptor signaling seems continuously switched on. © 2015 Elsevier Inc.',\n",
              "   'title': '8\\nRegulators of G-Protein-Signaling Proteins: Negative Modulators of G-Protein-Coupled Receptor Signaling Woodard, G.E., Jardín, I., Berna-Erro, A., Salido, G.M., Rosado, J.A. 2015 International Review of Cell and Molecular Biology\\n317, pp. 97-183\\n22'},\n",
              "  {'abstract': 'In this study, we presented the construction of a library of expression clones for the herpes simplex virus type 1 (HSV-1) proteome and subcellular localization map of HSV-1 proteins in living cells using yellow fluorescent protein (YFP) fusion proteins. As a result, 21 proteins showed cytoplasmic or subcytoplasmic localization, 16 proteins showed nuclear or subnuclear localization, and others were present both in the nucleus and cytoplasm. Interestingly, most capsid proteins showed enriched or exclusive localization in the nucleus, and most of the envelope proteins showed cytoplasmic localization, suggesting that subcellular localization of the proteins correlated with their functions during virus replication. These results present a subcellular localization map of HSV-1 proteins in living cells, which provide useful information to further characterize the functions of these proteins. © 2010 Springer-Verlag.',\n",
              "   'title': '9\\nCharacterization of the subcellular localization of herpes simplex virus type 1 proteins in living cells Xing, J., Wang, S., Li, Y., (...), Wang, L., Zheng, C. 2011 Medical Microbiology and Immunology\\n200(1), pp. 61-68\\n25'},\n",
              "  {'abstract': 'Protein-protein interactions regulate almost all aspects of cellular signaling and aberrant protein-protein interactions have the potential to cause or contribute to human disease. The modulation of these interactions by drug-like molecules would offer previously unavailable opportunities to explore the relevance of pre-selected protein-protein interactions for cellular signaling, as well as benefits to patients. After an initial period of skepticism with regards to feasibility, there is now an encouraging set of data indicating that the effective and selective modulation of protein-protein interactions by drug-like molecules is attainable. This review highlights selected areas of current research that are aimed at identifying potent inhibitors of disease-relevant protein-protein interactions. © The Thomson Corporation.',\n",
              "   'title': '10\\nSmall-molecule inhibitors of protein-protein interactions Berg, T. 2008 Current Opinion in Drug Discovery and Development\\n11(5), pp. 666-674\\n123'},\n",
              "  {'abstract': 'Mitochondrial protein quality control incorporates an elaborate network of chaperones and proteases that survey the organelle for misfolded or unfolded proteins and toxic aggregates. Repair of misfolded or aggregated protein and proteolytic removal of irreversibly damaged proteins are carried out by the mitochondrial protein quality control system. Initial maturation and folding of the nuclear or mitochondrial-encoded mitochondrial proteins are mediated by processing peptidases and chaperones that interact with the protein translocation machinery. Mitochondrial proteins are subjected to cumulative oxidative damage. Thus, impairment of quality control processes may cause mitochondrial dysfunction. Aging has been associated with a marked decline in the effectiveness of mitochondrial protein quality control. Here, we present an overview of the chaperones and proteases involved in the initial folding and maturation of new, incoming precursor molecules, and the subsequent repair and removal of oxidized aggregated proteins. In addition, we highlight the link between mitochondrial protein quality control mechanisms and the aging process.This article is part of a Special Issue entitled: Posttranslational Protein modifications in biology and Medicine. © 2013 Elsevier B.V.',\n",
              "   'title': '11\\nOxidative stress and mitochondrial protein quality control in aging Lionaki, E., Tavernarakis, N. 2013 Journal of Proteomics\\n92, pp. 181-194\\n32'},\n",
              "  {'abstract': 'The ER (endoplasmic reticulum) is a major protein folding and modification organelle. In its lumen, the ER processes a third of all newly synthesized proteins. To accomplish this task, numerous resident proteins capture the nascent and newly synthesized proteins. The underlying luminal protein-protein interactions, however, are inherently difficult to analyse, mainly due to their transient nature and the rather specialized environment of the ER. To overcome these limitations, we developed a PCA (protein fragment complementation assay) based on the citrine variant of YFP (yellow fluorescent protein). YFP PCA was successfully applied to visualize the protein interactions of the cargo transport receptor ERGIC-53 (endoplasmic reticulum-Golgi intermediate compartment protein of 53 kDa) with its luminal interaction partner MCF02 (multiple coagulation factor deficiency protein 2) and its cargo proteins cathepsin Z and cathepsin C in a specific manner. With the prospect of screening cDNA libraries for novel protein-protein interactions, YFP PCA is a promising emerging technique for mapping protein interactions inside the secretory pathway in a genome-wide setting. ©The Authors.',\n",
              "   'title': '12\\nVisualization of protein interactions inside the secretory pathway Nyfeler, B., Hauri, H.-P. 2007 Biochemical Society Transactions\\n35(5), pp. 970-973\\n9'},\n",
              "  {'abstract': 'Exploring the function of the genome and the encoded proteins has emerged as a new and exciting challenge in the postgenomic era. Novel technologies come into view that promise to be valuable for the investigation not only of single proteins, but of entire protein networks. Protein microarrays are the innovative assay platform for highly parallel in vitro studies of protein-protein interactions. Due to their flexibility and multiplexing capacity, protein microarrays benefit basic research, diagnosis and biomedicine. This review provides an overview on the basic principles of protein microarrays and their potential to multiplex protein-protein interaction studies.',\n",
              "   'title': '13\\nProtein microarrays as a discovery tool for studying protein-protein interactions Korf, U., Wiemann, S. 2005 Expert Review of Proteomics\\n2(1), pp. 13-26\\n22'},\n",
              "  {'abstract': '© 2017 Wiley Periodicals, Inc. Proteins of the nucleic acid-binding proteins superfamily perform such functions as processing, transport, storage, stretching, translation, and degradation of RNA. It is one of the 16 superfamilies containing the OB-fold in protein structures. Here, we have analyzed the superfamily of nucleic acid-binding proteins (the number of sequences exceeds 200,000) and obtained that this superfamily prevalently consists of proteins containing the cold shock DNA-binding domain (ca. 131,000 protein sequences). Proteins containing the S1 domain compose 57% from the cold shock DNA-binding domain family. Furthermore, we have found that the S1 domain was identified mainly in the bacterial proteins (ca. 83%) compared to the eukaryotic and archaeal proteins, which are available in the UniProt database. We have found that the number of multiple repeats of S1 domain in the S1 domain-containing proteins depends on the taxonomic affiliation. All archaeal proteins contain one copy of the S1 domain, while the number of repeats in the eukaryotic proteins varies between 1 and 15 and correlates with the protein size. In the bacterial proteins, the number of repeats is no more than 6, regardless of the protein size. The large variation of the repeat number of S1 domain as one of the structural variants of the OB-fold is a distinctive feature of S1 domain-containing proteins. Proteins from the other families and superfamilies have either one OB-fold or change slightly the repeat numbers. On the whole, it can be supposed that the repeat number is a vital for multifunctional activity of the S1 domain-containing proteins. Proteins 2017; 85:602–613. © 2016 Wiley Periodicals, Inc.',\n",
              "   'title': '14\\nTaxonomic distribution, repeats, and functions of the S1 domain-containing proteins as members of the OB-fold family Deryusheva, E.I., Machulin, A.V., Selivanova, O.M., Galzitskaya, O.V. 2017 Proteins: Structure, Function and Bioinformatics\\n85(4), pp. 602-613\\n5'},\n",
              "  {'abstract': 'Accessory proteins involved in signal processing through heterotrimeric G proteins are generally defined as proteins distinct from G protein-coupled receptor (GPCR), G protein, or classical effectors that regulate the strength/efficiency/specificity of signal transfer upon receptor activation or position these entities in the right microenvironment, contributing to the formation of a functional signal transduction complex. A flurry of recent studies have implicated an additional class of accessory proteins for this system that provide signal input to heterotrimeric G proteins in the absence of a cell surface receptor, serve as alternative binding partners for G protein subunits, provide unexpected modes of G protein regulation, and have introduced additional functional roles for G proteins. This group of accessory proteins includes the recently discovered Activators of G protein Signaling (AGS) proteins identified in a functional screen for receptor-independent activators of G protein signaling as well as several proteins identified in protein interaction screens and genetic screens in model organisms. These accessory proteins may influence GDP dissociation and nucleotide exchange at the G α subunit, alter subunit interactions within heterotrimeric Gαβγ independent of nucleotide exchange, or form complexes with Gα or Gβγ independent of the typical Gαβγ heterotrimer. AGS and related accessory proteins reveal unexpected diversity in G protein subunits as signal transducers within the cell. Copyright © 2006 by Annual Reviews. All rights reserved.',\n",
              "   'title': '15\\nAccessory proteins for G proteins: Partners in signaling Sato, M., Blumer, J.B., Simon, V., Lanier, S.M. 2006 Annual Review of Pharmacology and Toxicology\\n46, pp. 151-187\\n127'},\n",
              "  {'abstract': 'Ribosomal protein S19 (RPS19) is a 16-kDa protein found mainly as a component of the ribosomal 40 S subunit. Its mutations are responsible for Diamond Blackfan anemia, a congenital disease characterized by defective erythroid progenitor maturation. Dysregulation of RPS19 has therefore been implicated in this defective erythropoiesis, although the link between them is still unclear. Two not mutually exclusive hypotheses have been proposed: altered protein synthesis and loss of unknown functions not directly connected with the structural role of RPS19 in the ribosome. A role in rRNA processing has been surmised for the yeast ortholog, whereas the extracellular RPS19 dimer has a monocyte chemotactic activity. Three proteins are known to interact with RPS19: FGF2, complement component 5 receptor 1, and a nucleolar protein called RPS19-binding protein. We have used a yeast two-hybrid approach to identify a fourth protein: the serine-threonine kinase PIM1. The present study describes our use of proteomics strategies to look for proteins interacting with RPS19 to determine its functions. Proteins were isolated by affinity purification with a GST-RPS19 recombinant protein and identified using LCMS/MS analysis coupled to bioinformatics tools. We identified 159 proteins from the following Gene Ontology categories: NT-Pases (ATPases and GTPases; five proteins), hydrolases/helicases (19 proteins), isomerases (two proteins), kinases (three proteins), splicing factors (five proteins), structural constituents of ribosome (29 proteins), transcription factors (11 proteins), transferases (five proteins), transporters (nine proteins), DNA/RNA-binding protein species (53 proteins), other (one dehydrogenase protein, one ligase protein, one peptidase protein, one receptor protein, and one translation elongation factor), and 13 proteins of still unknown function. Proteomics results were validated by affinity purification and Western blotting. These interactions were further confirmed by co-immunoprecipitation using a monoclonal RPS19 antibody. Many interactors are nucleolar proteins and thus are expected to take part in the RPS19 interactome; however, some proteins suggest additional functional roles for RPS19. © 2007 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '16\\nAnalysis of the ribosomal protein S19 interactome\\nOpen Access\\nOrrù, S., Aspesi, A., Armiraglio, M., (...), Santoro, C., Dianzani, I. 2007 Molecular and Cellular Proteomics\\n6(3), pp. 382-393\\n44'},\n",
              "  {'abstract': 'Down syndrome (DS) is the most common genetic disorder with mental retardation and a host of deranged proteins has already been described. Protein hunting leads to rapid accumulation of aberrant proteins and proteomics methods not only allow unambiguous identification of proteins, they are also a powerful tools to identify new or predicted proteins. We applied two-dimensional gel electrophoresis with in-gel digestion of proteins and subsequent MALDI-TOF mass-spectrometrical identification and quantification of spots using specific software on cortical brain samples from 7 controls and 7 samples from fetal DS at the early second trimester. Nine hypothetical proteins were identified: three of them (4833418L03Rik protein Q9D614, mitochondrial inner membrane protein Q16891 and Nit protein 2 Q8WUF0) were significantly and about doublefold reduced in fetal DS brain. Hypothetical proteins CGI 99, FLJ10463, 70 kDa WD-repeat tumor rejection antigen homolog, KSRP, Hypothetical protein 49.6 kDa and Elongin A were comparable between groups. Domain analysis of deranged structures revealed a t_SNARE domain for the Rik protein, indicating involvement of this protein in the exocytotic-synaptic machinery impaired in DS, a CN hydrolase domain for Nit protein 2, possibly reflecting aberrant nitrilase-related metabolism and handling and an inner mitochondrial protein, extending knowledge on the mitochondrial deficit in in fetal DS early in life.',\n",
              "   'title': '17\\nDeranged hypothetical proteins Rik protein, Nit protein 2 and mitochondrial inner membrane protein, Mitofilin, in fetal Down syndrome brain. Myung, J.K., Gulesserian, T., Fountoulakis, M., Lubec, G. 2003 Cellular and molecular biology (Noisy-le-Grand, France)\\n49(5), pp. 739-746\\n34'},\n",
              "  {'abstract': 'Protein microarrays allow unique approaches for interrogating global protein interaction networks. Protein arrays can be divided into two categories: antibody arrays and functional protein arrays. Antibody arrays consist of various antibodies and are appropriate for profiling protein abundance and modifications. Functional full-length protein arrays employ full-length proteins with various post-translational modifications. A key advantage of the latter is rapid parallel processing of large number of proteins for studying highly controlled biochemical activities, protein-protein interactions, protein-nucleic acid interactions, and protein-small molecule interactions. This unit presents a protocol for constructing functional yeast protein microarrays for global kinase substrate identification. This approach enables the rapid determination of protein interaction networks in yeast on a proteome-wide level. The same methodology can be readily applied to higher eukaryotic systems with careful consideration of overexpression strategy. © 2013 John Wiley & Sons, Inc.',\n",
              "   'title': '18\\nPreparation of recombinant protein spotted arrays for proteome-wide identification of kinase targets Im, H., Snyder, M. 2013 Current Protocols in Protein Science\\n(SUPPL.72),27.4\\n5'},\n",
              "  {'abstract': 'Saccharomyces cerevisiae STE5 is an essential component of the pheromone-mediated-mitogen activated protein kinase (MAPK) pathway. The STE5 protein recruits MAPK module kinases (STE11, STE7, and FUS3) to give a specificity for the pheromone pathway. The STE5 protein contains a putative PEST motif for ubiquitin-dependent protein degradation, and its level may be important for regulation of pheromone signal transduction. In this article, we studied the roles of proteins associated with the STE5 protein for its stabilization by analyzing ste deletion mutants. Here, we found that the STE11 kinase performed the most important role in stabilization of the STE5 protein. The level of STE5 protein was significantly low in the absence of STE11 kinase, suggesting essential roles of STE11 in stabilization of the STE5 protein. Immunodetection and Northern blot analyses showed that the low level of the STE5 protein in the ste11Δ mutant is not due to the level of gene expression but to that of protein stability. Measurement of relative binding affinities showed that the STE11 protein tightly interacts with the STE5 protein for its stabilization.',\n",
              "   'title': '19\\nSaccharomyces cerevisiae STE11 May Contribute to the Stabilities of a Scaffold Protein, STE5, in the Pheromone Signaling Pathway Kim, S.-H., Lee, S.-K., Choi, K.-Y. 1998 Molecules and Cells\\n8(2), pp. 130-137\\n5'},\n",
              "  {'abstract': '© 2014 Wiley Periodicals, Inc. Target structure-based virtual screening, which employs protein-small molecule docking to identify potential ligands, has been widely used in small-molecule drug discovery. In the present study, we used a protein-protein docking program to identify proteins that bind to a specific target protein. In the testing phase, an all-to-all protein-protein docking run on a large dataset was performed. The three-dimensional rigid docking program SDOCK was used to examine protein-protein docking on all protein pairs in the dataset. Both the binding affinity and features of the binding energy landscape were considered in the scoring function in order to distinguish positive binding pairs from negative binding pairs. Thus, the lowest docking score, the average Z-score, and convergency of the low-score solutions were incorporated in the analysis. The hybrid scoring function was optimized in the all-to-all docking test. The docking method and the hybrid scoring function were then used to screen for proteins that bind to tumor necrosis factor-α (TNFα), which is a well-known therapeutic target for rheumatoid arthritis and other autoimmune diseases. A protein library containing 677 proteins was used for the screen. Proteins with scores among the top 20% were further examined. Sixteen proteins from the top-ranking 67 proteins were selected for experimental study. Two of these proteins showed significant binding to TNFα in an in vitro binding study. The results of the present study demonstrate the power and potential application of protein-protein docking for the discovery of novel binding proteins for specific protein targets.',\n",
              "   'title': '20\\nDiscovery of binding proteins for a protein target using protein-protein docking-based virtual screening Zhang, C., Tang, B., Wang, Q., Lai, L. 2014 Proteins: Structure, Function and Bioinformatics\\n82(10), pp. 2472-2482\\n3'},\n",
              "  {'abstract': 'The fasciculation and elongation protein ζ1 (FEZ1) is a mammalian orthologue of the Caenorhabditis elegans protein UNC-76, which is necessary for axon growth in that nematode. In previous studies FEZ1 has been found to interact with protein kinase Cζ, DISC1, the agnoprotein of the human polyomavirus JC virus, and E4B, a U-box-type ubiquitin-protein isopeptide ligase. We reported previously that FEZ1 and its paralogue FEZ2 are proteins that interact with NEK1, a protein kinase involved in polycystic kidney disease and DNA repair mechanisms at the G2/M phase of the cell cycle. Here we report the identification of 16 proteins that interact with human FEZ1-(221-396) in a yeast two-hybrid assay of a human fetal brain cDNA library. The 13 interacting proteins of known functions take part either in transcription regulation and chromatin remodeling (6 proteins), the regulation of neuronal cell development (2 proteins) and cellular transport mechanisms (3 proteins) or participate in apoptosis (2 proteins). We were able to confirm eight of the observed interactions by in vitro pull-down assays with recombinant fusion proteins. The confirmed interacting proteins include FEZ1 itself and three transcription controlling proteins (SAP30L, DRAP1, and BAF60a). In mapping studies we found that the C-terminal regions of FEZ1, and especially its coiled-coil region, are involved in its dimerization, its heterodimerization with FEZ2, and in the interaction with 10 of the identified interacting proteins. Our results give further support to the previous speculation of the functional involvement of FEZ1 in neuronal development but suggest further that FEZ1 may also be involved in transcriptional control. © 2006 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '21\\nFEZ1 dimerization and interaction with transcription regulatory proteins involves its coiled-coil region\\nOpen Access\\nAssmann, E.M., Alborghetti, M.R., Camargo, M.E.R., Kobarg, J. 2006 Journal of Biological Chemistry\\n281(15), pp. 9869-9881\\n38'},\n",
              "  {'abstract': 'GPCRs represent the largest family of integral membrane proteins and were first identified as receptor proteins that couple via heterotrimeric G-proteins to regulate a vast variety of effector proteins to modulate cellular function. It is now recognized that GPCRs interact with a myriad of proteins that not only function to attenuate their signalling but also function to couple these receptors to heterotrimeric G-protein-independent signalling pathways. In addition, intracellular and transmembrane proteins associate with GPCRs and regulate their processing in the endoplasmic reticulum, trafficking to the cell surface, compartmentalization to plasma membrane microdomains, endocytosis and trafficking between intracellular membrane compartments. The present review will overview the functional consequence of β-arrestin, receptor activity-modifying proteins (RAMPS), regulators of G-protein signalling (RGS), GPCR-associated sorting proteins (GASPs), Homer, small GTPases, PSD95/Disc Large/Zona Occludens (PDZ), spinophilin, protein phosphatases, calmodulin, optineurin and Src homology 3 (SH3) containing protein interactions with GPCRs. © 2011 The British Pharmacological Society.',\n",
              "   'title': '22\\nRegulation of GPCR activity, trafficking and localization by GPCR-interacting proteins Magalhaes, A.C., Dunn, H., Ferguson, S.S.G. 2012 British Journal of Pharmacology\\n165(6), pp. 1717-1736\\n206'},\n",
              "  {'abstract': '© 2018 by the authors. Licensee MDPI, Basel, Switzerland. Shewanella oneidensis MR-1 can transfer electrons from the intracellular environment to the extracellular space of the cells to reduce the extracellular insoluble electron acceptors (Extracellular Electron Transfer, EET). Benefiting from this EET capability, Shewanella has been widely used in different areas, such as energy production, wastewater treatment, and bioremediation. Genome-wide proteomics data was used to determine the active proteins involved in activating the EET process. We identified 1012 proteins with decreased expression and 811 proteins with increased expression when the EET process changed from inactivation to activation. We then networked these proteins to construct the active protein networks, and identified the top 20 key active proteins by network centralization analysis, including metabolism- and energy-related proteins, signal and transcriptional regulatory proteins, translation-related proteins, and the EET-related proteins. We also constructed the integrated protein interaction and transcriptional regulatory networks for the active proteins, then found three exclusive active network motifs involved in activating the EET process—Bi-feedforward Loop, Regulatory Cascade with a Feedback, and Feedback with a Protein–Protein Interaction (PPI)—and identified the active proteins involved in these motifs. Both enrichment analysis and comparative analysis to the whole-genome data implicated the multiheme c-type cytochromes and multiple signal processing proteins involved in the process. Furthermore, the interactions of these motif-guided active proteins and the involved functional modules were discussed. Collectively, by using network-based methods, this work reported a proteome-wide search for the key active proteins that potentially activate the EET process.',\n",
              "   'title': '23\\nNetwork-based methods for identifying key active proteins in the extracellular electron transfer process in Shewanella oneidensis MR-1\\nOpen Access\\nDing, D., Sun, X. 2018 Genes\\n9(1),41\\n3'},\n",
              "  {'abstract': '© 2015 Taylor & Francis. The molecular architecture of protein-RNA interfaces are analyzed using a non-redundant dataset of 152 protein-RNA complexes. We find that an average protein-RNA interface is smaller than an average protein-DNA interface but larger than an average protein-protein interface. Among the different classes of protein-RNA complexes, interfaces with tRNA are the largest, while the interfaces with the single-stranded RNA are the smallest. Significantly, RNA contributes more to the interface area than its partner protein. Moreover, unlike protein-protein interfaces where the side chain contributes less to the interface area compared to the main chain, the main chain and side chain contributions flipped in protein-RNA interfaces. We find that the protein surface in contact with the RNA in protein-RNA complexes is better packed than that in contact with the DNA in protein-DNA complexes, but loosely packed than that in contact with the protein in protein-protein complexes. Shape complementarity and electrostatic potential are the two major factors that determine the specificity of the protein-RNA interaction. We find that the H-bond density at the protein-RNA interfaces is similar with that of protein-DNA interfaces but higher than the protein-protein interfaces. Unlike protein-DNA interfaces where the deoxyribose has little role in intermolecular H-bonds, due to the presence of an oxygen atom at the 2′ position, the ribose in RNA plays significant role in protein-RNA H-bonds. We find that besides H-bonds, salt bridges and stacking interactions also play significant role in stabilizing protein-nucleic acids interfaces; however, their contribution at the protein-protein interfaces is insignificant.',\n",
              "   'title': '24\\nMolecular architecture of protein-RNA recognition sites Barik, A., Nithin, C., Pilla, S.P., Bahadur, R.P. 2015 Journal of Biomolecular Structure and Dynamics\\n33(12), pp. 2738-2751\\n22'},\n",
              "  {'abstract': 'Functional protein microarrays offer the capability for high-throughput protein interaction analysis and have long promised to be a powerful tool for understanding protein interactions at the proteome scale. Although popular techniques for protein-protein interaction mapping like yeast-two-hybrid and affinity-purification mass spectrometry have performed well for identifying intracellular protein-protein interactions, the study of interactions between extracellular proteins has remained challenging for these methods. Instead, the use of protein microarrays appears to be a robust and efficient method for the identification of interactions among the members of this class of protein. This unit describes methods for extracellular protein microarray production, screening, and analysis. A protocol is described for enhanced detection of low-affinity interactions by generating multivalent complexes using Fc-fusion bait proteins and protein A microbeads, along with a statistical method for hit scoring and identification of nonspecific interactions. © 2013 John Wiley & Sons, Inc.',\n",
              "   'title': '25\\nProtein microarrays for identification of novel extracellular protein-protein interactions Tom, I., Lewin-Koh, N., Ramani, S.R., Gonzalez, L.C. 2013 Current Protocols in Protein Science\\n(SUPPL.72),27.3\\n9'},\n",
              "  {'abstract': 'Protein turnover is a key post-translational event that regulates numerous cellular processes. It enables cells to respond rapidly to intracellular signals and changing environmental conditions by adjusting the levels of pivotal proteins. A major proteolytic pathway involves the ubiquitination of target proteins and subsequent targeting to the 26S proteasome for degradation. Many F-box proteins play a determining role in the substrate specificity of this degradation pathway. In most cases, selective recognition of the target proteins relies on protein-protein interactions mediated by the C-terminal domain of the F-box proteins. In mammals, the occurrence of F-box proteins with a C-terminal SBD (sugar-binding domain) that specifically interacts with high-mannose N-glycans on target glycoproteins has been documented. The identification and characterization of these sugar-binding F-box proteins demonstrated that F-box proteins do not exclusively use protein-protein interactions but also protein-carbohydrate interactions in the Ub (ubiquitin)/proteasome pathway. Recently, putative sugar-binding F-box proteins have been identified in plants. Genome analyses in Arabidopsis and rice revealed the presence of F-box proteins with a C-terminal lectin-related domain homologous with Nictaba, a jasmonate-inducible lectin from tobacco that was shown to interact with the core structure of high-mannose and complex N-glycans. Owing to the high similarity in structure and specificity between Nictaba and the SBD of the mammalian Fbs proteins, a similar role for the plant F-box proteins with a Nictaba domain in nucleocytoplasmic protein degradation in plant cells is suggested. © The Authors Journal Compilation. © 2008 Biochemical Society.',\n",
              "   'title': '26\\nDo F-box proteins with a C-terminal domain homologous with the tobacco lectin play a role in protein degradation in plants? Lannoo, N., Peumans, W.J., Van Damme, E.J.M. 2008 Biochemical Society Transactions\\n36(5), pp. 843-847\\n24'},\n",
              "  {'abstract': 'Proteomics, the study of protein function on a global scale, will play an important role in furthering our understanding of gene functions, complex biological pathways, and discovery of novel drug targets. A number of techniques have been developed for proteomic studies to identify and analyze proteins, compare protein expression levels, and study protein-protein interactions. Recent developments have applied a DNA array-type approach to immobilize proteins on a surface for high-throughput analysis. Here we report the development and construction of protein chips using derivatized glass and nitrocellulose-coated slides and the employment of recombinant proteins fused with green and red fluorescent proteins for detection. Fluorescent signals were found to be proportional to the amount of arrayed proteins and could be readily detected with a conventional fluorescence slide scanner. This technique allows the investigation of protein-protein interactions without the need for additional labeling steps of probe proteins. © 2002 Elsevier Science (USA).',\n",
              "   'title': '27\\nProtein microarrays to detect protein-protein interactions using red and green fluorescent proteins Kukar, T., Eckenrode, S., Gu, Y., (...), She, J.-X., Wu, D. 2002 Analytical Biochemistry\\n306(1), pp. 50-54\\n84'},\n",
              "  {'abstract': 'Many proteins exert their biological roles as components of complexes, and the functions of proteins are often determined by their specific interactions with other proteins. Because of the central importance of protein-protein interactions for cellular processes, the ability to interfere with specific protein-protein interactions provides a powerful means of influencing the function of selected proteins within the cell. Cell-permeable small organic modulators of protein-protein interactions are thus highly desirable tools both for the study of physiological cellular processes and for the treatment of numerous diseased states. Herein a number of protein-protein interactions that are considered to be pharmaceutical targets are presented, which will familiarize the reader with the strategies that have been employed for the successful identification of small molecule modulators of these protein-protein interactions. These encouraging examples suggest that combined research efforts in the areas of functional proteomics, assay development, and organic synthesis will open up novel possibilities for the treatment of human diseases in the future.',\n",
              "   'title': '28\\nModulation of protein-protein interactions with small organic molecules Berg, T. 2003 Angewandte Chemie - International Edition\\n42(22), pp. 2462-2481\\n264'},\n",
              "  {'abstract': 'The S100 proteins comprise at least 25 members, forming the largest group of EF-hand signalling proteins in humans. Although the proteins are expressed in many tissues, each S100 protein has generally been shown to have a preference for expression in one particular tissue or cell type. Three-dimensional structures of several S100 family members have shown that the proteins assume a dimeric structure consisting of two EF-hand motifs per monomer. Calcium binding to these S100 proteins, with the exception of S100A10, results in an approx. 40° alteration in the position of helix III, exposing a broad hydrophobic surface that enables the S100 proteins to interact with a variety of target proteins. More than 90 potential target proteins have been documented for the S100 proteins, including the cytoskeletal proteins tubulin, glial fibrillary acidic protein and F-actin, which have been identified mostly from in vitro experiments. In the last 5 years, efforts have concentrated on quantifying the protein interactions of the S100 proteins, identifying in vivo protein partners and understanding the molecular specificity for target protein interactions. Furthermore, the S100 proteins are the only EF-hand proteins that are known to form both homo- and hetero-dimers, and efforts are underway to determine the stabilities of these complexes and structural rationales for their formation and potential differences in their biological roles. This review highlights both the calcium-dependent and -independent interactions of the S100 proteins, with a focus on the structures of the complexes, differences and similarities in the strengths of the interactions, and preferences for homo- compared with heterodimeric S100 protein assembly. © 2006 Biochemical Society.',\n",
              "   'title': '29\\nCalcium-dependent and -independent interactions of the S100 protein family Santamaria-Kisiel, L., Rintala-Dempsey, A.C., Shaw, G.S. 2006 Biochemical Journal\\n396(2), pp. 201-214\\n438'},\n",
              "  {'abstract': '© 2020 The Author(s) Proteins and their interactions control a plethora of biological functions and enable life. Protein–protein interactions can be highly dynamic, involve proteins with different degrees of “foldedness,” and are often regulated through an intricate network of post-translational modifications. Central parts of protein–protein networks are intrinsically disordered proteins (IDPs). IDPs act as regulatory interaction hubs, enabled by their flexible nature. They employ various modes of binding mechanisms, from folding upon ligand binding to formation of highly dynamic “fuzzy” protein–protein complexes. Mutations or perturbations in regulation of IDPs are hallmarks of many diseases. Protein surfaces play key roles in protein–protein interactions. However, protein surfaces and protein surface accessibility are difficult to study experimentally. NMR-based solvent paramagnetic relaxation enhancement (sPRE) can provide quantitative experimental information on protein surface accessibility, which can be further used to obtain distance information for structure determination, identification of interaction surfaces, conformational changes, and identification of low-populated transient structure and long-range contacts in IDPs and dynamic protein–protein interactions. In this review, we present and discuss state-of the art sPRE techniques and their applications to investigate structure and dynamics of IDPs and protein–protein interactions. Finally, we provide an outline for potential future applications of the sPRE approach in combination with complementary techniques and modeling, to study novel paradigms, such as liquid–liquid phase separation, regulation of IDPs and protein–protein interactions by post-translational modifications, and targeting of disordered proteins.',\n",
              "   'title': '30\\nProbing Surfaces in Dynamic Protein Interactions\\nOpen Access\\nSpreitzer, E., Usluer, S., Madl, T. 2020 Journal of Molecular Biology\\n432(9), pp. 2949-2972\\n1'},\n",
              "  {'abstract': 'Members of the AAA+ superfamily have been identified in all organisms studied to date. They are involved in a wide range of cellular events. In bacteria, representatives of this superfamily are involved in functions as diverse as transcription and protein degradation and play an important role in the protein quality control network. Often they employ a common mechanism to mediate an ATP-dependent unfolding/disassembly of protein-protein or DNA-protein complexes. In an increasing number of examples it appears that the activities of these AAA+ proteins may be modulated by a group of otherwise unrelated proteins, called adaptor proteins. These usually small proteins specifically modify the substrate recognition of their AAA+ partner protein. The occurrence of such adaptor proteins are widespread; representatives have been identified not only in Escherichia coli but also in Bacillus subtilis, not to mention yeast and other eukaryotic organisms. Interestingly, from the currently known examples, it appears that the N domain of AAA+ proteins (the most divergent region of the protein within the family) provides a common platform for the recognition of these diverse adaptor proteins. Finally, the use of adaptor proteins to modulate AAA+ activity is, in some cases, an elegant way to redirect the activity of an AAA+ protein towards a particular substrate without necessarily affecting other activities of that AAA+ protein while, in other cases, the adaptor protein triggers a complete switch in AAA+ activity. © 2002 Published by Elsevier Science B.V. on behalf of the Federation of European Biochemical Societies.',\n",
              "   'title': '31\\nAAA+ proteins and substrate recognition, it all depends on their partner in crime\\nOpen Access\\nDougan, D.A., Mogk, A., Zeth, K., Turgay, K., Bukau, B. 2002 FEBS Letters\\n529(1), pp. 6-10\\n172'},\n",
              "  {'abstract': 'Riemerella anatipestifer was cultured in both iron restriction media and normal media. Two-dimensional gel electrophoresis identified 23 proteins that significantly increased in the iron restriction media. Of them 12 proteins were analyzed with mass spectrography. Nine of 12 proteins belong to 6 different protein families: fibronectin type iii domain protein, secreted subtilase family protein, phosphoglycerate kinase, translation elongation factor, leucine-rich repeat-containing protein, and Galactose-binding domain-like protein. Other 3 proteins were novel with unknown function. Two novel proteins (Riean_1750 and Riean_1752) were expressed in prokaryotic expression systems. The specificities of these 2 novel proteins to R. anatipestifer were confirmed by western-blotting analysis. The ducks immunized with either protein had low mortality challenged by R. anatipestifer, 33.3% and 16.7%, respectively. The ducks developed 100% immunity when immunized with combined Riean_1750 and Riean_1752 proteins. The data suggested 2 novel proteins play important roles in the bacterial survival in the iron restricted environment. They could be used as subunit vaccines of R. anatipestifer. © 2013 Yang et al.',\n",
              "   'title': '32\\nProtein Expressions and Their Immunogenicity from Riemerella anatipestifer Cultured in Iron Restriction Medium\\nOpen Access\\nYang, Y., Gu, C., Liao, Y., (...), Shao, H., Cheng, G. 2013 PLoS ONE\\n8(6),e65901\\n2'},\n",
              "  {'abstract': 'Memory consolidation in mammalian brain is accompanied by widespread reorganization of synaptic contacts and dendritic structure. Understanding of the protein-protein interactions that underlie these structural changes has been hampered by the difficulty of studying protein-protein interactions produced in vivo by signaling, learning, and other physiological responses using current methodologies. Using a novel technique that separates interacting proteins from noninteracting proteins on the basis of their protein-target affinity, we identified 16 proteins for which protein-target binding is altered in vivo by spatial learning, including stathmin, complexin I, 14-3-3, and several structural proteins including F-actin capping protein, tubulin, GFAP, and actin. Interactions between complexin and its targets (p25α and Drac1-like protein) and the interaction between CapZ and tubulin were calcium-dependent. The preponderance of structural proteins and proteins involved in synapse formation and reorganization of growth cones among proteins undergoing memory-specific changes in protein-protein interactions suggests that synaptic structural reorganization is a predominant feature of the consolidation phase of memory. © 2004 Wiley-Liss, Inc.',\n",
              "   'title': '33\\nHippocampal protein-protein interactions in spatial memory Nelson, T.J., Backlund Jr., P.S., Alkon, D.L. 2004 Hippocampus\\n14(1), pp. 46-57\\n42'},\n",
              "  {'abstract': '© 2018, Springer Science+Business Media, LLC, part of Springer Nature. Protein–protein interactions (PPIs) are responsible for a number of key physiological processes in the living cells and underlie the pathomechanism of many diseases. Nowadays, along with the concept of so-called “hot spots” in protein–protein interactions, which are well-defined interface regions responsible for most of the binding energy, these interfaces can be targeted with modulators. In order to apply structure-based design techniques to design PPIs modulators, a three-dimensional structure of protein complex has to be available. In this context in silico approaches, in particular protein–protein docking, are a valuable complement to experimental methods for elucidating 3D structure of protein complexes. Protein–protein docking is easy to use and does not require significant computer resources and time (in contrast to molecular dynamics) and it results in 3D structure of a protein complex (in contrast to sequence-based methods of predicting binding interfaces). However, protein–protein docking cannot address all the aspects of protein dynamics, in particular the global conformational changes during protein complex formation. In spite of this fact, protein–protein docking is widely used to model complexes of water-soluble proteins and less commonly to predict structures of transmembrane protein assemblies, including dimers and oligomers of G protein-coupled receptors (GPCRs). In this chapter we review the principles of protein–protein docking, available algorithms and software and discuss the recent examples, benefits, and drawbacks of protein–protein docking application to water-soluble proteins, membrane anchoring and transmembrane proteins, including GPCRs.',\n",
              "   'title': '34\\nProtein–protein docking in drug design and discovery Kaczor, A.A., Bartuzi, D., Stępniewski, T.M., Matosiuk, D., Selent, J. 2018 Methods in Molecular Biology\\n1762, pp. 285-305\\n6'},\n",
              "  {'abstract': 'Post-translational modifications (PTM) of proteins can control complex and dynamic cellular processes via regulating interactions between key proteins. To understand these regulatory mechanisms, it is critical that we can profile the PTM-dependent protein-protein interactions. However, identifying these interactions can be very difficult using available approaches, as PTMs can be dynamic and often mediate relatively weak protein-protein interactions. We have recently developed CLASPI (cross-linking-assisted and stable isotope labeling in cell culture-based protein identification), a chemical proteomics approach to examine protein-protein interactions mediated by methylation in human cell lysates. Here, we report three extensions of the CLASPI approach. First, we show that CLASPI can be used to analyze methylation-dependent protein-protein interactions in lysates of fission yeast, a genetically tractable model organism. For these studies, we examined trimethylated histone H3 lysine-9 (H3K9Me3)-dependent protein-protein interactions. Second, we demonstrate that CLASPI can be used to examine phosphorylation-dependent protein-protein interactions. In particular, we profile proteins recognizing phosphorylated histone H3 threonine-3 (H3T3-Phos), a mitotic histone \"mark\" appearing exclusively during cell division. Our approach identified survivin, the only known H3T3-Phos-binding protein, as well as other proteins, such as MCAK and KIF2A, that are likely to be involved in weak but selective interactions with this histone phosphorylation \"mark\". Finally, we demonstrate that the CLASPI approach can be used to study the interplay between histone H3T3-Phos and trimethylation on the adjacent residue lysine 4 (H3K4Me3). Together, our findings indicate the CLASPI approach can be broadly applied to profile protein-protein interactions mediated by PTMs. © 2012 The Protein Society.',\n",
              "   'title': '35\\nExamining post-translational modification-mediated protein-protein interactions using a chemical proteomics approach Li, X., Foley, E.A., Kawashima, S.A., (...), Chait, B.T., Kapoor, T.M. 2013 Protein Science\\n22(3), pp. 287-295\\n22'},\n",
              "  {'abstract': 'Protein arrays hold great promise for proteome-scale analysis of protein-protein interaction networks, but the technical challenges have hindered their adoption by proteomics researchers. The crucial issue of design and fabrication of protein arrays have been addressed in several studies, but the detection strategies used for identifying protein-protein interactions have received little attention. In this study, we evaluated six different detection strategies to identify four different protein-protein interaction pairs. We discuss each detection approach in terms of signal-to-background (S/B) ratio, ease of use, and adaptability to high-throughput format. Protein arrays for this study were made by expressing both the bait proteins (proteins captured at the surface) and prey proteins (probes) in cell-free rabbit reticulocyte lysate (RRL) systems. Bait proteins were expressed as HaloTag fusions that allow covalent capture on a HaloTag ligand-coated glass without any prior protein purification step. Prey proteins were expressed and modified with either tags (protein or peptides) or labels (fluorescent or radiometric) for detection. This simple method for creating protein arrays in combination with our analyses of several detection strategies should increase the usefulness of protein array technologies. © 2009 Elsevier Inc. All rights reserved.',\n",
              "   'title': '36\\nProtein-protein interaction studies on protein arrays: Effect of detection strategies on signal-to-background ratios Hurst, R., Hook, B., Slater, M.R., (...), Storts, D.R., Nath, N. 2009 Analytical Biochemistry\\n392(1), pp. 45-53\\n33'},\n",
              "  {'abstract': \"Molecular Recognition Features (MoRFs) are defined as short, intrinsically disordered regions in proteins that undergo disorder-to-order transition upon binding to their partners. As their name suggests, they are implicated in molecular recognition, which serves as the initial step for protein-protein interactions. Membrane proteins constitute approximately 30% of fully sequenced proteomes and are responsible for a wide variety of cellular functions. The aim of the current study was to identify and analyze MoRFs in membrane proteins. Two datasets of MoRFs, transmembrane and peripheral membrane protein MoRFs, were constructed from the Protein Data Bank, and sequence, structural and functional analysis was performed. Characterization of our datasets revealed their unique compositional biases and membrane protein MoRFs were categorized depending on their secondary structure after the interaction with their partners. Moreover, the position of transmembrane protein MoRFs in relation with the protein's topology was determined. Further studies were focused on functional analyses of MoRF-containing proteins and MoRFs' partners, associating them with protein binding, regulation and cell signaling, indicating half of them as putative hubs in protein-protein interaction networks. In conclusion, we provide insights into the disorder-based protein-protein interactions involving membrane proteins. © 2013 Elsevier B.V.\",\n",
              "   'title': '37\\nAnalysis of Molecular Recognition Features (MoRFs) in membrane proteins Kotta-Loizou, I., Tsaousis, G.N., Hamodrakas, S.J. 2013 Biochimica et Biophysica Acta - Proteins and Proteomics\\n1834(4), pp. 798-807\\n15'},\n",
              "  {'abstract': 'Background: Whether a change in protein biosynthesis modulates protein degradation is unknown. Results: Inhibition of protein biosynthesis induces activation of AKT (protein kinase B), leading to activation of E3 ubiquitin ligase and thus degradation of its substrate proteins. Conclusion: Protein degradation is regulated by protein biosynthesis. Significance: This study reveals a novel regulation of protein degradation. © 2013 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '38\\nInhibition of protein synthesis alters protein degradation through activation of protein kinase B (AKT)\\nOpen Access\\nDai, C.-L., Shi, J., Chen, Y., (...), Liu, F., Gong, C.-X. 2013 Journal of Biological Chemistry\\n288(33), pp. 23875-23883\\n26'},\n",
              "  {'abstract': 'In this series of four minireviews the field of scaffold proteins and proteins of similar molecular/cellular functions is overviewed. By binding and bringing into proximity two or more signaling proteins, these proteins direct the flow of information in the cell by activating, coordinating and regulating signaling events in regulatory networks. Here we discuss the categories of scaffolds, anchors, docking proteins and adaptors in some detail, and using many examples we demonstrate that they cover a wide range of functional modes that appear to segregate into three practical categories, simple proteins binding two partners together (adaptors), larger multidomain proteins targeting and regulating more proteins in complex ways (scaffold/anchoring proteins) and proteins specialized to initiate signaling cascades by localizing partners at the cell membrane (docking proteins). It will also be shown, however, that the categories partially overlap and often their names are used interchangeably in the literature. In addition, although not usually considered as scaffolds, several other proteins, such as regulatory proteins with catalytic activity, phosphatase targeting subunits, E3 ubiquitin ligases, ESCRT proteins in endosomal sorting and DNA damage sensors also function by bona fide scaffolding mechanisms. Thus, the field is in a state of continuous advance and expansion, which demands that the classification scheme be regularly updated and, if needed, revised. © 2010 The Authors Journal compilation © 2010 FEBS.',\n",
              "   'title': '39\\nFunctional classification of scaffold proteins and related molecules Buday, L., Tompa, P. 2010 FEBS Journal\\n277(21), pp. 4348-4355\\n54'},\n",
              "  {'abstract': 'The so-called \"hub proteins\" are those proteins in a protein-protein interaction network system that have remarkably higher interaction relations (or degrees) than the others. Therefore, the information of hub proteins can provide very useful insights for selecting or prioritizing targets during drug development. In this paper, by combining the multiagent-based method with the graphical spectrum analysis and immune-genetic algorithm, a novel simulator for identifying the hub proteins from membrane protein interaction networks is proposed. As a demonstration of using the simulator, two hub membrane proteins, YPL227C and YIL147C, were identified from a complicated network system consisting of 1500 membrane proteins. Meanwhile, along with the two identified hub proteins, their molecular functions, biological processes, and cellular components were also revealed. It is anticipated that the hub-protein-simulator may become a very useful tool for system biology and drug development, particularly in deciphering unknown protein functions, determining protein complexes, and in identifying the key targets from a complicated disease system. © 2010 Bentham Science Publishers Ltd.',\n",
              "   'title': '40\\nIdentifying the hub proteins from complicated membrane protein network systems Shen, Y.-Z., Ding, Y.-S., Gu, Q., Chou, K.-C. 2010 Medicinal Chemistry\\n6(3), pp. 165-173\\n15'},\n",
              "  {'abstract': 'Protein microarrays provide a powerful tool for the study of protein function. However, they are not widely used, in part because of the challenges in producing proteins to spot on the arrays. We generated protein microarrays by printing complementary DNAs onto glass slides and then translating target proteins with mammalian reticulocyte lysate. Epitope tags fused to the proteins allowed them to be immobilized in situ. This obviated the need to purify proteins, avoided protein stability problems during storage, and captured sufficient protein for functional studies. We used the technology to map pairwise interactions among 29 human DNA replication initiation proteins, recapitulate the regulation of Cdt1 binding to select replication proteins, and map its geminin-binding domain.',\n",
              "   'title': '41\\nSelf-assembling protein microarrays Ramachandran, N., Hainsworth, E., Bhullar, B., (...), Walter, J.C., LaBaer, J. 2004 Science\\n305(5680), pp. 86-90\\n471'},\n",
              "  {'abstract': '© 2017 Elsevier B.V. In plant organelles specific nucleotide motifs at C to U RNA editing sites are recognized by the PLS-class of pentatricopeptide repeat (PPR) proteins, which are additionally characterized by a C-terminal E domain. The PPR elements bind the nucleotides in the target RNA, while the function of the E domain has remained unknown. At most sites RNA editing also requires multiple organellar RNA editing factor (MORF) proteins. To understand how these two types of proteins are involved in RNA editing complexes, we systematically analyzed their protein-protein interactions. In vivo pull-down and yeast two-hybrid assays show that MORF proteins connect with selected PPR proteins. In a loss of function mutant of MORF1, a single amino acid alteration in the conserved MORF domain abrogates interactions with many PLS-class PPR proteins, implying the requirement of direct interaction to PPR proteins for the RNA editing function of MORF1. Subfragment analyses show that predominantly the N-terminal/central regions of the MORF domain in MORF1 and MORF3 bind the PPR proteins. Within the PPR proteins, the E domains in addition to PPR elements contact MORF proteins. In chimeric PPR proteins, different E domains alter the specificity of the interaction with MORF proteins. The selective interactions between E domain containing PPR and MORF proteins suggest that the E domains and MORF proteins play a key role for specific protein complexes to assemble at different RNA editing sites.',\n",
              "   'title': '42\\nThe conserved domain in MORF proteins has distinct affinities to the PPR and E elements in PPR RNA editing factors Bayer-Császár, E., Haag, S., Jörg, A., (...), Brennicke, A., Takenaka, M. 2017 Biochimica et Biophysica Acta - Gene Regulatory Mechanisms\\n1860(8), pp. 813-828\\n15'},\n",
              "  {'abstract': 'The first step in the secretion of most mammalian proteins is their transport into the lumen of the endoplasmic reticulum (ER). Transport of pre-secretory proteins into the mammalian ER requires signal peptides in the precursor proteins and a protein translocase in the ER membrane. In addition, hitherto unidentified lumenal ER proteins have been shown to be required for vectorial protein translocation. This requirement was confirmed in this study by using proteoliposomes that were made from microsomal detergent extracts and contained either low or high concentrations of lumenal ER proteins. Furthermore, immunoglobulin-heavy-chain-binding protein (BiP) was shown to he able to substitute for the full set of lumenal proteins and, in the case of biotinylated precursor proteins, avidin was found to be able to substitute for lumenal proteins. Thus, the polypeptide-chain-binding protein BiP was identified as one lumenal protein that is involved in efficient vectorial protein translocation into the mammalian ER.',\n",
              "   'title': '43\\nPolypeptide-binding proteins mediate completion of co-translational protein translocation into the mammalian endoplasmic reticulum Tyedmers, J., Lerner, M., Wiedmann, M., Volkmer, J., Zimmermann, R. 2003 EMBO Reports\\n4(5), pp. 505-510\\n52'},\n",
              "  {'abstract': 'Protein-protein and protein-lipid interactions are thought to play the vital role in maintenance and deformation of red blood cell (RBC) membrane. Protein 4.2, a 76-KDa peripheral protein, binds to the cytoplasmic domain of band 3 (CDB3) and also interacts with ankyrin in RBCs. In order to explore the characteristics of protein 4.2-CDB3-ankyrin interactions, three protein 4.2-derived recombinant proteins encompassing amino acid residues 31-200, 1-300, and 187-260 respectively were expressed in Escherichia coli. Their interactions with CDB3 and ankyrin were investigated by using Far-Western blot and pull-down assay. The results showed that the CDB3-binding site of protein 4.2 is located in the region of residues 200-211 and the ankyrin-binding site is located in the region of residues 187-200 of protein 4.2. Our findings also suggested that the ankyrin D34 domain can interact directly with protein 4.2. The proper tertiary structures of these protein 4.2 fragments are essential for protein 4.2-ankyrin interaction. Meanwhile, ankyrin can enhance the interaction between protein 4.2 and CDB3. © Springer 2006.',\n",
              "   'title': '44\\nAssociations of protein 4.2 with band 3 and ankyrin Su, Y., Ding, Y., Jiang, M., (...), Hu, X., Zhang, Z. 2006 Molecular and Cellular Biochemistry\\n289(1-2), pp. 159-166\\n17'},\n",
              "  {'abstract': '© 2015, International Society of Oncology and BioMarkers (ISOBM). The cullin proteins are a family of scaffolding proteins that associate with RING proteins and ubiquitin E3 ligases and mediate substrate–receptor bindings. Thus, cullin proteins regulate the specificity of ubiquitin targeting in the regulation of proteins involved in various cellular processes, including proliferation, differentiation, and apoptosis. There are seven cullin proteins that have been identified in eukaryotes: CUL1, CUL2, CUL3, CUL4A, CUL4B, CUL5, and CUL7/p53-associated parkin-like cytoplasmic protein. All of these proteins contain a conserved cullin homology domain that binds to RING box proteins. Cullin–RING ubiquitin ligase complexes are activated upon post-translational modification by neural precursor cell-expressed, developmentally downregulated protein 8. The aberrant expression of several cullin proteins has been implicated in many cancers though the significance in gastric cancer has been less well investigated. This review provides the first systematic discussion of the associations between all members of the cullin protein family and gastric cancer. Functional and regulatory mechanisms of cullin proteins in gastric carcinoma progression are also summarized along with a discussion concerning future research areas. Accumulating evidence suggests a critical role of cullin proteins in tumorigenesis, and a better understanding of the function of these individual cullin proteins and their targets will help identify potential biomarkers and therapeutic targets.',\n",
              "   'title': '45\\nThe role of cullin proteins in gastric cancer Chen, P., Yao, G.-D. 2016 Tumor Biology\\n37(1), pp. 29-37\\n14'},\n",
              "  {'abstract': 'The combination of leucine-rich repeat (LRR) and immunoglobulin-like (Ig) domains is found in the domain architecture of the Trk neurotrophin receptor protein. Recently dozens of such proteins simultaneously carrying LRR and Ig domains as the Trk receptors have been identified. Given the significant biological roles of Trk and such newly identified proteins, we have searched the public database for human proteins with LRR and Ig domains (collectively termed the leucine-rich repeat and Ig domain-containing protein, LRRIG protein, in this study), and have analyzed the mRNA expression pattern of mouse orthologs of obtained human LRRIG proteins at embryonic day 10. The list of the LRRIG proteins includes 36 human proteins: four LINGO, three NGL, five SALM, three NLRR, three Pal, two ISLR, three LRIG, two GPR, two Adlican, two Peroxidasin-like proteins, three Trk neurotrophin receptors, a yet unnamed protein AAI11068, and three AMIGO. Some molecules (LINGO2, LINGO4, NGL1, SALM1, SALM5, and TrkB) were expressed exclusively in neuronal tissues, whereas others (ISLR1, GPR124, and Adlican2) exhibited non-neuronal expression profiles. However, the majority of LRRIG protein family exhibited broad mRNA tissue-expression profiles. © 2008 Elsevier B.V. All rights reserved.',\n",
              "   'title': '46\\nExpression pattern of LRR and Ig domain-containing protein (LRRIG protein) in the early mouse embryo Homma, S., Shimada, T., Hikake, T., Yaginuma, H. 2009 Gene Expression Patterns\\n9(1), pp. 1-26\\n51'},\n",
              "  {'abstract': 'Background: In the postgenomic era, high throughput protein expression and protein microarray technologies have progressed markedly permitting screening of therapeutic reagents and discovery of novel protein functions. Hexa-histidine is one of the most commonly used fusion tags for protein expression due to its small size and convenient purification via immobilized metal ion affinity chromatography (IMAC). This purification process has been adapted to the protein microarray format, but the quality of in situ His-tagged protein purification on slides has not been systematically evaluated. We established methods to determine the level of purification of such proteins on metal chelate-modified slide surfaces. Optimized in situ purification of His-tagged recombinant proteins has the potential to become the new gold standard for cost-effective generation of high-quality and high-density protein microarrays. Results: Two slide surfaces were examined, chelated Cu2+ slides suspended on a polyethylene glycol (PEG) coating and chelated Ni2+ slides immobilized on a support without PEG coating. Using PEG-coated chelated Cu2+ slides, consistently higher purities of recombinant proteins were measured. An optimized wash buffer (PBST) composed of 10 mM phosphate buffer, 2.7 mM KCl, 140 mM NaCl and 0.05% Tween 20, pH 7.4, further improved protein purity levels. Using Escherichia coli cell lysates expressing 90 recombinant Streptococcus pneumoniae proteins, 73 proteins were successfully immobilized, and 66 proteins were in situ purified with greater than 90% purity. We identified several antigens among the in situ-purified proteins via assays with anti-S. pneumoniae rabbit antibodies and a human patient antiserum, as a demonstration project of large scale microarray-based immunoproteomics profiling. The methodology is compatible with higher throughput formats of in vivo protein expression, eliminates the need for resin-based purification and circumvents protein solubility and denaturation problems caused by buffer exchange steps and freeze-thaw cycles, which are associated with resin-based purification, intermittent protein storage and deposition on microarrays. Conclusion: An optimized platform for in situ protein purification on microarray slides using His-tagged recombinant proteins is a desirable tool for the screening of novel protein functions and protein-protein interactions. In the context of immunoproteomics, such protein microarrays are complimentary to approaches using non-recombinant methods to discover and characterize bacterial antigens. © 2009 Kwon et al; licensee BioMed Central Ltd.',\n",
              "   'title': '47\\nHigh quality protein microarray using in situ protein purification\\nOpen Access\\nKwon, K., Grose, C., Pieper, R., (...), Fleischmann, R.D., Peterson, S.N. 2009 BMC Biotechnology\\n9,1472, pp. 72\\n9'},\n",
              "  {'abstract': '© Cai et al. Epstein-Barr virus (EBV) is the pathogenic factor of numerous human tumors, yet certain of its encoded proteins have not been studied. As a first step for functional identification, we presented the construction of a library of expression constructs for most of the EBV encoded proteins and an explicit subcellular localization map of 81 proteins encoded by EBV in mammalian cells. Viral open reading frames were fused with enhanced yellow fluorescent protein (EYFP) tag in eukaryotic expression plasmid then expressed in COS-7 live cells, and protein localizations were observed by fluorescence microscopy. As results, 34.57% (28 proteins) of all proteins showed pan-nuclear or subnuclear localization, 39.51% (32 proteins) exhibitted pancytoplasmic or subcytoplasmic localization, and 25.93% (21 proteins) were found in both the nucleus and cytoplasm. Interestingly, most envelope proteins presented pancytoplasmic or membranous localization, and most capsid proteins displayed enriched or complete localization in the nucleus, indicating that the subcellular localization of specific proteins are associated with their roles during viral replication. Taken together, the subcellular localization map of EBV proteins in live cells may lay the foundation for further illustrating the functions of EBV-encoded genes in human diseases especially in its relevant tumors.',\n",
              "   'title': '48\\nCharacterization of the subcellular localization of Epstein-Barr virus encoded proteins in live cells\\nOpen Access\\nCai, M., Liao, Z., Chen, T., (...), Hong, G., Li, M. 2017 Oncotarget\\n8(41), pp. 70006-70034\\n11'},\n",
              "  {'abstract': 'Signal processing via heterotrimeric G-proteins in response to cell surface receptors is a central and much investigated aspect of how cells integrate cellular stimuli to produce coordinated biological responses. The system is a target of numerous therapeutic agents and plays an important role in adaptive processes of organs; aberrant processing of signals through these transducing systems is a component of various disease states. In addition to G-protein coupled receptor (GPCR)-mediated activation of G-protein signaling, nature has evolved creative ways to manipulate and utilize the Gαβγ heterotrimer or Gα and Gβγ subunits independent of the cell surface receptor stimuli. In such situations, the G-protein subunits (Gα and Gβγ) may actually be complexed with alternative binding partners independent of the typical heterotrimeric Gαβγ. Such regulatory accessory proteins include the family of regulator of G-protein signaling (RGS) proteins that accelerate the GTPase activity of Gα and various entities that influence nucleotide binding properties and/or subunit interaction. The latter group of proteins includes receptor-independent activators of G-protein signaling (AGS) proteins that play surprising roles in signal processing. This review provides an overview of our current knowledge regarding AGS proteins. AGS proteins are indicative of a growing number of accessory proteins that influence signal propagation, facilitate cross talk between various types of signaling pathways, and provide a platform for diverse functions of both the heterotrimeric Gαβγ and the individual Gα and Gβγ subunits. © 2006 Elsevier Inc. All rights reserved.',\n",
              "   'title': '49\\nMechanistic pathways and biological roles for receptor-independent activators of G-protein signaling Blumer, J.B., Smrcka, A.V., Lanier, S.M. 2007 Pharmacology and Therapeutics\\n113(3), pp. 488-506\\n87'},\n",
              "  {'abstract': \"While protein-surface interactions have been widely studied, relatively little is understood at this time regarding how protein-surface interaction effects are influenced by protein-protein interactions and how these effects combine with the internal stability of a protein to influence its adsorbed-state structure and bioactivity. The objectives of this study were to develop a method to study these combined effects under widely varying protein-protein interaction conditions using hen egg-white lysozyme (HEWL) adsorbed on silica glass, poly(methyl methacrylate), and polyethylene as our model systems. In order to vary protein-protein interaction effects over a wide range, HEWL was first adsorbed to each surface type under widely varying protein solution concentrations for 2. h to saturate the surface, followed by immersion in pure buffer solution for 15. h to equilibrate the adsorbed protein layers in the absence of additionally adsorbing protein. Periodic measurements were made at selected time points of the areal density of the adsorbed protein layer as an indicator of the level of protein-protein interaction effects within the layer, and these values were then correlated with measurements of the adsorbed protein's secondary structure and bioactivity. The results from these studies indicate that protein-protein interaction effects help stabilize the structure of HEWL adsorbed on silica glass, have little influence on the structural behavior of HEWL on HDPE, and actually serve to destabilize HEWL's structure on PMMA. The bioactivity of HEWL on silica glass and HDPE was found to decrease in direct proportion to the degree of adsorption-induce protein unfolding. A direct correlation between bioactivity and the conformational state of adsorbed HEWL was less apparent on PMMA, thus suggesting that other factors influenced HEWL's bioactivity on this surface, such as the accessibility of HEWL's bioactive site being blocked by neighboring proteins or the surface itself. The developed methods provide an effective means to characterize the influence of protein-protein interaction effects and provide new molecular-level insights into how protein-protein interaction effects combine with protein-surface interaction and internal protein stability effects to influence the structure and bioactivity of adsorbed protein. © 2013 Elsevier B.V.\",\n",
              "   'title': '50\\nQuantification of the influence of protein-protein interactions on adsorbed protein structure and bioactivity Wei, Y., Thyparambil, A.A., Latour, R.A. 2013 Colloids and Surfaces B: Biointerfaces\\n110, pp. 363-371\\n26'},\n",
              "  {'abstract': 'Regulators of G protein signaling (RGS) are a family of proteins that attenuate the activity of the trimeric G proteins. RGS proteins act as GTPase-activating proteins (GAPs) for the α subunits of several trimeric G proteins, much like the GAPs that regulate the activity of monomeric G proteins such as Ras. RGS proteins have been cloned from many eukaryotes, and those whose biochemical activity has been characterized regulate the members of the G(i) family of G proteins; some forms can also act on G(q) proteins. In an ongoing effort to elucidate the role of G(z)α in cell signaling, the yeast two-hybrid system was employed to identify proteins that could interact with a mutationally activated form of G(z)α. A novel RGS, termed RGSZ1, was identified that is most closely related to two existing RGS proteins termed RetRGS1 and GAIP. Northern blot analysis revealed that expression of RGSZ1 was limited to brain, and expression was particularly high in the caudate nucleus. Biochemical characterization of recombinant RGSZ1 protein revealed that RGSZ1 was indeed a GAP and, most significantly, showed a marked preference for G(z)α over other members of the G(i)α family. Phosphorylation of G(z)α by protein kinase C, an event known to occur in cells and that was previously shown to influence α-βγ interactions of G(z), rendered the G protein much less susceptible to RGSZ1 action.',\n",
              "   'title': '51\\nRGSZ1, a G(z)-selective regulator of g protein signaling whose action is sensitive to the phosphorylation state of G(z)α\\nOpen Access\\nGlick, J.L., Meigs, T.E., Miron, A., Casey, P.J. 1998 Journal of Biological Chemistry\\n273(40), pp. 26008-26013\\n109'},\n",
              "  {'abstract': '© 2016 Elsevier Ltd. Protein domains and peptide sequences are a powerful tool for conferring specific functions to engineered biomaterials. Protein sequences with a wide variety of functionalities, including structure, bioactivity, protein-protein interactions, and stimuli responsiveness, have been identified, and advances in molecular biology continue to pinpoint new sequences. Protein domains can be combined to make recombinant proteins with multiple functionalities. The high fidelity of the protein translation machinery results in exquisite control over the sequence of recombinant proteins and the resulting properties of protein-based materials. In this review, we discuss protein domains and peptide sequences in the context of functional protein-based materials, composite materials, and their biological applications.',\n",
              "   'title': '52\\nModular protein domains: An engineering approach toward functional biomaterials Lin, C.-Y., Liu, J.C. 2016 Current Opinion in Biotechnology\\n40, pp. 56-63\\n20'},\n",
              "  {'abstract': \"© 2017 Elsevier Ltd Protein misfolding and aggregation unify several devastating neurodegenerative disorders, including Alzheimer's disease, Parkinson's disease, and amyotrophic lateral sclerosis. There are no effective therapeutics for these disorders and none that target the reversal of the aberrant protein misfolding and aggregation that cause disease. Here, I showcase important advances to define, engineer, and apply protein disaggregases to mitigate deleterious protein misfolding and counter neurodegeneration. I focus on two exogenous protein disaggregases, Hsp104 from yeast and gene 3 protein from bacteriophages, as well as endogenous human protein disaggregases, including: (a) Hsp110, Hsp70, Hsp40, and small heat-shock proteins; (b) HtrA1; and (c) NMNAT2 and Hsp90. I suggest that protein-disaggregase modalities can be channeled to treat numerous fatal and presently incurable neurodegenerative diseases.\",\n",
              "   'title': '53\\nDesigner protein disaggregases to counter neurodegenerative disease Shorter, J. 2017 Current Opinion in Genetics and Development\\n44, pp. 1-8\\n30'},\n",
              "  {'abstract': '© Springer Science+Business Media New York 2017. Prediction of protein-protein interaction sites in a protein structure provides important information for elucidating the mechanism of protein function and can also be useful in guiding a modeling or design procedures of protein complex structures. Since prediction methods essentially assess the propensity of amino acids that are likely to be part of a protein docking interface, they can help in designing proteinprotein interactions. Here, we introduce BindML and BindMLþ protein-protein interaction sites prediction methods. BindML predicts protein-protein interaction sites by identifying mutation patterns found in known protein-protein complexes using phylogenetic substitution models. BindMLþ is an extension of BindML for distinguishing permanent and transient types of protein-protein interaction sites. We developed an interactive web-server that provides a convenient interface to assist in structural visualization of protein-protein interactions site predictions. The input data for the web-server are a tertiary structure of interest. BindML and BindMLþ are available at http://kiharalab.org/bindml/ and http://kiharalab.org/ bindml/plus/.',\n",
              "   'title': '54\\nBindML/BindMLþ: Detecting protein-protein interaction interface propensity from amino acid substitution patterns Wei, Q., La, D., Kihara, D. 2017 Methods in Molecular Biology\\n1529, pp. 279-289\\n1'},\n",
              "  {'abstract': '© 2018 Wiley Periodicals, Inc. We compare side chain prediction and packing of core and non-core regions of soluble proteins, protein-protein interfaces, and transmembrane proteins. We first identified or created comparable databases of high-resolution crystal structures of these 3 protein classes. We show that the solvent-inaccessible cores of the 3 classes of proteins are equally densely packed. As a result, the side chains of core residues at protein-protein interfaces and in the membrane-exposed regions of transmembrane proteins can be predicted by the hard-sphere plus stereochemical constraint model with the same high prediction accuracies (>90%) as core residues in soluble proteins. We also find that for all 3 classes of proteins, as one moves away from the solvent-inaccessible core, the packing fraction decreases as the solvent accessibility increases. However, the side chain predictability remains high (80% within (Formula presented.)) up to a relative solvent accessibility, (Formula presented.), for all 3 protein classes. Our results show that (Formula presented.) % of the interface regions in protein complexes are “core”, that is, densely packed with side chain conformations that can be accurately predicted using the hard-sphere model. We propose packing fraction as a metric that can be used to distinguish real protein-protein interactions from designed, non-binding, decoys. Our results also show that cores of membrane proteins are the same as cores of soluble proteins. Thus, the computational methods we are developing for the analysis of the effect of hydrophobic core mutations in soluble proteins will be equally applicable to analyses of mutations in membrane proteins.',\n",
              "   'title': \"55\\nComparing side chain packing in soluble proteins, protein-protein interfaces, and transmembrane proteins\\nOpen Access\\nGaines, J.C., Acebes, S., Virrueta, A., (...), Regan, L., O'Hern, C.S. 2018 Proteins: Structure, Function and Bioinformatics\\n86(5), pp. 581-591\\n4\"},\n",
              "  {'abstract': 'Protein-protein interactions play important roles in nearly all events that take place in a cell. High-throughput experimental techniques enable the study of protein-protein interactions at the proteome scale through systematic identification of physical interactions among all proteins in an organism. High-throughput protein-protein interaction data, with ever-increasing volume, are becoming the foundation for new biological discoveries. A great challenge to bioinformatics is to manage, analyze, and model these data. In this review, we describe several databases that store, query, and visualize protein-protein interaction data. Comparison between experimental techniques shows that each high-throughput technique such as yeast two-hybrid assay or protein complex identification through mass spectrometry has its limitations in detecting certain types of interactions and they are complementary to each other. In silico methods using protein/DNA sequences, domain and structure information to predict protein-protein interaction can expand the scope of experimental data and increase the confidence of certain protein-protein interaction pairs. Protein-protein interaction data correlate with other types of data, including protein function, subcellular location, and gene expression profile. Highly connected proteins are more likely to be essential based on the analyses of the global architecture of large-scale interaction network in yeast. Use of protein-protein interaction networks, preferably in conjunction with other types of data, allows assignment of cellular functions to novel proteins and derivation of new biological pathways. As demonstrated in our study on the yeast signal transduction pathway for amino acid transport, integration of high-throughput data with traditional biology resources can transform the protein-protein interaction data from noisy information into knowledge of cellular mechanisms.',\n",
              "   'title': '56\\nComputational analyses of high-throughput protein-protein interaction data Chen, Y., Xu, D. 2003 Current Protein and Peptide Science\\n4(3), pp. 159-181\\n57'},\n",
              "  {'abstract': 'Bacteria show asymmetric subcellular distribution of many proteins involved in diverse cellular processes such as chemotaxis, motility, actin polymerization, chromosome partitioning and cell division. In many cases, the specific subcellular localization of these proteins is critical for proper regulation and function. Although cellular organization of the bacterial cell clearly plays an important role in cell physiology, systematic studies to uncover asymmetrically distributed proteins have not been reported previously. In this study, we undertook a proteomics approach to uncover polar membrane proteins in Escherichia coli. We identified membrane proteins enriched in E. coli minicells using a combination of two-dimensional electrophoresis and mass spectrometry. Among a total of 173 membrane protein spots that were consistently detected, 36 spots were enriched in minicell membranes, whereas 15 spots were more abundant in rod cell membranes. The minicell-enriched proteins included the inner membrane proteins MCPs, AtpA, AtpB, YiaF and AcrA, the membrane-associated FtsZ protein and the outer membrane proteins YbhC, OmpW, Tsx, Pal, FadL, OmpT and BtuB. We immunolocalized two of the minicell-enriched proteins, OmpW and YiaF, and showed that OmpW is a bona fide polar protein whereas YiaF displays a patchy membrane distribution with a polar and septal bias.',\n",
              "   'title': '57\\nProteomic screening and identification of differentially distributed membrane proteins in Escherichia coli\\nOpen Access\\nLai, E.-M., Nair, U., Phadke, N.D., Maddock, J.R. 2004 Molecular Microbiology\\n52(4), pp. 1029-1044\\n72'},\n",
              "  {'abstract': 'HIV proteins target host hub proteins for transient binding interactions. The presence of viral proteins in the infected cell results in out-competition of host proteins in their interaction with hub proteins, drastically affecting cell physiology. Functional genomics and interactome datasets can be used to quantify the sequence hotspots on the HIV proteome mediating interactions with host hub proteins. In this study, we used the HIV and human interactome databases to identify HIV targeted host hub proteins and their host binding partners (H2). We developed a high throughput computational procedure utilizing motif discovery algorithms on sets of protein sequences, including sequences of HIV and H2 proteins. We identified as HIV sequence hotspots those linear motifs that are highly conserved on HIV sequences and at the same time have a statistically enriched presence on the sequences of H2 proteins. The HIV protein motifs discovered in this study are expressed by subsets of H2 host proteins potentially outcompeted by HIV proteins. A large subset of these motifs is involved in cleavage, nuclear localization, phosphorylation, and transcription factor binding events. Many such motifs are clustered on an HIV sequence in the form of hotspots. The sequential positions of these hotspots are consistent with the curated literature on phenotype altering residue mutations, as well as with existing binding site data. The hotspot map produced in this study is the first global portrayal of HIV motifs involved in altering the host protein network at highly connected hub nodes. © 2011 Sarmady et al.',\n",
              "   'title': '58\\nHIV protein sequence hotspots for crosstalk with host hub proteins\\nOpen Access\\nSarmady, M., Dampier, W., Tozeren, A. 2011 PLoS ONE\\n6(8),e23293\\n14'},\n",
              "  {'abstract': '© The Author(s) 2018. Published by Oxford University Press. All rights reserved. Crystallographic structures of protein complexes are essential to develop proteomic and structural biology methods, as prediction of protein-protein interaction (PPI) sites and protein-protein docking. Such structures can aid the development of protein complexation inhibitors. Complex DataBase (CDB), accessible at www.jct-bioinfo.com/cdb/search, is a database web application for heterodimeric protein crystallographic complexes along with the crystallographic structures of each individual unbound protein. Direct access to crystallographic structures of protein complexes, along with provided annotations, can serve as starting point for constructing new experimental protein complexes sets of any type, for protein binding studies, and the development and evaluation of PPIs prediction methods.',\n",
              "   'title': '59\\nCDB - A database for protein heterodimeric complexes Aker, M., Ohanona, S., Fisher, S., (...), Barnett-Itzhaki, Z., Amitay, M. 2018 Protein Engineering, Design and Selection\\n31(10), pp. 361-365\\n0'},\n",
              "  {'abstract': '© 2018 Elsevier Ltd Prion-like proteins overlap with intrinsically disordered and low-complexity sequence families. These proteins are widespread, especially among mRNA-binding proteins. A salient feature of these proteins is the ability to form protein assemblies with distinct biophysical and functional properties. While prion-like proteins are involved in myriad of cellular processes, we propose potential roles for protein assemblies in regulated protein synthesis. Since proteins are the ultimate functional output of gene expression, when, where, and how much of a particular protein is made dictates the functional state of a cell. Recent finding suggests that the prion-like proteins offer unique advantages in translation regulation and also raises questions regarding formation and regulation of protein assemblies.',\n",
              "   'title': '60\\nTranslational Control by Prion-like Proteins Li, L., McGinnis, J.P., Si, K. 2018 Trends in Cell Biology\\n28(6), pp. 494-505\\n6'},\n",
              "  {'abstract': '© 2017 Elsevier Ltd Solvent molecules interact intimately with proteins and can profoundly regulate their structure and function. However, accurately and efficiently modeling protein solvation effects at the molecular level has been challenging. Here, we present a method that improves the atomic-level modeling of soluble and membrane protein structures and binding by efficiently predicting de novo protein-solvent molecule interactions. The method predicted with unprecedented accuracy buried water molecule positions, solvated protein conformations, and challenging mutational effects on protein binding. When applied to homology modeling, solvent-bound membrane protein structures, pockets, and cavities were recapitulated with near-atomic precision even from distant homologs. Blindly refined atomic-level structures of evolutionary distant G protein-coupled receptors imply strikingly different functional roles of buried solvent between receptor classes. The method should prove useful for refining low-resolution protein structures, accurately modeling drug-binding sites in structurally uncharacterized receptors, and designing solvent-mediated protein catalysis, recognition, ligand binding, and membrane protein signaling. Lai et al. present a method to improve atomic-level modeling of protein structures and binding by efficiently predicting protein-solvent molecule interactions. The approach recapitulated mutational effects on protein binding, modeled solvated cavities from distant homologs, and buried solvent networks in membrane proteins with unprecedented accuracy.',\n",
              "   'title': '61\\nEnhancing Structure Prediction and Design of Soluble and Membrane Proteins with Explicit Solvent-Protein Interactions\\nOpen Access\\nLai, J.K., Ambia, J., Wang, Y., Barth, P. 2017 Structure\\n25(11), pp. 1758-1770.e8\\n7'},\n",
              "  {'abstract': 'To analyze the proteins interacting with the severe acute respiratory syndrome coronavirus (SARS-CoV) envelope (E) protein, a SARS-CoV was engineered including two tags associated to the E protein. Using this virus, complexes of SARS-CoV E and other proteins were purified using a tandem affinity purification system. Several viral and cell proteins including spike, membrane, non-structural protein 3 (nsp3), dynein heavy chain, fatty acid synthase and transmembrane protein 43 bound E protein. In the present work, we focused on the binding of E protein to nsp3 in infected cells and cell-free systems. This interaction was mediated by the N-terminal acidic domain of nsp3. Moreover, nsp3 and E protein colocalized during the infection. It was shown that E protein was ubiquitinated in vitro and in cell culture, suggesting that the interaction between nsp3 and E protein may play a role in the E protein ubiquitination status and therefore on its turnover. © 2010 Elsevier Inc.',\n",
              "   'title': '62\\nThe envelope protein of severe acute respiratory syndrome coronavirus interacts with the non-structural protein 3 and is ubiquitinated\\nOpen Access\\nÁlvarez, E., DeDiego, M.L., Nieto-Torres, J.L., (...), Marcos-Villar, L., Enjuanes, L. 2010 Virology\\n402(2), pp. 281-291\\n27'},\n",
              "  {'abstract': 'Membrane proteins of the Arabidopsis thaliana MRS2 (MGT) family have been characterised as magnesium transporters. Like their bacterial CorA homologues, the plant MRS2 proteins are characterised by an invariable GMN tripeptide motif terminating the first of two closely spaced transmembrane domains at the carboxy-termini. The functional Mg2 + transport channel is assembled as a pentamer in the case of CorA. However, in contrast to the single CorA genes of bacteria, plant genomes encode up to 10 highly divergent MRS2 proteins. To elucidate structure-function relationships and the possibility of plant MRS2 hetero-pentamer formation, we performed protein-protein interaction studies in the yeast mating-based split-ubiquitin system (mbSUS) and concomitant protein modelling using I-TASSER. Despite very restricted sequence similarities and variable polypeptide insertions all AtMRS2 proteins feature the key structural elements determined for the CorA crystal structure. The mbSUS setup conclusively demonstrates protein-protein interactions of any given AtMRS2 protein not only with itself but also highly permissive interactions to varying degrees among all AtMRS2 proteins. AtMRS2-3 seems particularly prone to non-selective, strong interactions with the other homologues. Deletion constructs show that six amino acids may be deleted from the carboxy-terminus and 27 (but not 41) from the amino-terminus of AtMRS2-7 without impairment of homologous or heterologous protein interactions. Despite significant diversification, the plant MRS2 proteins have obviously retained an ancient CorA/MRS2 core structure and the capacity for protein-protein interactions. Plant magnesium homeostasis may be influenced by hetero-oligomer channel formation where different plant MRS2 proteins meet in the same membrane naturally or in transgenic approaches. © 2013 Elsevier B.V.',\n",
              "   'title': '63\\nMembrane protein interactions between different Arabidopsis thaliana MRS2-type magnesium transporters are highly permissive\\nOpen Access\\nSchmitz, J., Tierbach, A., Lenz, H., Meschenmoser, K., Knoop, V. 2013 Biochimica et Biophysica Acta - Biomembranes\\n1828(9), pp. 2032-2040\\n5'},\n",
              "  {'abstract': 'Helix-loop-helix (HLH) proteins are essential factors for lymphocyte development and function. One class of HLH proteins, the E-proteins, regulate many aspects of lymphocyte maturation, survival, proliferation, and differentiation. E-proteins are negatively regulated by another class of HLH proteins known as the Id proteins. The Id proteins function as dominant negative inhibitors of E-proteins by inhibiting their ability to bind DNA. Here we discuss the function and regulation of the Id proteins in lymphocyte development.',\n",
              "   'title': '64\\nThe regulation and function of the Id proteins in lymphocyte development Rivera, R., Murre, C. 2001 Oncogene\\n20(58 REV. ISS. 8), pp. 8308-8316\\n66'},\n",
              "  {'abstract': '© 2015, American Society for Microbiology. We have examined the interactions of wild-type (WT) and matrix protein-deleted (ΔMA) HIV-1 precursor Gag (PrGag) proteins in virus-producing cells using a biotin ligase-tagging approach. To do so, WT and ΔMA PrGag proteins were tagged with the Escherichia coli promiscuous biotin ligase (BirA*), expressed in cells, and examined. Localization patterns of PrGag proteins and biotinylated proteins overlapped, consistent with observations that BirA*-tagged proteins biotinylate neighbor proteins that are in close proximity. Results indicate that BirA*-tagged PrGag proteins biotinylated themselves as well as WT PrGag proteins in trans. Previous data have shown that the HIV-1 Envelope (Env) protein requires an interaction with MA for assembly into virions. Unexpectedly, ΔMA proteins biotinylated Env, whereas WT BirA*-tagged proteins did not, suggesting that the presence of MA made Env inaccessible to biotinylation. We also identified over 50 cellular proteins that were biotinylated by BirA*-tagged PrGag proteins. These included membrane proteins, cytoskeleton-associated proteins, nuclear transport factors, lipid metabolism regulators, translation factors, and RNA-processing proteins. The identification of these biotinylated proteins offers new insights into HIV-1 Gag protein trafficking and activities and provides new potential targets for antiviral interference.',\n",
              "   'title': '65\\nAnalysis of HIV-1 gag protein interactions via biotin ligase tagging\\nOpen Access\\nRitchie, C., Cylinder, I., Platt, E.J., Barklis, E. 2015 Journal of Virology\\n89(7), pp. 3988-4001\\n26'},\n",
              "  {'abstract': '© 2019, The Author(s), under exclusive licence to Springer Nature Limited. Viruses manipulate cellular signalling by inducing the degradation of crucial signal transducers, usually via the ubiquitin–proteasome pathway. Here, we show that the murine cytomegalovirus (Murid herpesvirus 1) M45 protein induces the degradation of two cellular signalling proteins, the nuclear factor κ-light-chain-enhancer of activated B cells (NF-κB) essential modulator (NEMO) and the receptor-interacting protein kinase 1 (RIPK1), via a different mechanism: it induces their sequestration as insoluble protein aggregates and subsequently facilitates their degradation by autophagy. Aggregation of target proteins requires a distinct sequence motif in M45, which we termed ‘induced protein aggregation motif’. In a second step, M45 recruits the retromer component vacuolar protein sorting 26B (VPS26B) and the microtubule-associated protein light chain 3 (LC3)-interacting adaptor protein TBC1D5 to facilitate degradation of aggregates by selective autophagy. The induced protein aggregation motif is conserved in M45-homologous proteins of several human herpesviruses, including herpes simplex virus, Epstein–Barr virus and Kaposi’s sarcoma-associated herpesvirus, but is only partially conserved in the human cytomegalovirus UL45 protein. We further show that the HSV-1 ICP6 protein induces RIPK1 aggregation and degradation in a similar fashion to M45. These data suggest that induced protein aggregation combined with selective autophagy of aggregates (aggrephagy) represents a conserved viral immune-evasion mechanism.',\n",
              "   'title': '66\\nHerpesviruses induce aggregation and selective autophagy of host signalling proteins NEMO and RIPK1 as an immune-evasion mechanism Muscolino, E., Schmitz, R., Loroch, S., (...), Ostermann, E., Brune, W. 2020 Nature Microbiology\\n5(2), pp. 331-342\\n6'},\n",
              "  {'abstract': 'Novel discoveries in molecular disease pathways within the cell, combined with increasing information regarding protein binding partners has lead to a new approach in drug discovery. There is interest in designing drugs to modulate protein-protein interactions as opposed to solely targeting the catalytic active site within a single enzyme or protein. There are many challenges in this new approach to drug discovery, particularly since the protein-protein interface has a larger surface area, can comprise a discontinuous epitope, and is more amorphous and less well defined than the typical drug design target, a small contained enzyme-binding pocket. Computational methods to predict modes of protein-protein interaction, as well as protein interface hot spots, have garnered significant interest, in order to facilitate the development of drugs to successfully disrupt and inhibit protein-protein interactions. This review summarizes some current methods available for computational protein-protein docking, as well as tabulating some examples of the successful design of antagonists and small molecule inhibitors for protein-protein interactions. Several of these drugs are now beginning to appear in the clinic. © 2012 Bentham Science Publishers.',\n",
              "   'title': '67\\nComputational drug design targeting protein-protein interactions Bienstock, R.J. 2012 Current Pharmaceutical Design\\n18(9), pp. 1240-1254\\n56'},\n",
              "  {'abstract': '© 2020 The Author(s). Background: Influenza A virus (IAV) infection is a serious public health problem not only in South East Asia but also in European and African countries. Scientists are using network biology to dig deep into the essential host factors responsible for regulation of virus infections. Researchers can explore the virus invasion into the host cells by studying the virus-host relationship based on their protein-protein interaction network. Methods: In this study, we present a comprehensive IAV-host protein-protein interaction network that is obtained based on the literature-curated protein interaction datasets and some important interaction databases. The network is constructed in Cytoscape and analyzed with its plugins including CytoHubba, CytoCluster, MCODE, ClusterViz and ClusterOne. In addition, Gene Ontology and KEGG enrichment analyses are performed on the highly IAV-associated human proteins. We also compare the current results with those from our previous study on Hepatitis C Virus (HCV)-host protein-protein interaction network in order to find out valuable information. Results: We found out 1027 interactions among 829 proteins of which 14 are viral proteins and 815 belong to human proteins. The viral protein NS1 has the highest number of associations with human proteins followed by NP, PB2 and so on. Among human proteins, LNX2, MEOX2, TFCP2, PRKRA and DVL2 have the most interactions with viral proteins. Based on KEGG pathway enrichment analysis of the highly IAV-associated human proteins, we found out that they are enriched in the KEGG pathway of basal cell carcinoma. Similarly, the result of KEGG analysis of the common host factors involved in IAV and HCV infections shows that these factors are enriched in the infection pathways of Hepatitis B Virus (HBV), Viral Carcinoma, measles and certain other viruses. Conclusion: It is concluded that the list of proteins we identified might be used as potential drug targets for the drug design against the infectious diseases caused by Influenza A Virus and other viruses.',\n",
              "   'title': '68\\nA systems biology-driven approach to construct a comprehensive protein interaction network of influenza A virus with its host\\nOpen Access\\nFarooq, Q.U.A., Shaukat, Z., Aiman, S., Zhou, T., Li, C. 2020 BMC Infectious Diseases\\n20(1),480\\n0'},\n",
              "  {'abstract': 'PDZ proteins, named for the common structural domain shared by the postsynaptic density protein (PSD95), Drosophila disc large tumor suppressor (DlgA), and zonula occludens-1 protein (ZO-1), constitute a family of 200-300 recognized members. These cytoplasmic adapter proteins are capable of assembling a variety of membrane-associated proteins and signaling molecules in short-lived functional units. Here, we review PDZ proteins that participate in the regulation of signaling, trafficking, and function of G protein-coupled receptors. Salient structural features of PDZ proteins that allow them to recognize targeted GPCRs are considered. Scaffolding proteins harboring PDZ domains may contain single or multiple PDZ modules and may also include other protein-protein interaction modules. PDZ proteins may impact receptor signaling by diverse mechanisms that include retaining the receptor at the cell membrane, thereby increasing the duration of ligand binding, as well as importantly influencing GPCR internalization, trafficking, recycling, and intracellular sorting. PDZ proteins are also capable of modifying the assembled complex of accessory proteins such as β-arrestins that themselves regulate GPCR signaling. Additionally, PDZ proteins may modulate GPCR signaling by altering the G protein to which the receptor binds, or affect other regulatory proteins that impact GTPase activity, protein kinase A, phospholipase C, or modify downstream signaling events. Small molecules targeting the PDZ protein-GPCR interaction are being developed and may become important and selective drug candidates. © 2011 Elsevier Inc.',\n",
              "   'title': '69\\nRole of PDZ Proteins in Regulating Trafficking, Signaling, and Function of GPCRs. Means, Motif, and Opportunity Romero, G., Von Zastrow, M., Friedman, P.A. 2011 Advances in Pharmacology\\n62, pp. 279-314\\n95'},\n",
              "  {'abstract': 'Heterotrimeric G-proteins comprised of Gα, Gβ and Gγ proteins are important signal transducers in all eukaryotes. The Gγ protein of the G-protein heterotrimer is crucial for its proper targeting at the plasma membrane and correct functioning. Gγ proteins are significantly smaller and more diverse than the Gα and Gβ proteins. In model plants Arabidopsis and rice that have a single Gα and Gβ protein, the presence of two canonical Gγ proteins provide some diversity to the possible heterotrimeric combinations. Our recent analysis of the latest version of the soybean genome has identified ten Gγ proteins which belong to three distinct families based on their C-termini. We amplified the full length cDNAs, analyzed their detailed expression profile by quantitative PCR, assessed their localization and performed yeast-based interaction analysis to evaluate interaction specificity with different Gβ proteins. Our results show that ten Gγ genes are retained in the soybean genome and have interesting expression profiles across different developmental stages. Six of the newly identified proteins belong to two plant-specific Gγ protein families. Yeast-based interaction analyses predict some degree of interaction specificity between different Gβ and Gγ proteins. This research thus identifies a highly diverse G-protein network from a plant species. Homologs of these novel proteins have been previously identified as QTLs for grain size and yield in rice. © 2011 Choudhury et al.',\n",
              "   'title': '70\\nConventional and novel gγ protein families constitute the heterotrimeric g-protein signaling network in soybean\\nOpen Access\\nChoudhury, S.R., Bisht, N.C., Thompson, R., Todorov, O., Pandey, S. 2011 PLoS ONE\\n6(8),e23361\\n41'},\n",
              "  {'abstract': 'The MoFe protein of the complex metalloenzyme nitrogenase folds as a heterotetramer containing two copies each of the homologous α and β subunits, encoded by the nifD and the nifK genes respectively. Recently, the functional expression of a fusion NifD-K protein of nitrogenase was demonstrated in Azotobacter vinelandii, strongly implying that the MoFe protein is flexible as it could accommodate major structural changes, yet remain functional [M.H. Suh, L. Pulakat, N. Gavini, J. Biol. Chem. 278 (2003) 5353-5360]. This finding led us to further explore the type of interaction between the fused MoFe protein units. We aimed to determine whether an interaction exists between the two fusion MoFe proteins to form a homodimer that is equivalent to native heterotetrameric MoFe protein. Using the Bacteriomatch Two-Hybrid System, translationally fused constructs of NifD-K (fusion) with the full-length λCI of the pBT bait vector and also NifD-K (fusion) with the N-terminal α-RNAP of the pTRG target vector were made. To compare the extent of interaction between the fused NifD-K proteins to that of the β-β interactions in the native MoFe protein, we proceeded to generate translationally fused constructs of NifK with the α-RNAP of the pTRG vector and λCI protein of the pBT vector. The strength of the interaction between the proteins in study was determined by measuring the β-galactosidase activity and extent of ampicillin resistance of the colonies expressing these proteins. This analysis demonstrated that direct protein-protein interaction exists between NifD-K fusion proteins, suggesting that they exist as homodimers. As the interaction takes place at the β-interfaces of the NifD-K fusion proteins, we propose that these homodimers of NifD-K fusion protein may function in a similar manner as that of the heterotetrameric native MoFe protein. The observation that the extent of protein-protein interaction between the β-subunits of the native MoFe protein in BacterioMatch Two-Hybrid System is comparable to the extent of protein-protein interaction observed between the NifD-K fusion proteins in the same system further supports this idea. © 2005 Elsevier Inc. All rights reserved.',\n",
              "   'title': '71\\nFunctional NifD-K fusion protein in Azotobacter vinelandii is a homodimeric complex equivalent to the native heterotetrameric MoFe protein Lahiri, S., Pulakat, L., Gavini, N. 2005 Biochemical and Biophysical Research Communications\\n337(2), pp. 677-684\\n8'},\n",
              "  {'abstract': '© 2019 The Protein Society Protein biogenesis and quality control are essential to maintaining a functional pool of proteins and involve numerous protein factors that dynamically and transiently interact with each other and with the substrate proteins in living cells. Conventional methods are hardly effective for studying dynamic, transient, and weak protein–protein interactions that occur in cells. Herein, we review how the site-directed photocrosslinking approach, which relies on the genetic incorporation of a photoreactive unnatural amino acid into a protein of interest at selected individual amino acid residue positions and the covalent trapping of the interacting proteins upon ultraviolent irradiation, has become a highly efficient way to explore the aspects of protein contacts in living cells. For example, in the past decade, this approach has allowed the profiling of the in vivo substrate proteins of chaperones or proteases under both physiologically optimal and stressful (e.g., acidic) conditions, mapping residues located at protein interfaces, identifying new protein factors involved in the biogenesis of membrane proteins, trapping transiently formed protein complexes, and snapshotting different structural states of a protein. We anticipate that the site-directed photocrosslinking approach will play a fundamental role in dissecting the detailed mechanisms of protein biogenesis, quality control, and dynamics in the future.',\n",
              "   'title': '72\\nBiogenesis, quality control, and structural dynamics of proteins as explored in living cells via site-directed photocrosslinking\\nOpen Access\\nFu, X., Chang, Z. 2019 Protein Science\\n28(7), pp. 1194-1209\\n1'},\n",
              "  {'abstract': \"The RGS (regulators of G-protein signalling) protein family sharpen signalling kinetics through heterotrimeric G-proteins by enhancing the GTPase activity of the G-protein α subunit. Paradoxically, they also accelerate receptor-stimulated activation. We investigated this paradox using the cloned G-protein gated K+ channel as a reporter of the G-protein cycle, and FRET (fluorescence resonance energy transfer) between cyan and yellow fluorescent protein tagged proteins to detect physical interactions. Our results with the neuronal protein, RGS8, show that the enhancement of activation kinetics is a variable phenomenon determined by receptor type, G-protein isoform and RGS8 expression levels. In contrast, deactivation was consistently accelerated after removal of agonist. FRET microscopy revealed a stable physical interaction between RGS8-yellow fluorescent protein and G0 αA-cyan fluorescent protein that occurred in the presence and absence of receptor activation and was not competed away by Gβγ overexpression. FRET was also seen between RGS8 and Gγ, demonstrating that RGS8 binds to the heterotrimeric G-protein as well as G-protein aα subunit-GTP and the transition complex. We propose a novel model for the action of RGS proteins on the G-protein cycle involving participation of the RGS in the ternary complex: for certain combinations of agonist, receptor and G-protein, RGS8 expression improves upon the 'kinetic efficacy' of G-protein activation.\",\n",
              "   'title': '73\\nParticipation of RGS8 in the ternary complex of agonist, receptor and G-protein Benians, A., Nobles, M., Tinker, A. 2004 Biochemical Society Transactions\\n32(6), pp. 1045-1047\\n8'},\n",
              "  {'abstract': 'Background: Currently there is a strong need for methods that help to obtain an accurate description of protein interfaces in order to be able to understand the principles that govern molecular recognition and protein function. Many of the recent efforts to computationally identify and characterize protein networks extract protein interaction information at atomic resolution from the PDB. However, they pay none or little attention to small protein ligands and solvent. They are key components and mediators of protein interactions and fundamental for a complete description of protein interfaces. Interactome profiling requires the development of computational tools to extract and analyze protein-protein, protein-ligand and detailed solvent interaction information from the PDB in an automatic and comparative fashion. Adding this information to the existing one on protein-protein interactions will allow us to better understand protein interaction networks and protein function. Description: SCOWLP (Structural Characterization Of Water, Ligands and Proteins) is a user-friendly and publicly accessible web-based relational database for detailed characterization and visualization of the PDB protein interfaces. The SCOWLP database includes proteins, peptidic-ligands and interface water molecules as descriptors of protein interfaces. It contains currently 74,907 protein interfaces and 2,093,976 residue-residue interactions formed by 60,664 structural units (protein domains and peptidic-ligands) and their interacting solvent. T he SCOWLP web-server allows detailed structural analysis and comparisons of protein interfaces at atomic level by text query of PDB codes and/or by navigating a SCOP-based tree. It includes a visualization tool to interactively display the interfaces and label interacting residues and interface solvent by atomic physicochemical properties. SCOWLP is automatically updated with every SCOP release. Conclusion: SCOWLP enriches substantially the description of protein interfaces by adding detailed interface information of peptidic-ligands and solvent to the existing protein-protein interaction databases. SCOWLP may be of interest to many structural bioinformaticians. It provides a platform for automatic global mapping of protein interfaces at atomic level, representing a useful tool for classification of protein interfaces, protein binding comparative studies, reconstruction of protein complexes and understanding protein networks. The web-server with the database and its additional summary tables used for our analysis are available at http://www.scowlp.org. © 2006Teyra et al; licensee BioMed Central Ltd.',\n",
              "   'title': '74\\nSCOWLP: A web-based database for detailed characterization and visualization of protein interfaces\\nOpen Access\\nTeyra, J., Doms, A., Schroeder, M., Pisabarro, M.T. 2006 BMC Bioinformatics\\n7,104\\n38'},\n",
              "  {'abstract': 'The hepatitis B virus core protein consists of an amino-terminal capsid-assembly domain and a carboxyl-terminal RNA-binding domain. By using the yeast two-hybrid system, two Hsp40/DnaJ chaperone-family proteins, Hdj1 and hTid1, that interact with the carboxyl-terminal region (aa 94-185) of the core protein were identified. Hdj1 is the prototype member of the family and hTid1 is the human homologue of the Drosophila tumour-suppressor protein Tid56. Binding of the viral core protein with the Hsp40 proteins was confirmed by affinity chromatography and immunoprecipitation of transiently expressed proteins. Moreover, in a sucrose gradient, the precursor form of hTid1 co-sedimented with capsid-like particles composed of the full-length core protein. Unlike the general perception of the role of the cellular chaperone proteins in assisting viral protein folding and thus enhancing virus replication, ectopic expression of Hdj1 and hTid1 suppressed replication of HBV in transfected human hepatoma cells. Conversely, RNA interference-mediated knock-down of hTid1 resulted in increased HBV replication. It was found that both Hsp40 proteins specifically accelerated degradation of the viral core and HBx proteins. Our results suggest that the cellular chaperones, through destabilization of viral proteins, exert inhibitory functions on virus replication and hence may play suppressive roles in hepatocellular carcinoma. © 2006 SGM.',\n",
              "   'title': '75\\nNegative regulation of hepatitis B virus replication by cellular Hsp40/DnaJ proteins through destabilization of viral core and X proteins\\nOpen Access\\nSohn, S.-Y., Kim, S.-B., Kim, J., Ahn, B.-Y. 2006 Journal of General Virology\\n87(7), pp. 1883-1891\\n41'},\n",
              "  {'abstract': 'PIAS (protein inhibitor of activated STAT) proteins interact with and modulate the activities of various transcription factors. In this work, we demonstrate that PIAS proteins xα, xβ, 1, and 3 interact with the small ubiquitin-related modifier SUMO-1 and its E2 conjugase, Ubc9, and that PIAS proteins themselves are covalently modified by SUMO-1 (sumoylated). PIAS proteins also tether other sumoylated proteins in a noncovalent fashion. Furthermore, recombinant PIASxα enhances Ubc9-mediated sumoylation of the androgen receptor and c-Jun in vitro. Importantly, PIAS proteins differ in their abilities to promote sumoylation in intact cells. The ability to stimulate protein sumoylation and the interaction with sumoylated proteins are dependent on the conserved PIAS RING finger-like domain. These functions are linked to the activity of PIASxα on androgen receptor-dependent transcription. Collectively, our results imply that PIAS proteins function as SUMO-1-tethering proteins and zinc finger-dependent E3 SUMO protein ligases, and these properties are likely to explain their ability to modulate the activities of various transcription factors.',\n",
              "   'title': '76\\nPIAS proteins modulate transcription factors by functioning as SUMO-1 ligases\\nOpen Access\\nKotaja, N., Karvonen, U., Jänne, O.A., Palvimo, J.J. 2002 Molecular and Cellular Biology\\n22(14), pp. 5222-5234\\n341'},\n",
              "  {'abstract': 'G protein coupled receptors (GPCR) sense diverse ligands and signal via heterotrimeric G proteins. The Saccharomyces cerevisiae GPCR Gpr1 senses glucose and controls filamentous growth via an unusual Gα protein, Gpa2, which lacks any known Gβγ subunits. Our genetic and biochemical studies identify Gpa2 interaction partners (Gpb1/2, Gpg1) and provide evidence that these proteins function as G protein subunit mimics and signaling effectors. Gpb1 and Gpb2 lack the seven WD-40 repeats found in Gβ subunits and instead contain seven kelch repeats implicated in protein-protein interactions. Gβ subunits and the kelch repeat protein galactose oxidase fold into strikingly similar seven-bladed β propellers. Our studies demonstrate that Gpa2 signals in conjunction with Gβ structural mimics and that homologous G protein subunits or effectors may be conserved in multicellular eukaryotes.',\n",
              "   'title': '77\\nThe Gα protein Gpa2 controls yeast differentiation by interacting with kelch repeat proteins that mimic Gβ subunits\\nOpen Access\\nHarashima, T., Heitman, J. 2002 Molecular Cell\\n10(1), pp. 163-173\\n118'},\n",
              "  {'abstract': \"Protein perdeuteration approaches have tremendous value in protein NMR studies, but are limited by the high cost of perdeuterated media. Here, we demonstrate that E. coli cultures expressing proteins using either the condensed single protein production method (cSPP), or conventional pET expression plasmids, can be condensed prior to protein expression, thereby providing high-quality 2H, 13C, 15N-enriched protein samples at 2.5-10% the cost of traditional methods. As an example of the value of such inexpensively-produced perdeuterated proteins, we produced 2H, 13C, 15N-enriched E. coli cold shock protein A (CspA) and EnvZb in 409 condensed phase media, and obtained NMR spectra suitable for 3D structure determination. The cSPP system was also used to produce 2H, 13C, 15N-enriched E. coli plasma membrane protein YaiZ and outer membrane protein X (OmpX) in condensed phase. NMR spectra can be obtained for these membrane proteins produced in the cSPP system following simple detergent extraction, without extensive purification or reconstitution. This allows a membrane protein's structural and functional properties to be characterized prior to reconstitution, or as a probe of the effects of subsequent purification steps on the structural integrity of membrane proteins. We also provide a standardized protocol for production of perdeuterated proteins using the cSPP system. The 10-40 fold reduction in costs of fermentation media provided by using a condensed culture system opens the door to many new applications for perdeuterated proteins in spectroscopic and crystallographic studies. © Springer Science+Business Media B.V. 2010.\",\n",
              "   'title': '78\\nEfficient condensed-phase production of perdeuterated soluble and membrane proteins Schneider, W.M., Tang, Y., Vaiphei, S.T., (...), Roth, M.J., Montelione, G.T. 2010 Journal of Structural and Functional Genomics\\n11(2), pp. 143-154\\n14'},\n",
              "  {'abstract': 'Flooding stress restricts soybean growth, it results in decrease the production. In this report, to understand how nuclear proteins in soybean affected by flooding, abundance changes of those proteins was analyzed. Nuclear proteins were extracted from the root tips of soybean treated with or without flooding stress. The extracted proteins were analyzed using a label-free quantitative proteomic technique. Of a total of 94 nuclear proteins that were found to be responsive to flooding, the 19 and 75 proteins were increased and decreased, respectively. The identified flooding-responsive proteins were functionally classified, revealing that 8 increased proteins changed in protein synthesis, posttranslational modification, and protein degradation, while 34 decreased proteins were involved in transcription, RNA processing, DNA synthesis, and chromatin structure maintenance. Among these proteins, those whose levels changed more than 10 fold included two poly ADP-ribose polymerases and a novel G-domain-containing protein that might be involved in RNA binding. The mRNA expression levels of these three proteins indicated a similar tendency to their protein abundance changes. These results suggest that acceleration of protein poly-ADP-ribosylation and suppression of RNA metabolism may be involved in root tip of soybean under flooding stress. © 2014 Bentham Science Publishers.',\n",
              "   'title': '79\\nIdentification of nuclear proteins in soybean under flooding stress using proteomic technique Oh, M.W., Nanjo, Y., Komatsu, S. 2014 Protein and Peptide Letters\\n21(5), pp. 458-467\\n20'},\n",
              "  {'abstract': 'The E5 proteins are short transmembrane proteins encoded by many animal and human papillomaviruses. These proteins display transforming activity in cultured cells and animals, and they presumably also play a role in the productive virus life cycle. The E5 proteins are thought to act by modulating the activity of cellular proteins. Here, we describe the biological activities of the best-studied E5 proteins and discuss the evidence implicating specific protein targets and pathways in mediating these activities. The primary target of the 44-amino acid BPV1 E5 protein is the PDGF β receptor, whereas the EGF receptor appears to be an important target of the 83-amino acid HPV16 E5 protein. Both E5 proteins also bind to the vacuolar ATPase and affect MHC class I expression and cell-cell communication. Continued studies of the E5 proteins will elucidate important aspects of transmembrane protein-protein interactions, cellular signal transduction, cell biology, virus replication, and tumorigenesis. © 2013 Elsevier Inc.',\n",
              "   'title': '80\\nThe E5 proteins\\nOpen Access\\nDiMaio, D., Petti, L.M. 2013 Virology\\n445(1-2), pp. 99-114\\n121'},\n",
              "  {'abstract': '© 2015 Rene Kizek. Protein microarray technology became an important research tool for study and detection of proteins, protein-protein interactions and a number of other applications. The utilization of nanoparticle-based materials and nanotechnology-based techniques for immobilization allows us not only to extend the surface for biomolecule immobilization resulting in enhanced substrate binding properties, decreased background signals and enhanced reporter systems for more sensitive assays. Generally in contemporarily developed microarray systems, multiple nanotechnology-based techniques are combined. In this review, applications of nanoparticles and nanotechnologies in creating protein microarrays, proteins immobilization and detection are summarized. We anticipate that advanced nanotechnologies can be exploited to expand promising fields of proteins identification, monitoring of protein-protein or drug-protein interactions, or proteins structures.',\n",
              "   'title': '81\\nNanotechnologies in protein microarrays Krizkova, S., Heger, Z., Zalewska, M., (...), Adam, V., Kizek, R. 2015 Nanomedicine\\n10(17), pp. 2743-2755\\n17'},\n",
              "  {'abstract': 'Background: Most cellular proteins function as part of stable protein complexes. We recently showed that around 38% of proteins associate with mRNAs that encode interacting proteins, reflecting the cotranslational formation of the complex between the bait protein and the nascent peptides encoded by the interacting mRNAs. Here we hypothesise that these cotranslational protein-mRNA associations can be used to predict protein-protein interactions.Results: We found that the fission yeast Exo2 protein, which encodes an exonuclease of the XRN1 family, coimmunoprecipitates with the eti1 mRNA, which codes for a protein of unknown function and uninformative sequence. Based on this protein-mRNA association, we predicted that the Exo2 and Eti1 protein are part of the same complex, and confirmed this hypothesis by coimmunoprecipitation and colocalization of the proteins. Similarly, we show that the cotranslational interaction between the Sty1 MAP kinase and the cip2 mRNA, which encodes an RNA-binding protein, predicts a complex between Sty1 and Cip2.Conclusions: Our results demonstrate that cotranslational protein-mRNA associations can be used to identify new components of protein complexes. © 2014 Duncan and Mata; licensee BioMed Central Ltd.',\n",
              "   'title': '82\\nCotranslational protein-RNA associations predict protein-protein interactions\\nOpen Access\\nDuncan, C.D.S., Mata, J. 2014 BMC Genomics\\n15(1),298\\n8'},\n",
              "  {'abstract': 'The identification of AGS proteins as receptor-independent activators of G-protein signaling reveals unexpected mechanisms for the regulation of heterotrimeric G-protein activation and has opened up new areas of research related to the role of G proteins as signal transducers. In addition to their obvious interest associated with G-protein-coupled receptor signaling, AGS proteins might provide alternative binding partners for G-protein subunits that enable them to serve unexpected functions related to cell division, differentiation and organelle structure that might operate independently of a GPCR. Thus, these proteins and the concepts advanced with their discovery highlight the diversity associated with G-protein signaling and present new avenues for the development of therapeutics that target G-protein signaling. © 2005 Elsevier Ltd. All rights reserved.',\n",
              "   'title': '83\\nAGS proteins: Receptor-independent activators of G-protein signaling Blumer, J.B., Cismowski, M.J., Sato, M., Lanier, S.M. 2005 Trends in Pharmacological Sciences\\n26(9), pp. 470-476\\n72'},\n",
              "  {'abstract': 'The yeast Saccharomyces cerevisiae is a widely used platform for the production of heterologous proteins of medical or industrial interest. However, heterologous protein productivity is often restricted due to the limitations of the host strain. In the protein secretory pathway, the protein trafficking between different organelles is catalyzed by the soluble NSF (N-ethylmaleimide-sensitive factor) receptor (SNARE) complex and regulated by the Sec1/Munc18 (SM) proteins. In this study, we report that over-expression of the SM protein encoding genes SEC1 and SLY1, improves the protein secretion in S. cerevisiae. Engineering Sec1p, the SM protein that is involved in vesicle trafficking from Golgi to cell membrane, improves the secretion of heterologous proteins human insulin precursor and α-amylase, and also the secretion of an endogenous protein invertase. Enhancing Sly1p, the SM protein regulating the vesicle fusion from endoplasmic reticulum (ER) to Golgi, increases α-amylase production only. Our study demonstrates that strengthening the protein trafficking in ER-to-Golgi and Golgi-to-plasma membrane process is a novel secretory engineering strategy for improving heterologous protein production in S. cerevisiae. © 2012 Elsevier Inc..',\n",
              "   'title': '84\\nEngineering of vesicle trafficking improves heterologous protein secretion in Saccharomyces cerevisiae Hou, J., Tyo, K., Liu, Z., Petranovic, D., Nielsen, J. 2012 Metabolic Engineering\\n14(2), pp. 120-127\\n59'},\n",
              "  {'abstract': '© 2016 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim. The aims of the study were to: (i) identify differentially regulated proteins in cerebrospinal fluid (CSF) between multiple sclerosis (MS) patients and non-MS controls; (ii) examine the effect of matching the CSF samples on either total protein amount or volume, and compare four protein normalization strategies for CSF protein quantification. CSF from MS patients (n = 37) and controls (n = 64), consisting of other noninflammatory neurological diseases (n = 50) and non neurological spinal anesthetic subjects (n = 14), were analyzed using label-free proteomics, quantifying almost 800 proteins. In total, 122 proteins were significantly regulated (p < 0.05), where 77 proteins had p-value <0.01 or AUC value >0.75. Hierarchical clustering indicated that there were two main groups of MS patients, those with increased levels of inflammatory response proteins and decreased levels of proteins involved in neuronal tissue development (n = 30), and those with normal protein levels for both of these protein groups (n = 7). The main subgroup of controls clustering with the MS patients showing increased inflammation and decreased neuronal tissue development were patients suffering from chronic fatigue. Our data indicate that the preferable way to quantify proteins in CSF is to first match the samples on total protein amount and then normalize the data based on the median intensities, preferably from the CNS-enriched proteins.',\n",
              "   'title': '85\\nLabel-free analysis of human cerebrospinal fluid addressing various normalization strategies and revealing protein groups affected by multiple sclerosis Opsahl, J.A., Vaudel, M., Guldbrandsen, A., (...), Kroksveen, A.C., Berven, F.S. 2016 Proteomics\\n16(7), pp. 1154-1165\\n15'},\n",
              "  {'abstract': 'BTG/Tob family proteins, which are characterized by similarities in their N-terminal BTG/Tob homology domains, control cell growth negatively. Among the BTG/Tob family members, BTG2/TIS21/PC3 proteins have been reported to have short lives and to be degraded by the proteasome. However, the mechanisms regulating the stabilities of other BTG/Tob family proteins have not yet been clarified. Here, we report that BTG1, Tob, and Tob2 proteins, as well as BTG2 protein, are degraded by the ubiquitin-proteasome system; the degradation of Tob protein in HeLa cells and the degradation of BTG1, BTG2, Tob and Tob2 proteins transiently expressed in HEK293 cells were inhibited by treatments with proteasome-specific inhibitors. Co-expression of BTG1, BTG2, Tob, or Tob2 protein with ubiquitin in HEK293 cells revealed specific multiubiquitination of each of the four proteins. Although the full-length and N-terminal truncated forms of BTG1, BTG2, Tob, and Tob2 proteins were unstable, the respective C-terminal truncated forms were found to be almost stable, suggesting that the C-terminal regions control the stabilities of BTG1, BTG2, Tob, and Tob2 proteins. In addition, it was found that the respective C-terminal regions confer instability on green fluorescent protein, a normally stable protein. Thus, it can be concluded that the C-terminal regions are necessary and sufficient to control the stabilities of BTG1, BTG2, Tob, and Tob2 proteins.',\n",
              "   'title': '86\\nAntiproliferative proteins of the BTG/Tob family are degraded by the ubiquitin-proteasome system Sasajima, H., Nakagawa, K., Yokosawa, H. 2002 European Journal of Biochemistry\\n269(14), pp. 3596-3604\\n46'},\n",
              "  {'abstract': '© 2020 The Author(s) Tight and adherens junctions are specialized sites of cell-cell interaction in epithelia and endothelia, and are involved in barrier, adhesion, and signaling functions. These functions are orchestrated by a highly organized meshwork of macromolecules in the membrane and cytoplasmic compartments. In this review, we discuss the structural organization and functions of the major cytoplasmic scaffolding and adaptor proteins of vertebrate apical junctions (ZO proteins, afadin, PLEKHA7, cingulin, paracingulin, polarity complex proteins, and a few others), focusing on their interactions with cytoskeletal and signaling proteins. Furthermore, we discuss recent results highlighting how mechanical tension, protein-protein interactions and post-translational modifications regulate the conformation and function of scaffolding proteins, and how spontaneous phase separation into biomolecular condensates contributes to apical junction assembly. Using a sequence-based algorithm, a large fraction of cytoplasmic proteins of apical junctions are predicted to be phase separating proteins (PSPs), suggesting that formation of biomolecular condensates is a general mechanism to organize cell-cell contacts by clustering proteins.',\n",
              "   'title': '87\\nScaffolding proteins of vertebrate apical junctions: structure, functions and biophysics\\nOpen Access\\nRouaud, F., Sluysmans, S., Flinois, A., (...), Vasileva, E., Citi, S. 2020 Biochimica et Biophysica Acta - Biomembranes\\n1862(10),183399\\n1'},\n",
              "  {'abstract': '© 2018 American Chemical Society. Although the legume-rhizobium symbiosis is a most-important biological process, there is a limited knowledge about the protein interaction network between host and symbiont. Using interolog- A nd domain-based approaches, we constructed an interspecies protein interactome containing 5115 protein-protein interactions between 2291 Glycine max and 290 Bradyrhizobium diazoefficiens USDA 110 proteins. The interactome was further validated by the expression pattern analysis in nodules, gene ontology term semantic similarity, co-expression analysis, and luciferase complementation image assay. In the G. max-B. diazoefficiens interactome, bacterial proteins are mainly ion channel and transporters of carbohydrates and cations, while G. max proteins are mainly involved in the processes of metabolism, signal transduction, and transport. We also identified the top 10 highly interacting proteins (hubs) for each species. Kyoto Encyclopedia of Genes and Genomes pathway analysis for each hub showed that a pair of 14-3-3 proteins (SGF14g and SGF14k) and 5 heat shock proteins in G. max are possibly involved in symbiosis, and 10 hubs in B. diazoefficiens may be important symbiotic effectors. Subnetwork analysis showed that 18 symbiosis-related soluble N-ethylmaleimide sensitive factor attachment protein receptor proteins may play roles in regulating bacterial ion channels, and SGF14g and SGF14k possibly regulate the rhizobium dicarboxylate transport protein DctA. The predicted interactome provide a valuable basis for understanding the molecular mechanism of nodulation in soybean.',\n",
              "   'title': '88\\nBradyrhizobium diazoefficiens USDA 110-Glycine max Interactome Provides Candidate Proteins Associated with Symbiosis\\nOpen Access\\nZhang, L., Liu, J.-Y., Gu, H., (...), Zhang, Z., Zhang, Y.-M. 2018 Journal of Proteome Research\\n17(9), pp. 3061-3074\\n1'},\n",
              "  {'abstract': 'This chapter discusses protein–protein interactions within the endoplasmic reticulum. The translocation of proteins into the endoplasmic reticulum (ER) is the initial transport step for proteins with ER, Golgi, lysosomal, vacuolar, or extracellular destinations. The process of protein translocation into the ER involves a complex series of events and interactions among numerous proteins, including the nascent polypeptide, signal recognition particle, signal-sequence receptor, signal peptidase, and several ER-resident proteins. Recent work in yeast, animal, and plant systems has implicated ER-resident proteins in the folding, assembly, and modification of proteins translocated into the ER. Most seed storage proteins are transported through the ER en route to specialized storage organelles. This chapter focuses on the identification of protein–protein interactions within the ER and ER-derived protein bodies. Successful detection of the interaction between two polypeptides requires a relatively stable protein–protein complex, steric availability of recognizable epitopes upon protein association, and no cross-reactivity between antibodies and associated proteins. © 1995, Academic Press, Inc.',\n",
              "   'title': '89\\nProtein—Protein Interactions within the Endoplasmic Reticulum Gillikin, J.W., Fontes, E.P.B., Boston, R.S. 1995 Methods in Cell Biology\\n50(C), pp. 309-323\\n9'},\n",
              "  {'abstract': 'In living cells, membrane proteins are essential to signal transduction, nutrient use, and energy exchange between the cell and environment. Due to challenges in protein expression, purification and crystallization, deposition of membrane protein structures in the Protein Data Bank lags far behind existing structures for soluble proteins. This review describes recent advances in solution NMR allowing the study of a select set of peripheral and integral membrane proteins. Surface-binding proteins discussed include amphitropic proteins, antimicrobial and anticancer peptides, the HIV-1 gp41 peptides, human α-synuclein and apolipoproteins. Also discussed are transmembrane proteins including bacterial outer membrane β-barrel proteins and oligomeric α-helical proteins. These structural studies are possible due to solubilization of the proteins in membrane-mimetic constructs such as detergent micelles and bicelles. In addition to protein dynamics, protein-lipid interactions such as those between arginines and phosphatidylglycerols have been detected directly by NMR. These examples illustrate the unique role solution NMR spectroscopy plays in structural biology of membrane proteins. © 2008 Bentham Science Publishers Ltd.',\n",
              "   'title': '90\\nNMR of membrane-associated peptides and proteins Wang, G. 2008 Current Protein and Peptide Science\\n9(1), pp. 50-69\\n46'},\n",
              "  {'abstract': 'The nucleocapsid protein N of Chandipura virus is prone to aggregation in vitro. We have shown that this aggregation occurs in two phases in a nucleation-dependent manner. Electron microscopy suggests that the aggregated state may have a ring-like structure. Using a GFP fusion, we have shown that the N-protein also aggregates in vivo. The P-protein suppresses the N-protein aggregation efficiently, both in vitro and in vivo. Increased lag phase in the presence of the P-protein suggests that chaperone-like action of the P-protein occurs before the nucleation event. The P-protein, however, does not exert any chaperone-like action against other proteins, suggesting that it binds to the N-protein specifically. Surface plasmon resonance and fluorescence enhancement indeed suggest that the P-protein binds tightly to the native N-protein. The P-protein is thus an N-protein-specific chaperone which inhibits the nucleation phase of N-protein aggregation, thus keeping a pool of encapsidation-competent N-protein for viral maturation.',\n",
              "   'title': '91\\nP-Protein of Chandipura Virus Is an N-Protein-Specific Chaperone that Acts at the Nucleation Stage Majumdar, A., Bhattacharya, R., Basak, S., (...), Chattopadhyay, D., Roy, S. 2004 Biochemistry\\n43(10), pp. 2863-2870\\n17'},\n",
              "  {'abstract': 'Bacterial periplasmic substrate-binding proteins are initial receptors in the process of active transport across cell membranes and/or chemotaxis. Each of them binds a specific substrate (e.g. sugar, amine acid, or ion) with high affinity. For transport, each binding protein interacts with a cognate membrane complex consisting of two hydrophobic proteins and two subunits of a hydrophilic ATPase. For chemotaxis, binding proteins interact with specific membrane chemotaxis receptors. We report, herewith, that the oligopeptide- binding protein OppA of Escherichia coli, the maltose-binding protein MalE of E. coli, and the galactose-binding protein MglB of Salmonella typhimurium interact with unfolded and denatured proteins, such as the molecular chaperones that are involved in protein folding and protein renaturation after stress. These periplasmic substrate-binding proteins promote the functional folding of citrate synthase and α-glucosidase after urea denaturation. They prevent the aggregation of citrate synthase under heat shock conditions, and they form stable complexes with several unfolded proteins, such as reduced carboxymethyl α-lactalbumin and unfolded bovine pancreatic trypsin inhibitor. These chaperone-like functions are displayed by both the liganded and ligand-free forms of binding proteins, and they occur at binding protein concentrations that are 10-100-fold lower than their periplasmic concentration. These results suggest that bacterial periplasmic substrate-binding proteins, in addition to their function in transport and chemotaxis, might be implicated in protein folding and protection from stress in the periplasm.',\n",
              "   'title': '92\\nChaperone properties of the bacterial periplasmic substrate-binding proteins\\nOpen Access\\nRicharme, G., Caldas, T.D. 1997 Journal of Biological Chemistry\\n272(25), pp. 15607-15612\\n109'},\n",
              "  {'abstract': 'Reversible protein phosphorylation plays a central role in cellular signal transduction and is a focus of biomedical studies. However, it is a challenging task to study the effects of protein phosphorylation in the presence of protein phosphatase activities, especially for protein tyrosine phosphatases SHP1, SHP2 and LMW-PTP, which are themselves regulated by protein tyrosine phosphorylation. Expressed protein ligation, by combining chemical peptide synthesis with recombinant protein expression, allows for site-specific unnatural modifications of semisynthetic proteins. In this review, we describe how semisynthetic proteins were prepared to incorporate nonhydrolyzable phosphotyrosine analogs, and utilized in combination with site-directed mutagenesis and other means to elucidate regulatory mechanisms of protein tyrosine phosphatases. © 2007 Elsevier Inc. All rights reserved.',\n",
              "   'title': '93\\nAnalyzing protein tyrosine phosphatases by phosphotyrosine analog integration Shen, K. 2007 Methods\\n42(3), pp. 234-242\\n4'},\n",
              "  {'abstract': 'Signals between a cell and its environment are often transmitted through membrane proteins; therefore, many membrane proteins, including G protein-coupled receptors (GPCRs) and ion channels, are important drug targets. Structural information about membrane proteins remains limited owing to challenges in protein expression, purification and the selection of membrane-mimicking systems that will retain protein structure and function. This review describes recent advances in solution NMR applied to the structural study of integral membrane proteins. The examples herein demonstrate that solution NMR spectroscopy will play a unique role not only in structural analysis, but also drug discovery of membrane proteins. © 2011 Elsevier Ltd.',\n",
              "   'title': '94\\nSolution NMR study of integral membrane proteins Kang, C., Li, Q. 2011 Current Opinion in Chemical Biology\\n15(4), pp. 560-569\\n62'},\n",
              "  {'abstract': 'Protein-protein interactions are important for nearly all biological processes, and it is known that aberrant protein-protein interactions can lead to human disease and cancer. Recent evidence has suggested that protein interaction interfaces describe a new class of attractive targets for drug development. Full characterization of protein interaction networks of protein complexes and their dynamics in response to various cellular cues will provide essential information for us to understand how protein complexes work together in cells to maintain cell viability and normal homeostasis. Affinity purification coupled with quantitative mass spectrometry has become the primary method for studying in vivo protein interactions of protein complexes and whole organism proteomes. Recent developments in sample preparation and affinity purification strategies allow the capture, identification, and quantification of protein interactions of protein complexes that are stable, dynamic, transient, and/or weak. Current efforts have mainly focused on generating reliable, reproducible, and high confidence protein interaction data sets for functional characterization. The availability of increasing amounts of information on protein interactions in eukaryotic systems and new bioinformatics tools allow functional analysis of quantitative protein interaction data to unravel the biological significance of the identified protein interactions. Existing studies in this area have laid a solid foundation toward generating a complete map of in vivo protein interaction networks of protein complexes in cells or tissues. © 2010 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '95\\nProfiling of protein interaction networks of protein complexes using affinity purification and quantitative mass spectrometry\\nOpen Access\\nKaake, R.M., Wang, X., Huang, L. 2010 Molecular and Cellular Proteomics\\n9(8), pp. 1650-1665\\n74'},\n",
              "  {'abstract': 'F-box proteins, which are the substrate-recognition subunits of SKP1-cullin 1-F-box protein (SCF) E3 ligase complexes, have pivotal roles in multiple cellular processes through ubiquitylation and subsequent degradation of target proteins. Dysregulation of F-box protein-mediated proteolysis leads to human malignancies. Notably, inhibitors that target F-box proteins have shown promising therapeutic potential, urging us to review the current understanding of how F-box proteins contribute to tumorigenesis. As the physiological functions for many of the 69 putative F-box proteins remain elusive, additional genetic and mechanistic studies will help to define the role of each F-box protein in tumorigenesis, thereby paving the road for the rational design of F-box protein-targeted anticancer therapies. © 2014 Macmillan Publishers Limited. All rights reserved.',\n",
              "   'title': '96\\nRoles of F-box proteins in cancer Wang, Z., Liu, P., Inuzuka, H., Wei, W. 2014 Nature Reviews Cancer\\n14(4), pp. 233-247\\n243'},\n",
              "  {'abstract': '© 2018 The Authors. Published under the terms of the CC BY 4.0 license Sequestration of protein aggregates in inclusion bodies and their subsequent degradation prevents proteostasis imbalance, cytotoxicity, and proteinopathies. The underlying molecular mechanisms controlling the turnover of protein aggregates are mostly uncharacterized. Herein, we show that a TRIM family protein, TRIM16, governs the process of stress-induced biogenesis and degradation of protein aggregates. TRIM16 facilitates protein aggregate formation by positively regulating the p62-NRF2 axis. We show that TRIM16 is an integral part of the p62-KEAP1-NRF2 complex and utilizes multiple mechanisms for stabilizing NRF2. Under oxidative and proteotoxic stress conditions, TRIM16 activates ubiquitin pathway genes and p62 via NRF2, leading to ubiquitination of misfolded proteins and formation of protein aggregates. We further show that TRIM16 acts as a scaffold protein and, by interacting with p62, ULK1, ATG16L1, and LC3B, facilitates autophagic degradation of protein aggregates. Thus, TRIM16 streamlines the process of stress-induced aggregate clearance and protects cells against oxidative/proteotoxic stress-induced toxicity in vitro and in vivo. Taken together, this work identifies a new mechanism of protein aggregate turnover, which could be relevant in protein aggregation-associated diseases such as neurodegeneration.',\n",
              "   'title': '97\\nTRIM16 controls assembly and degradation of protein aggregates by modulating the p62-NRF2 axis and autophagy\\nOpen Access\\nJena, K.K., Kolapalli, S.P., Mehto, S., (...), Chauhan, S., Chauhan, S. 2018 EMBO Journal\\n37(18),e98358\\n23'},\n",
              "  {'abstract': '© 2019 by the authors. Licensee MDPI, Basel, Switzerland. One of the unique characteristics of intrinsically disordered proteins (IPDs) is the existence of functional segments in intrinsically disordered regions (IDRs). A typical function of these segments is binding to partner molecules, such as proteins and DNAs. These segments play important roles in signaling pathways and transcriptional regulation. We conducted bioinformatics analysis to search these functional segments based on IDR predictions and database annotations. We found more than a thousand potential functional IDR segments in disease-related proteins. Large fractions of proteins related to cancers, congenital disorders, digestive system diseases, and reproductive system diseases have these functional IDRs. Some proteins in nervous system diseases have long functional segments in IDRs. The detailed analysis of some of these regions showed that the functional segments are located on experimentally verified IDRs. The proteins with functional IDR segments generally tend to come and go between the cytoplasm and the nucleus. Proteins involved in multiple diseases tend to have more protein-protein interactors, suggesting that hub proteins in the protein-protein interaction networks can have multiple impacts on human diseases.',\n",
              "   'title': '98\\nFunctional segments on intrinsically disordered regions in disease-related proteins\\nOpen Access\\nAnbo, H., Sato, M., Okoshi, A., Fukuchi, S. 2019 Biomolecules\\n9(3),88\\n6'},\n",
              "  {'abstract': 'Hepatitis C virus (HCV) is a causative agent of chronic hepatitis and hepatocellular carcinoma. The core protein of HCV packages the viral RNA genome to form a nucleocapsid. In addition to its function as a structural protein, core protein is involved in regulation of cellular transcription, virus-induced transformation, and pathogenesis. To gain insights into cellular functions of the core protein by identification of cellular proteins interacting with the core protein, we employed a proteomic approach. Hepatocytes soluble cytoplasmic proteins were applied to the core proteins immobilized on Ni-nitrilotriacetic resin and total bound cellular proteins were resolved by 2-DE. Analyses of interacting proteins by matrix-assisted laser desorption/ionization-time of flight mass spectrometry allowed identification of 14 cellular proteins binding to the core protein. These proteins include DEAD-box polypeptide 5, similar in function to a known protein identified previously by yeast two-hybrid screening and 13 newly identified cellular proteins. Interestingly, nine protein spots were identified as intermediate microfilament proteins, including cytokeratins (five spots for cytokeratin 8, two for cytokeratin 19, and one for cytokeratin 18) and vimentin. Cytokeratin 8 and vimentin, which were previously shown to be involved in the infection processes of other viruses, were further analyzed to confirm their in vivo interactions with the core protein by immunoblotting and immunofluorescence microscopy. We discuss the functional implications of the interactions of the core protein with newly identified cellular proteins in HCV infection and pathogenesis. © 2005 WILEY-VCH Verlag GmbH & Co. KGaA.',\n",
              "   'title': '99\\nProteomic profiling of cellular proteins interacting with the hepatitis C virus core protein Kang, S.-M., Shin, M.-J., Ki, J.-H., Oh, J.-W. 2005 Proteomics\\n5(8), pp. 2227-2237\\n40'},\n",
              "  {'abstract': 'The homeostatic regulation of essential elements such as copper requires many proteins whose activities are often mediated and tightly coordinated through protein-protein interactions. This regulation ensures that cells receive enough copper without intracellular concentrations reaching toxic levels. To date, only a small number of proteins implicated in copper homeostasis have been identified, and little is known of the protein-protein interactions required for this process. To identify other proteins important for copper homeostasis, while also elucidating the protein-protein interactions that are integral to the process, we have utilized a known copper protein, the copper ATPase ATP7A, as a bait in a yeast two-hybrid screen of a human cDNA library to search for interacting partners. One of the ATP7A-interacting proteins identified is a novel protein with a single PDZ domain. This protein was recently identified to interact with the plasma membrane calcium ATPase b-splice variants. We propose a change in name for this protein from PISP (plasma membrane calcium ATPase-interacting single-PDZ protein) to AIPP1 (ATPase-interacting PDZ protein) and suggest that it represents the protein that interacts with the class I PDZ binding motif identified at the ATP7A C terminus. The interaction in mammalian cells was confirmed and an additional splice variant of AIPP1 was identified. This study represents an essential step forward in identifying the proteins and elucidating the network of protein-protein interactions involved in maintaining copper homeostasis and validates the use of the yeast two-hybrid approach for this purpose. © 2005 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '100\\nA single PDZ domain protein interacts with the menkes copper ATPase, ATP7A: A new protein implicated in copper homeostasis\\nOpen Access\\nStephenson, S.E.M., Dubach, D., Lim, C.M., Mercer, J.F.B., La Fontaine, S. 2005 Journal of Biological Chemistry\\n280(39), pp. 33270-33279\\n25'},\n",
              "  {'abstract': '© 2015 by The American Society for Biochemistry and Molecular Biology, Inc. Several cytoplasmic proteins that are involved in G protein-coupled receptor signaling cascades are known to translocate to the plasma membrane upon receptor activation, such as beta-arrestin2. Based on this example and in order to identify new cytoplasmic proteins implicated in the ON-and-OFF cycle of G protein-coupled receptor, a live-imaging screen of fluorescently labeled cytoplasmic proteins was performed using translocation criteria. The screening of 193 fluorescently tagged human proteins identified eight proteins that responded to activation of the tachykinin NK2 receptor by a change in their intracellular localization. Previously we have presented the functional characterization of one of these proteins, REDD1, that translocates to the plasma membrane. Here we report the results of the entire screening. The process of cell activation was recorded on videos at different time points and all the videos can be visualized on a dedicated website. The proteins BAIAP3 and BIN1, partially translocated to the plasma membrane upon activation of NK2 receptors. Proteins ARHGAP12 and PKM2 translocated toward membrane blebs. Three proteins that associate with the cytoskeleton were of particular interest : PLEKHH2 rearranged from individual dots located near the cell-substrate adhesion surface into lines of dots. The speriolin-like protein, SPATC1L, redistributed to cell-cell junctions. The Chloride intracellular Channel protein, CLIC2, translocated from actin-enriched plasma membrane bundles to cell-cell junctions upon activation of NK2 receptors. CLIC2, and one of its close paralogs, CLIC4, were further shown to respond with the same translocation pattern to muscarinic M3 and lysophosphatidic LPA receptors. This screen allowed us to identify potential actors in signaling pathways downstream of G protein-coupled receptors and could be scaled-up for high-content screening.',\n",
              "   'title': '101\\nA fluorescent live imaging screening assay based on translocation criteria identifies novel cytoplasmic proteins implicated in G protein-coupled receptor signaling pathways\\nOpen Access\\nLecat, S., Matthes, H.W.D., Pepperkok, R., Simpson, J.C., Galzi, J.-L. 2015 Molecular and Cellular Proteomics\\n14(5), pp. 1385-1399\\n12'},\n",
              "  {'abstract': 'Proteome analysis of human umbilical endothelial cells was performed to identify proteins that are modified during vascular endothelial cell growth factor (VEGF)-induced transition from the quiescent into the proliferating-migrative phenotype. Subtractive analysis of two-dimensional gel patterns of human endothelial cells, before and after stimulation with VEGF 165, revealed differences in 85 protein spots. All proteins were identified by peptide sequencing and peptide mass fingerprinting using an electrospray spectrometer. The proteins identified were members of specific families including Ca2+-binding proteins, fatty-acid binding proteins, structural proteins, and chaperones. Remarkably, there was a massive activation of cellular machinery for both protein synthesis and protein degradation. Thus, among up-regulated proteins there were members of all groups of heat shock proteins (HSPs; HSP 27, HSP 60, HSP 70p5, HSP 70p8, HSP 90, and HSP 96) and some other proteins showing either chaperone activity or which participate in assembly of multimolecular structures (TCP-1, desmoplakins, junction plakoglobin, GRP 94, thioredoxin related protein, and peptidylprolyl isomerase). The increased expression of HSPs was confirmed at the mRNA level at different stages of treatment with VEGF. Similarly, components of the proteolytic machinery for the degradation of misfolded proteins (ER-60, cathepsin D, proteasome subunits, and protease inhibitor 6) were also up-regulated. On the other hand, changes in the expression of structural proteins (T-plastin, vimentin, α tubulin, actin, and myosin) could account, at least in part, for the different morphologies displayed by migrating endothelial cells. In summary, our data show that VEGF levels similar to those during physiological stresses induce a number of genes and multiple endogenous pathways seem to be engaged in restoring cellular homeostasis. To ensure cell survival, the molecular chaperones (the heat shock family of stress proteins) are highly up-regulated providing protein-folding machinery to repair or degrade misfolded proteins. © 2005 WILEY-VCH Verlag GmbH & Co. KGaA.',\n",
              "   'title': '102\\nHeat shock proteins and other components of cellular machinery for protein are up-regulated in vascular endothelial cell growth factor-activated human endothelial cells Pawlowska, Z., Baranska, P., Jerczynska, H., Koziolkiewicz, W., Cierniewski, C.S. 2005 Proteomics\\n5(5), pp. 1217-1227\\n36'},\n",
              "  {'abstract': 'The regulators of G-protein signaling (RGS) proteins were initially characterized as inhibitors of signal transduction cascades initiated by G-protein-coupled receptors (GPCR) because of their ability to increase the intrinsic GTPase activity of heterotrimeric G proteins. This GTPase accelerating protein (GAP) activity enhances G protein deactivation and promotes desensitization. However, in addition to this signature trait, emerging data have revealed an expanding network of proteins, lipids, and ions that interact with RGS proteins and confer additional regulatory functions. This review highlights recent advances in our understanding of the physiological functions of one subfamily of RGS proteins with a high degree of homology (B/R4) gleaned from recent studies of knockout mice or cells with reduced RGS expression. We also discuss some of the newly appreciated interactions of RGS proteins with cellular factors that suggest RGS control of several components of G-protein-mediated pathways, as well as a diverse array of non-GPCR-mediated biological responses.',\n",
              "   'title': '103\\nR4 RGS proteins: Regulation of G-protein signaling and beyond Bansal, G., Druey, K.M., Xie, Z. 2007 Pharmacology and Therapeutics\\n116(3), pp. 473-495\\n141'},\n",
              "  {'abstract': 'Hepatitis C virus (HCV) F protein is encoded by the +1 reading frame of the viral genome. It overlaps with the core protein coding sequence, and multiple mechanisms for its expression have been proposed. The full-length F protein that is synthesized by translational ribosomal frameshift at codons 9 to 11 of the core protein sequence is a labile protein. By using a combination of genetic, biochemical, and cell biological approaches, we demonstrate that this HCV F protein can bind to the proteasome subunit protein α3, which reduces the F-protein level in cells in a dose-dependent manner. Deletion-mapping analysis identified amino acids 40 to 60 of the F protein as the α3-binding domain. This α3-binding domain of the F protein together with its upstream sequence could significantly destabilize the green fluorescent protein, an otherwise stable protein. Further analyses using an F-protein mutant lacking lysine and a cell line that contained a temperature-sensitive E1 ubiquitin-activating enzyme indicated that the degradation of the F protein was ubiquitin independent. Based on these observations as well as the observation that the F protein could be degraded directly by the 20S proteasome in vitro, we propose that the full-length HCV F protein as well as the F protein initiating from codon 26 is degraded by an ubiquitin-independent pathway that is mediated by the proteasome subunit α3. The ability of the F protein to bind to α3 raises the possibility that the HCV F protein may regulate protein degradation in cells. Copyright © 2009, American Society for Microbiology. All Rights Reserved.',\n",
              "   'title': '104\\nUbiquitin-independent degradation of hepatitis C virus F protein\\nOpen Access\\nYuksek, K., Chen, W.-L., Chien, D., Ou, J.-H.J. 2009 Journal of Virology\\n83(2), pp. 612-621\\n41'},\n",
              "  {'abstract': 'The Arabidopsis thaliana genome encodes about 386 proteins with coiled-coil domains of at least 50 amino acids in length. In mammalian systems, many coiled-coil proteins are part of various cytoskeletal networks including intermediate filament protein, actin-binding proteins and MAP (microtubule-associated proteins). Immunological evidence suggests that some of these cytoskeletal proteins, such as lamins, keratins and tropomyosins, may be conserved in Arabidopsis. However, coiledcoil proteins are of low complexity, and thus, traditional sequence comparison algorithms, such as BLAST may not detect homologies. Here, we use the PROPSEARCH algorithm to detect putative coiled-coil cytoskeletal protein homologues in Arabidopsis. This approach reveals putative intermediate filament protein homologues of filensin, lamin and keratin; putative actin-binding homologues of ERM(ezrin/radixin/moesin), periplakin, utrophin, tropomyosin and paramyosin, and putative MAP homologues of restin/CLIP-170 (cytoplasmic linker protein-170). We suggest that the AtFPP (Arabiopsis thaliana filament-like plant protein) and AtMAP70 (Arabidopsis microtubule-associated protein 70) families of coiled-coil proteins may, in fact, be related to lamins and function as intermediate filament proteins. © The Author(s) Journal compilation © 2011 Portland Press Limited.',\n",
              "   'title': '105\\nPutative Arabidopsis homologues of metazoan coiled-coil cytoskeletal proteins Gardiner, J., Overall, R., Marc, J. 2011 Cell Biology International\\n35(8), pp. 767-774\\n17'},\n",
              "  {'abstract': 'Endocrine regulation frequently culminates in altered transcription of specific genes. The signal transduction pathways, which transmit the endocrine signal from cell surface to the transcription machinery, often involve posttranslational modifications of proteins. Although phosphorylation has been by far the most widely studied protein modification, recent studies have indicated important roles for other types of modification, including protein arginine methylation. Ten different protein arginine methyltransferase (PRMT) family members have been identified in mammalian cells, and numerous substrates are being identified for these PRMTs. Whereas major attention has been focused on the methylation of histones and its role in chromatin remodeling and transcriptional regulation, there are many nonhistone substrates methylated by PRMTs. This review primarily focuses on recent progress on the roles of the nonhistone protein methylation in transcription. Protein methylation of coactivators, transcription factors, and signal transducers, among other proteins, plays important roles in transcriptional regulation. Protein methylation may affect protein-protein interaction, protein-DNA or protein-RNA interaction, protein stability, subcellular localization, or enzymatic activity. Thus, protein arginine methylation is critical for regulation of transcription and potentially for various physiological/pathological processes. (Molecular Endocrinology 23: 425-433, 2009) Copyright © 2009 by The Endocrine Society.',\n",
              "   'title': '106\\nMinireview: Protein arginine methylation of nonhistone proteins in transcriptional regulation\\nOpen Access\\nLee, Y.-H., Stallcup, M.R. 2009 Molecular Endocrinology\\n23(4), pp. 425-433\\n148'},\n",
              "  {'abstract': 'Protein domain frequency and distribution among kingdoms was statistically analyzed using the SCOP structural database. It appeared that among chosen protein domains with the best resolution, eukaryotic proteins more often belong to α-helical and β-structural proteins, while proteins of bacterial origin belong to α/β structural class. Statistical analysis of folding rates of 73 proteins with known experimental data revealed that bacterial proteins with simple kinetics (23 proteins) exhibit a higher folding rate compared to eukaryotic proteins with simple folding kinetics (27 proteins). Analysis of protein domain amino acid composition showed that the frequency of amino acid residues in proteins of eukaryotic and bacterial origin is different for proteins with simple and complex folding kinetics. © 2011 Pleiades Publishing, Ltd.',\n",
              "   'title': '107\\nBacterial proteins fold faster than eukaryotic proteins with simple folding kinetics Galzitskaya, O.V., Bogatyreva, N.S., Glyakina, A.V. 2011 Biochemistry (Moscow)\\n76(2), pp. 225-235\\n3'},\n",
              "  {'abstract': 'Microtubules play an essential role in the growth and development of plants and are known to be involved in regulating many cellular processes ranging from translation to signaling. In this article, we describe the proteomic characterization of Arabidopsis tubulin-binding proteins that were purified using tubulin affinity chromatography. Microtubule co-sedimentation assays indicated that most, if not all, of the proteins in the tubulin-binding protein fraction possessed microtubule-binding activity. Two-dimensional gel electrophoresis of the tubulin-binding protein fraction was performed, and 86 protein spots were excised and analyzed for protein identification. A total of 122 proteins were identified with high confidence using LC-MS/MS. These proteins were grouped into six categories based on their predicted functions: microtubule-associated proteins, translation factors, RNA-binding proteins, signaling proteins, metabolic enzymes, and proteins with other functions. Almost one-half of the proteins identified in this fraction were related to proteins that have previously been reported to interact with microtubules. This study represents the first large-scale proteomic identification of eukaryotic cytoskeleton-binding proteins, and provides insight on subcellular trafficking, metabolic channeling, and signaling in plant cells. © 2004 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '108\\nLarge-scale identification of tubulin-binding proteins provides insight on subcellular trafficking, metabolic channeling, and signaling in plant cells\\nOpen Access\\nChuong, S.D.X., Good, A.G., Taylor, G.J., (...), Moorhead, G.B.G., Muench, D.G. 2004 Molecular and Cellular Proteomics\\n3(10), pp. 970-983\\n101'},\n",
              "  {'abstract': 'The mammalian epididymis secretes numerous proteins important for sperm maturation. Among these are proteins D and E, which belong to the CRISP family (cysteine-rich secretory proteins) and are the product of the Crisp-1 gene. These proteins have been the focus of a number of studies and have been implicated in sperm/egg fusion. Protein D and protein E have been purified to apparent homogeneity in several laboratories. Polyclonal antibodies raised against each protein typically cross-reacted with both proteins, suggesting that they were immunologically similar, if not identical. Our laboratory has previously reported the generation of a monoclonal antibody (mAb 4E9) that recognizes only protein E. Using mAb 4E9, the localization of protein E was shown to be domain specific on the sperm surface and there is processing of the protein in the fluid, with only the lowest molecular weight form associating with sperm. Subsequent purification and amino acid sequencing of protein D confirmed that proteins D and E are nearly identical and differ only by presence of the 4E9 epitope on protein E. Here we report the generation of antibodies to regions of amino acid sequence identity in proteins D and E. Using these antibodies, we demonstrate that protein D associates with the sperm head and that a portion of this protein may be proteolytically processed. In addition, we demonstrate that the proteolytic processing of protein E occurs in the carboxy terminal region of this protein. The data also suggest that a portion of protein D may also undergo processing, similar to that of protein E. Finally, we use these antibodies to demonstrate that proteins D and E are differentially expressed by the epididymal epithelium. Taken together, these data suggest that proteins D and E may have individual roles in sperm function.',\n",
              "   'title': '109\\nA comparative analysis of expression and processing of the rat epididymal fluid and sperm-bound forms of proteins D and E\\nOpen Access\\nRoberts, K.P., Ensrud, K.M., Hamilton, D.W. 2002 Biology of Reproduction\\n67(2), pp. 525-533\\n39'},\n",
              "  {'abstract': 'Background: The increasing number of protein sequences and 3D structure obtained from genomic initiatives is leading many of us to focus on proteomics, and to dedicate our experimental and computational efforts on the creation and analysis of information derived from 3D structure. In particular, the high-throughput generation of protein-protein interaction data from a few organisms makes such an approach very important towards understanding the molecular recognition that make-up the entire protein-protein interaction network. Since the generation of sequences, and experimental protein-protein interactions increases faster than the 3D structure determination of protein complexes, there is tremendous interest in developing in silico methods that generate such structure for prediction and classification purposes. In this study we focused on classifying protein family members based on their protein-protein interaction distinctiveness. Structure-based classification of protein-protein interfaces has been described initially by Ponstingl et al. and more recently by Valdar et al. and Mintseris et al., from complex structures that have been solved experimentally. However, little has been done on protein classification based on the prediction of protein-protein complexes obtained from homology modeling and docking simulation. Results: We have developed an in silico classification system entitled HODOCO (Homology modeling, Docking and Classification Oracle), in which protein Residue Potential Interaction Profiles (RPIPS) are used to summarize protein-protein interaction characteristics. This system applied to a dataset of 64 proteins of the death domain superfamily was used to classify each member into its proper subfamily. Two classification methods were attempted, heuristic and support vector machine learning. Both methods were tested with a 5-fold cross-validation. The heuristic approach yielded a 61% average accuracy, while the machine learning approach yielded an 89% average accuracy. Conclusion: We have confirmed the reliability and potential value of classifying proteins via their predicted interactions. Our results are in the same range of accuracy as other studies that classify protein-protein interactions from 3D complex structure obtained experimentally. While our classification scheme does not take directly into account sequence information our results are in agreement with functional and sequence based classification of death domain family members. © 2004 Lett et al; licensee BioMed Central Ltd.',\n",
              "   'title': '110\\nInteraction profile-based protein classification of death domain\\nOpen Access\\nLett, D., Hsing, M., Pio, F. 2004 BMC Bioinformatics\\n5,75\\n2'},\n",
              "  {'abstract': 'Protein-protein and protein-nucleic acid interactions are involved in many regulatory cellular pathways, playing a key role in cell growth and proliferation, as well as in the progression and development of various diseases such as infectious diseases. Especially in the anti-AIDS research, protein-protein and protein-nucleic acid complexes are being considered as promising targets for pharmaceutical interventions aimed at overcoming the drug resistance observed for most of the classic enzyme inhibitors. Consequently, more and more protein-protein and protein-nucleic acid interaction inhibitors have being identified and developed as candidate agents for antiretroviral therapy. Here, we review the state of the art in the discovery and development of protein-protein and protein-nucleic acid interaction inhibitors that block the main steps of the HIV-1 replication cycle, giving a medicinal chemistry-oriented view of strategies for inhibiting these regulatory interactions that are involved in the entry process, in the dimerization of reverse transcriptase and protease enzymes, and in the activity of the nucleocapsid protein by means of small molecular potential therapeutic agents. © 2011 Bentham Science Publishers.',\n",
              "   'title': '111\\nTargeting protein-protein and protein-nucleic acid interactions for Anti-HIV therapy Mori, M., Manetti, F., Botta, M. 2011 Current Pharmaceutical Design\\n17(33), pp. 3713-3728\\n19'},\n",
              "  {'abstract': 'A number of structurally divergent proteins with J domains, called J proteins, interact with and activate the ATPase of Hsp70s, thereby harnessing the ATPase activity for conformational work on target proteins. The precise role of most mammalian J proteins remains undefined. In this paper, we demonstrate that transient expression of the J protein, Rdj2, in HEK 293 cells increased cellular cyclic adenosine monophosphate (cAMP) levels in the presence of the β-adrenergic agonist isoproterenol. In CNS-derived catechol-aminergic neuronal cell line (CAD) neuroblastoma cells, expression of Rdj2 increased isoproterenol-stimulated phosphorylation of cAMP response element binding protein (CREB). Moreover, we have characterized the binding properties of Rdj2 and observed a direct interaction between Rdj2 and receptor-coupled trimeric GTP-binding proteins (G proteins). We further show that the composition of the Rdj2-chaperone complex and the cysteine string protein (CSPα)-chaperone complex, another J protein, is distinct. Our data demonstrate that Rdj2 modulates G protein signaling and further suggest that chaperoning G proteins is an emerging theme of the J protein network. © Cell Stress Society International 2008.',\n",
              "   'title': '112\\nRDJ2 (DNAJA2) chaperones neural G protein signaling pathways Rosales-Hernandez, A., Beck, K.E., Zhao, X., Braun, A.P., Braun, J.E.A. 2009 Cell Stress and Chaperones\\n14(1), pp. 71-82\\n15'},\n",
              "  {'abstract': 'Regulator of G protein signalling (RGS) proteins are GTPase-activating proteins for heterotrimeric G protein α subunits, and are therefore physiologically and pathophysiologically important negative regulators of G-protein-coupled receptor signalling in the cardiovascular system. Owing to the functional redundancy of many of the 20 RGS, and more than 20 RGS-like, proteins even within a single cell, animal models shedding light on the functions of individual RGS proteins are often missing. Nevertheless, RGS2 is a member of this protein family, for which specific functions in the vasculature and the heart are now emerging. Recent data show that the 519-amino acid RGS3, the only RGS protein with an additional G protein βγ dimer binding domain, largely alters the signalling of Gi proteins to the monomeric GTPases Rac1 and RhoA in cardiomyocytes. In addition, an alternative approach using transgenic animals expressing RGS-resistant G protein α subunits now highlights the contributions of RGS proteins to distinct signalling pathways in the heart. © 2007 Elsevier Ltd. All rights reserved.',\n",
              "   'title': '113\\nRegulators of G protein signalling: a spotlight on emerging functions in the cardiovascular system Wieland, T., Lutz, S., Chidiac, P. 2007 Current Opinion in Pharmacology\\n7(2), pp. 201-207\\n53'},\n",
              "  {'abstract': '© 2014 Elsevier Inc. Protein turnover reflects the balance between synthesis and degradation of proteins, and it is a crucial process for the maintenance of the cellular protein pool. The folding of proteins, refolding of misfolded proteins, and also degradation of misfolded and damaged proteins are involved in the protein quality control (PQC) system. Correct protein folding and degradation are controlled by many different factors, one of the most important of which is the heat shock protein family. Heat shock proteins (HSPs) are in the class of molecular chaperones, which may prevent the inappropriate interaction of proteins and induce correct folding. On the other hand, these proteins play significant roles in the degradation pathways, including endoplasmic reticulum-associated degradation (ERAD), the ubiquitin-proteasome system, and autophagy. This review focuses on the emerging role of HSPs in the regulation of protein turnover; the effects of HSPs on the degradation machineries ERAD, autophagy, and proteasome; as well as the role of posttranslational modifications in the PQC system.',\n",
              "   'title': '114\\nRegulation of protein turnover by heat shock proteins Bozaykut, P., Ozer, N.K., Karademir, B. 2014 Free Radical Biology and Medicine\\n77, pp. 195-209\\n45'},\n",
              "  {'abstract': 'Leucine-rich repeats (LRRs) are 20-29-residue sequence motifs present in a number of proteins with diverse functions. The primary function of these motifs appears to be to provide a versatile structural framework for the formation of protein-protein interactions. The past two years have seen an explosion of new structural information on proteins with LRRs. The new structures represent different LRR subfamilies and proteins with diverse functions, including GTPase-activating protein rna1p from the ribonuclease-inhibitor-like subfamily; spliceosomal protein U2A′, Rab geranylgeranyltransferase, internalin B, dynein light chain 1 and nuclear export protein TAP from the SDS22-like subfamily; Skp2 from the cysteine-containing subfamily; and YopM from the bacterial subfamily. The new structural information has increased our understanding of the structural determinants of LRR proteins and our ability to model such proteins with unknown structures, and has shed new light on how these proteins participate in protein-protein interactions.',\n",
              "   'title': '115\\nThe leucine-rich repeat as a protein recognition motif Kobe, B., Kajava, A.V. 2001 Current Opinion in Structural Biology\\n11(6), pp. 725-732\\n1150'},\n",
              "  {'abstract': 'Sendal virus matrix protein (M protein) is critically important for virus assembly and budding and is presumed to interact with viral glycoproteins on the outer side and viral nucleocapsid on the inner side. However, since M protein alone binds to lipid membranes, it has been difficult to demonstrate the specific interaction of M protein with HN or F protein, the Sendai viral glycoproteins. Using Triton X-100 (TX-100) detergent treatment of membrane fractions and flotation in sucrose gradients, we report that the membrane-bound M protein expressed alone or coexpressed with heterologous glycoprotein (influenza virus HA) was totally TX-100 soluble but the membrane-bound M protein coexpressed with HN or F protein either individually or together was predominantly detergent-resistant and floated to the top of the density gradient. Furthermore, both the cytoplasmic tail and the transmembrane domain of F protein facilitated binding of M protein to detergent-resistant membranes. Analysis of the membrane association of M protein in the early and late phases of the Sendai virus infectious cycle revealed that the interaction of M protein with mature glycoproteins that associated with the detergent-resistant lipid rafts was responsible for the detergent resistance of the membrane-bound M protein. Immunofluorescence analysis by confocal microscopy also demonstrated that in Sendai virus-infected cells, a fraction of M protein colocalized with F and HN proteins and that some M protein also became associated with the F and HN proteins while they were in transit to the plasma membrane via the exocytic pathway. These studies indicate that F and HN interact with M protein in the absence of any other viral proteins and that F associates with M protein via its cytoplasmic tail and transmembrane domain. (C) 2000 Academic Press.',\n",
              "   'title': '116\\nAssembly of Sendal virus: M protein interacts with F and HN proteins and with the cytoplasmic tail and transmembrane domain of F protein\\nOpen Access\\nAli, A., Nayak, D.P. 2000 Virology\\n276(2), pp. 289-303\\n97'},\n",
              "  {'abstract': 'Purified P400 protein was phosphorylated by both purified Ca2+/calmodulin‐dependent protein kinase II (CaM kinase II) and the catalytic subunit of cyclic AMP‐dependent protein kinase (A‐kinase). Because P400 protein was suggested to function as an integral membrane protein, we investigated the phosphorylation of P400 protein using crude mitochondrial and microsomal fractions (P2/P3 fraction). Incubation of the P2/P3 fraction from mouse cerebellum with cyclic AMP or the catalytic subunit of A‐kinase stimulated the phosphorylation of P400 protein. The phosphorylation of P400 protein was not observed in the P2/P3 fraction from mouse forebrain. Cyclic AMP and A‐kinase enhanced the phosphorylation of several proteins, including P400 protein, suggesting that P400 protein is one of the best substrates for A‐kinase in the P2/ P3 fraction. Although endogenous and exogenous CaM kinase II stimulated the phosphorylation of some proteins in the P2/P3 fraction, the phosphorylation of P400 protein was weak. Immunoprecipitation with the monoclonal antibody to P400 protein confirmed that the P400 protein itself was definitely phosphorylated by the catalytic subunit of A‐kinase and CaM kinase II. A‐kinase phosphorylated only the seryl residue in P400 protein. Immunoblot analysis of the cells in primary culture of mouse cerebellum confirmed the expression of P400 protein, which migrated at the same position on sodium do‐decyl sulfate‐polyacrylamide gel electrophoresis as that in the P2/P3 fraction. Incubation of the cultured cerebellar cells with [32P]orthophosphate resulted in the labeling of P400 protein. These results suggest that P400 protein is one of the best substrates for A‐kinase and CaM kinase II in cerebellum and that the function of P400 protein is regulated by cyclic AMP‐and Ca2+/calmodulin‐dependent phosphorylation. Copyright © 1989, Wiley Blackwell. All rights reserved',\n",
              "   'title': '117\\nPhosphorylation of P400 Protein by Cyclic AMP‐Dependent Protein Kinase and Ca2+/Calmodulin‐Dependent Protein Kinase II Yamamoto, H., Maeda, N., Niinobe, M., Miyamoto, E., Mikoshiba, K. 1989 Journal of Neurochemistry\\n53(3), pp. 917-923\\n43'},\n",
              "  {'abstract': 'In response to advances in proteomics research and the use of proteins in medical and biotechnological applications, recombinant protein production and the design of specific protein characteristics and functions has become a widely used technology. In this context, protein fusion tags have been developed as indispensable tools for protein expression, purification, and the design of functionalized surfaces or artificially bifunctional proteins. Here we summarize how positively or negatively charged polyionic fusion peptides with or without an additional cysteine can be used as protein tags for protein expression and purification, for matrix-assisted refolding of aggregated protein, and for coupling of proteins either to technologically relevant matrices or to other proteins. In this context we used cysteine-containing polyionic fusion peptides for the design of immunotoxins. In general, polyionic fusion tags can be considered as a multifunctional module in protein technology.',\n",
              "   'title': '118\\nPolyionic and cysteine-containing fusion peptides as versatile protein tags Lilie, H., Richter, S., Bergelt, S., Frost, S., Gehle, F. 2013 Biological Chemistry\\n394(8), pp. 995-1004\\n2'},\n",
              "  {'abstract': 'It has previously been reported that protein complexity (i.e. number of subunits in a protein complex) is negatively correlated to gene duplicability in yeast as well as in humans. However, unlike in yeast, protein connectivity in a protein-protein interaction network has a positive correlation with gene duplicability in human genes. In the present study, we have analyzed 1732 human and 1269 yeast proteins that are present both in a protein-protein interaction network as well as in a protein complex network. In the human case, we observed that both protein connectivity and protein complexity complement each other in a mutually exclusive manner over gene duplicability in a positive direction. Analysis of human haploinsufficient proteins and large protein complexes (complex size >10) shows that when protein connectivity does not have any direct association with gene duplicability, there exists a positive correlation between gene duplicability and protein complexity. The same trend, however, is not found in case of yeast, where both protein connectivity and protein complexity independently guide gene duplicability in the negative direction. We conclude that the higher rate of duplication of human genes may be attributed to organismal complexity either by increasing connectivity in the protein-protein interaction network or by increasing protein complexity. © 2010 The Author.',\n",
              "   'title': '119\\nProtein connectivity and protein complexity promotes human gene duplicability in a mutually exclusive manner\\nOpen Access\\nBhattacharya, T., Ghosh, T.C. 2010 DNA Research\\n17(5), pp. 261-270\\n2'},\n",
              "  {'abstract': '© 2015 Elsevier Ltd. All rights reserved. Prolyl isomerizations are intrinsically slow processes. They determine the rates of many protein folding reactions and control regulatory events in folded proteins. Prolyl isomerases are able to catalyze these isomerizations, and thus, they have the potential to assist protein folding and to modulate protein function. Here, we provide examples for how prolyl isomerizations limit protein folding and are accelerated by prolyl isomerases and how native-state prolyl isomerizations regulate protein functions. The roles of prolines in protein folding and protein function are closely interrelated because both of them depend on the coupling between cis/trans isomerization and conformational changes that can involve extended regions of a protein.',\n",
              "   'title': '120\\nProlyl isomerization and its catalysis in protein folding and protein function Schmidpeter, P.A.M., Schmid, F.X. 2015 Journal of Molecular Biology\\n427(7),64685, pp. 1609-1631\\n35'},\n",
              "  {'abstract': 'It has been claimed that proteins with more interacting partners (hubs) are structurally more disordered and have a slow evolutionary rate. Here, in this paper we analyzed the evolutionary rate and structural disorderness of human hub and non-hub proteins present/absent in protein complexes. We observed that both non-hub and hub proteins present in protein complexes, are characterized by high structural disorderness. There exists no significant difference in average evolutionary rate of complex-forming hub and non-hub proteins while we have found a significant difference in the average evolutionary rate between hub and non-hub proteins which are not present in protein complexes. We concluded that higher disorderness in complex forming non-hub proteins facilitates higher number of interactions with a large number of protein subunits. High interaction among protein subunits of complex forming non-hub proteins imposes a selective constraint on their evolutionary rate. © 2008 Elsevier B.V. All rights reserved.',\n",
              "   'title': '121\\nEvolutionary constraints on hub and non-hub proteins in human protein interaction network: Insight from protein connectivity and intrinsic disorder Manna, B., Bhattacharya, T., Kahali, B., Ghosh, T.C. 2009 Gene\\n434(1-2), pp. 50-55\\n25'},\n",
              "  {'abstract': 'Polyhydroxyalkanoates (PHAs) granule associated protein PhaP has a strong affinity to PHA and other hydrophobic polymers. Human lipopolysaccharide binding protein (hLBP) is a natural endotoxin receptor in plasma. In this study, genes encoding hLBP fused with PhaP were expressed in Pichia pastoris GS115 for production of the fusion protein. The purified rhLBP-PhaP fusion protein was immobilized on particles of polyhydroxybutyrate (PHB), which is a member of microbial polyhydroxyalkanoates (PHA). The rhLBPPhaP- coated PHB particles were added to endotoxin containing water and protein solutions to study their endotoxin removal and protein recovery efficiencies. The influences of ionic strengths and pH on endotoxin removal and protein recovery in different protein solutions were also studied using acidic proteins including bovine serum albumin (BSA), ovalbumin, and basic protein R-chymotrypsinogen as model proteins. The results showed that rhLBP-PhaP particles could remove endotoxin with an efficiency of over 90%. All endotoxin removal and protein recovery efficiencies were only slightly affected by ionic strengths but were drastically affected by pH changes. Our results demonstrated that rhLBP-PhaP particles with their high efficiency, ease of preparation, and nontoxicity will be a suitable system for endotoxin removal in the protein purification industry.© 2011 American Chemical Society.',\n",
              "   'title': '122\\nEndotoxin removing method based on lipopolysaccharide binding protein and polyhydroxyalkanoate binding protein PhaP Li, J., Shang, G., You, M., (...), Wu, H., Chen, G.-Q. 2011 Biomacromolecules\\n12(3), pp. 602-608\\n23'},\n",
              "  {'abstract': '© 2015 Elsevier Ltd. Viral capsids exhibit elaborate and symmetrical architectures of defined sizes and remarkable mechanical properties not seen with cellular macromolecular complexes. Given the uniqueness of the higher-order organization of viral capsid proteins in the virosphere, we explored the question of whether the patterns of protein-protein interactions within viral capsids are distinct from those in generic protein complexes. Our comparative analysis involving a non-redundant set of 551 inter-subunit interfaces in viral capsids from VIPERdb and 20,014 protein-protein interfaces in non-capsid protein complexes from the Protein Data Bank found 418 generic protein-protein interfaces that share similar physicochemical patterns with some protein-protein interfaces in the capsid set, using the program PCalign we developed for comparing protein-protein interfaces. This overlap in the structural space of protein-protein interfaces is significantly small, with a p-value < 0.0001, based on a permutation test on the total set of protein-protein interfaces. Furthermore, the generic protein-protein interfaces that bear similarity in their spatial and chemical arrangement with capsid ones are mostly small in size with fewer than 20 interfacial residues, which results from the relatively limited choices of natural design for small interfaces rather than having significant biological implications in terms of functional relationships. We conclude based on this study that protein-protein interfaces in viral capsids are non-representative of patterns in the smaller, more compact cellular protein complexes. Our finding highlights the design principle of building large biological containers from repeated, self-assembling units and provides insights into specific targets for antiviral drug design for improved efficacy.',\n",
              "   'title': '123\\nProtein-Protein Interfaces in Viral Capsids Are Structurally Unique\\nOpen Access\\nCheng, S., Brooks, C.L. 2015 Journal of Molecular Biology\\n427(22), pp. 3613-3624\\n3'},\n",
              "  {'abstract': '© 2018 Elsevier Inc. Both acute proteotoxic stresses that unfold proteins and expression of disease-causing mutant proteins that expose aggregation-prone regions can promote protein aggregation. Protein aggregates can interfere with cellular processes and deplete factors crucial for protein homeostasis. To cope with these challenges, cells are equipped with diverse folding and degradation activities to rescue or eliminate aggregated proteins. Here, we review the different chaperone disaggregation machines and their mechanisms of action. In all these machines, the coating of protein aggregates by Hsp70 chaperones represents the conserved, initializing step. In bacteria, fungi, and plants, Hsp70 recruits and activates Hsp100 disaggregases to extract aggregated proteins. In the cytosol of metazoa, Hsp70 is empowered by a specific cast of J-protein and Hsp110 co-chaperones allowing for standalone disaggregation activity. Both types of disaggregation machines are supported by small Hsps that sequester misfolded proteins. In this issue of Molecular Cell, Mogk et al. provide a review on the mechanism of action of the different protein disaggregation machines in metazoan and non-metazoan and their potential importance for resistance to acute and chronic proteotoxic stress.',\n",
              "   'title': '124\\nCellular Handling of Protein Aggregates by Disaggregation Machines\\nOpen Access\\nMogk, A., Bukau, B., Kampinga, H.H. 2018 Molecular Cell\\n69(2), pp. 214-226\\n84'},\n",
              "  {'abstract': \"The ubiquitin proteasome system (UPS) plays a fundamental role in maintaining the correct balance of protein levels inside all living cells. Degradation of proteins by this pathway is essential for most cellular processes including cell signalling, DNA repair, apoptosis and gene transcription. Any disruption to the system is likely to have severe consequences which may lead to disorders including neurodegeneration and cancer. Ubiquitin protein ligases are a group of UPS proteins of particular importance because these proteins determine targeting specificity via recognition of a 'target' protein and its' subsequent 'tagging' with ubiquitin. The 26S proteasome recognises these mutli-ubiquitylated proteins, allowing the correct protein to be degraded at the correct time and place within each cell. Several types of ubiquitin protein ligase have now been identified, however, the largest group by far are those proteins containing a 'RING' motif. In this review, examples will be given whereby abnormal protein ubiquitylation due to absence or inefficiency of a RING protein ligase is proposed to be a key regulator of the disease process. Ways in which we may be able to reverse these effects or manipulate these proteins to restore function will be discussed. © 2009 Bentham Science Publishers Ltd.\",\n",
              "   'title': '125\\nRing finger ubiquitin protein ligases and their implication to the pathogenesis of human diseases Ardley, H.C. 2009 Current Pharmaceutical Design\\n15(31), pp. 3697-3715\\n12'},\n",
              "  {'abstract': 'Arginine is widely used in such applications as protein refolding, solubilization of proteins and small molecules, protein and small molecule formulation, column chromatography and viral inactivation as summarized in this review. What makes arginine effective in these applications is largely based on its ability to suppress protein-protein interactions and protein-surface interactions. The mechanism of these widespread effects of arginine on proteins can be explained at least in part from its unique interactions with the protein surface. Here we describe the modes of the interactions of arginine with model compounds and proteins and also water molecules, and then attempt to explain the mechanism of its effect on proteins by comparing with the interactions that occur between protein and protein denaturants or stabilizers. © 2014 Bentham Science Publishers.',\n",
              "   'title': '126\\nMulti-faceted arginine: Mechanism of the effects of arginine on protein Arakawa, T., Kita, Y. 2014 Current Protein and Peptide Science\\n15(6), pp. 608-620\\n23'},\n",
              "  {'abstract': '© 2014 Elsevier Ltd. All rights reserved. Rice bran protein (RBP) was prepared by alkali extraction and isoelectric precipitation from defatted rice bran. The protein quality of RPB was evaluated and compared to two vegetable proteins [soy protein (ISP) and rice endosperm protein (REP)] and two animal proteins [whey protein (WPI) and casein]. RPB contained 74.93% of protein and its pepsin digestibility and KOH solubility were 89.8% and 91.5%, respectively. In Sprague-Dawley rats, RBP showed protein efficiency ratio, net protein ratio, net protein utilisation, and biological value of 2.39, 3.77, 70.7, and 72.6, which were comparable to the qualities of animal proteins. The true digestibility of RBP (94.8%) was significantly higher than that of REP (90.8%), ISP (91.7%) and WPI (92.8%) and the same as that of casein. Protein digestibility corrected amino acid score (PDCAAS) of RBP was 0.90. These results suggest that rice bran protein appears to be a promising protein source with good biological values and digestibility.',\n",
              "   'title': '127\\nNutritional quality of rice bran protein in comparison to animal and vegetable protein Han, S.-W., Chee, K.-M., Cho, S.-J. 2015 Food Chemistry\\n172, pp. 766-769\\n76'},\n",
              "  {'abstract': 'Heterotrimeric G proteins are key transducers for signal transfer from outside of the cell. In addition to their regulation by the superfamily of G protein-coupled receptors, many if not all of the subtypes of heterotrimeric G proteins are also regulated by additional accessory proteins that influence guanine nucleotide binding and/or hydrolysis or subunit interactions. Activators of G protein signaling (AGS1-3) refer to a functionally defined group of proteins that activate G protein-signaling systems in the absence of a classical G protein-coupled receptor. AGS and related proteins provide unexpected insights into the regulation of the G protein activation/deactivation cycle and the functional roles of G proteins. These proteins likely play important roles in the generation of signaling complexes, the positioning of signaling proteins within the cell, and in biological roles of G proteins unrelated to a cell surface receptor. As such, these proteins and the concepts advanced with their discovery provide unexpected avenues for therapeutics and understanding disease mechanisms.',\n",
              "   'title': '128\\nAccessory proteins for G protein-signaling systems: Activators of G protein signaling and other nonreceptor proteins influencing the activation state of G proteins Blumer, J.B., Lanier, S.M. 2003 Receptors and Channels\\n9(3), pp. 195-204\\n32'},\n",
              "  {'abstract': '© 2018, The Author(s). The postprandial rise in essential amino acid (EAA) concentrations modulates the increase in muscle protein synthesis rates after protein ingestion. The EAA content and AA composition of the dietary protein source contribute to the differential muscle protein synthetic response to the ingestion of different proteins. Lower EAA contents and specific lack of sufficient leucine, lysine, and/or methionine may be responsible for the lower anabolic capacity of plant-based compared with animal-based proteins. We compared EAA contents and AA composition of a large selection of plant-based protein sources with animal-based proteins and human skeletal muscle protein. AA composition of oat, lupin, wheat, hemp, microalgae, soy, brown rice, pea, corn, potato, milk, whey, caseinate, casein, egg, and human skeletal muscle protein were assessed using UPLC–MS/MS. EAA contents of plant-based protein isolates such as oat (21%), lupin (21%), and wheat (22%) were lower than animal-based proteins (whey 43%, milk 39%, casein 34%, and egg 32%) and muscle protein (38%). AA profiles largely differed among plant-based proteins with leucine contents ranging from 5.1% for hemp to 13.5% for corn protein, compared to 9.0% for milk, 7.0% for egg, and 7.6% for muscle protein. Methionine and lysine were typically lower in plant-based proteins (1.0 ± 0.3 and 3.6 ± 0.6%) compared with animal-based proteins (2.5 ± 0.1 and 7.0 ± 0.6%) and muscle protein (2.0 and 7.8%, respectively). In conclusion, there are large differences in EAA contents and AA composition between various plant-based protein isolates. Combinations of various plant-based protein isolates or blends of animal and plant-based proteins can provide protein characteristics that closely reflect the typical characteristics of animal-based proteins.',\n",
              "   'title': '129\\nProtein content and amino acid composition of commercially available plant-based protein isolates\\nOpen Access\\nGorissen, S.H.M., Crombag, J.J.R., Senden, J.M.G., (...), Verdijk, L.B., van Loon, L.J.C. 2018 Amino Acids\\n50(12), pp. 1685-1695\\n67'},\n",
              "  {'abstract': \"Selective ubiquitin-dependent autophagy plays a pivotal role in the elimination of protein aggregates, assemblies, or organelles and counteracts the cytotoxicity of proteins linked to neurodegenerative diseases. Following substrate ubiquitylation, the cargo is delivered to autophagosomes involving adaptors like human p62 that bind ubiquitin and the autophagosomal ubiquitin-like protein Atg8/LC3; however, whether similar pathways exist in lower eukaryotes remained unclear. Here, we identify by a screen in yeast a new class of ubiquitin-Atg8 adaptors termed CUET proteins, comprising the ubiquitin-binding CUE-domain protein Cue5 from yeast and its human homolog Tollip. Cue5 collaborates with Rsp5 ubiquitin ligase, and the corresponding yeast mutants accumulate aggregation-prone proteins and are vulnerable to polyQ protein expression. Similarly, Tollip depletion causes cytotoxicity toward polyQ proteins, whereas Tollip overexpression clears human cells from Huntington's disease-linked polyQ proteins by autophagy. We thus propose that CUET proteins play a critical and ancient role in autophagic clearance of cytotoxic protein aggregates. © 2014 Elsevier Inc.\",\n",
              "   'title': '130\\nAutophagic clearance of PolyQ proteins mediated by ubiquitin-Atg8 adaptors of the conserved CUET protein family\\nOpen Access\\nLu, K., Psakhye, I., Jentsch, S. 2014 Cell\\n158(3), pp. 549-563\\n171'},\n",
              "  {'abstract': 'Cell-free protein synthesis (CFPS) is a valuable method for the fast expression of difficult-to-express proteins as well as posttranslationally modified proteins. Since cell-free systems circumvent possible cytotoxic effects caused by protein overexpression in living cells, they significantly enlarge the scale and variety of proteins that can be characterized. We demonstrate the high potential of eukaryotic CFPS to express various types of membrane proteins covering a broad range of structurally and functionally diverse proteins. Our eukaryotic cell-free translation systems are capable to provide high molecular weight membrane proteins, fluorescent-labeled membrane proteins, as well as posttranslationally modified proteins for further downstream analysis. © 2012 WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim.',\n",
              "   'title': '131\\nSynthesis of membrane proteins in eukaryotic cell-free systems\\nOpen Access\\nSachse, R., Wüstenhagen, D., Šamalíková, M., (...), Bier, F.F., Kubick, S. 2013 Engineering in Life Sciences\\n13(1), pp. 39-48\\n45'},\n",
              "  {'abstract': 'A novel method to produce protein polymer conjugates for protein-acrylamide hydrogel formation is described. Alkenes are incorporated onto the N-terminus of expressed proteins to produce polymerizable protein monomers that can be utilized in protein-acrylamide copolymerization. A 4-vinylbenzoic acid thioester was synthesized and attached to the N-termini of two protein models, the immunoglobulin-binding protein Protein G and the bacterial enzyme xanthine-guanine phosphoribosyltransferase (GPRT), utilizing native chemical ligation. N-terminal cysteine containing proteins utilized in native chemical ligation reactions were generated from His-tagged fusion proteins using tobacco etch virus NIa (TEV) protease cleavage. The 4-vinylbenzyl functionalized proteins were good substrates for immobilizing proteins into polyacrylamide hydrogels via free radical induced protein-acrylamide copolymerization. The protein copolymerization procedures developed in this report are mild enough to allow proteins to retain measurable biological activity as demonstrated by the retention of immunoglobulin binding ability by immobilized Protein G and enzymatic activity of immobilized GPRT. © 2009 American Chemical Society.',\n",
              "   'title': '132\\nSynthesis of polymerizable protein monomers for protein-acrylamide hydrogel formation Xiao, J., Tolbert, T.J. 2009 Biomacromolecules\\n10(7), pp. 1939-1946\\n11'},\n",
              "  {'abstract': 'To understand the structure-function relationship of muscle-regulatory- protein isoforms, mutations, and posttranslational modifications, it is necessary to probe functional effects at the level of the protein-protein interaction. Traditional methodologies assessing such protein-protein interactions are laborious and require significant amounts of purified protein, while many current methodologies require costly and specialized equipment or modification of the proteins, which may affect their interaction. To address these issues, we developed a novel method of microplate-based solid-phase protein-binding assay over the recent years. This method assesses specific protein-protein interactions at physiological conditions, utilizes relatively small amounts of protein, is free of protein modification, and does not require specialized instrumentation. Here we present detailed methodology for the solid-phase protein-binding assay with examples that we have successfully applied to quantify interactions of myofilament-regulatory proteins. We further provide considerations for optimization of the assay conditions and its broader application in studies of other protein-protein interactions. Copyright © 2011 Brandon J. Biesiadecki and J.-P. Jin.',\n",
              "   'title': '133\\nA high-throughput solid-phase microplate protein-binding assay to investigate interactions between myofilament proteins\\nOpen Access\\nBiesiadecki, B.J., Jin, J.-P. 2011 Journal of Biomedicine and Biotechnology\\n2011,421701\\n14'},\n",
              "  {'abstract': 'Living organisms produce a myriad of molecules to protect themselves from fungal pathogens. This review focuses on antifungal proteins from plants and mushrooms, many of which are components of the human diet or have medicinal value. Plant antifungal proteins can be classified into different groups comprising chitinases and chitinase-like proteins, chitin-binding proteins, cyclophilin-like proteins, defensins and defensin-like proteins, deoxyribonucleases, embryo-abundant protein-like proteins, glucanases, lectins, lipid transfer proteins, peroxidases, protease inhibitors, ribonucleases, ribosome-inactivating proteins, storage 2S albumins, and thaumatin-like proteins. Some of the aforementioned antifungal proteins also exhibit mitogenic activity towards spleen cells, nitric oxide inducing activity toward macrophages, antiproliferative activity toward tumor cells, antibacterial activity, and inhibitory activity toward HIV-1 reverse transcriptase. In contrast to the large diversity of plant antifungal proteins, only a small number of mushroom antifungal proteins have been reported. Mushroom antifungal proteins are distinct from their plant counterparts in N-terminal sequence. Nevertheless, some of the mushroom antifungal proteins have been shown to inhibit HIV-1 reverse transcriptase activity and tumor cell proliferation. © 2010 Springer-Verlag.',\n",
              "   'title': '134\\nProteins with antifungal properties and other medicinal applications from plants and mushrooms Wong, J.H., Ng, T.B., Cheung, R.C.F., (...), Jiang, Y., Liu, F. 2010 Applied Microbiology and Biotechnology\\n87(4), pp. 1221-1235\\n90'},\n",
              "  {'abstract': 'MPN (Mpr1/Pad1 N-terminal) domain-containing proteins are present throughout all domains of life. In eukaryotes, MPN domain-containing proteins are commonly found in association with other molecules in large protein complexes, where examples comprise; the 26S proteasome and the COP9 (Constitutive photomorphogenesis 9) signalosome complexes, including the MPN subunits, POH1 and Mov34, CSN5 and CSN6, respectively. Examples of MPN domaincontaining proteins that are not incorporated in a large multi-protein complex have also been reported and include AMSH (for associated molecule with the SH3 domain of STAM) and the AMSH-Like Protein (AMSH-LP). Within the MPN domain super-family, two main subclasses have been characterised: the MPN+ and MPN- domain-containing proteins. MPN+ domain-containing proteins are classified as metalloenzymes responsible for isopeptidase activity. These proteins display a JAMM (JAB1-MPN-MOV34) metalloisopeptidase motif, typically consisting of a canonical sequence (E-x[2]-H-S/T-Hx[ 7]-S-x[2]-D) and coordinating a zinc ion. The JAMM motif specifies a catalytic centre essential for selective hydrolysis of linkages, contained between ubiquitin/ubiquitin-like proteins and target proteins or between ubiquitin monomers within a polymeric chain. The MPN- family classifies proteins, which lack the key residues present in the typical JAMM motif. These MPN- proteins are void of catalytic activity, but recent studies have proposed a role in mediating protein-protein interactions, in acting as a scaffold or in activity regulation. In light of recent structural and functional studies, a more detailed understanding of these proteins has been gained and is given in the present review. © 2014 Bentham Science Publishers.',\n",
              "   'title': '135\\nStructure and function of MPN (Mpr1/Pad1 N-terminal) domain-containing proteins Birol, M., Echalier, A. 2014 Current Protein and Peptide Science\\n15(5), pp. 504-517\\n16'},\n",
              "  {'abstract': '© 2015 FEBS B-cell lymphoma 2 (BCL-2) family proteins mediate mitochondrial apoptosis by regulating mitochondrial outer membrane permeabilization (MOMP), which leads to the activation of the downstream caspase cascade to execute apoptosis. The pro-apoptotic and anti-apoptotic BCL-2 proteins function through protein-protein interactions in soluble and membrane-associated states. How soluble BCL-2 proteins interact is well understood. Anti-apoptotic proteins, such as BCL-2 and BCL-xL, and the pro-apoptotic effectors of MOMP, including BAK and BAX, interact with pro-apoptotic BCL-2 homology 3 (BH3)-only proteins similarly. Whereas anti-apoptotic BCL-2 proteins tightly bind all the BH3-only proteins to block apoptosis initiation, the effector BCL-2 proteins are potently triggered by specific BH3-only proteins to undergo conformational changes, membrane association and insertion, oligomerization, and pore formation. The anti-apoptotic BCL-2 proteins also inhibit the activated effectors. p53 is a direct BAX activator inhibited by BCL-xL, defining a prototype non-canonical modulator of BCL-2 proteins-mediated MOMP. How BCL-2 proteins cooperate in the presence of membranes remains poorly understood, impeding our understanding of MOMP and apoptosis. Here, we highlight the latest structural views of MOMP by BCL-2 proteins.',\n",
              "   'title': '136\\nDiscoveries and controversies in BCL-2 protein-mediated apoptosis Zheng, J.H., Viacava Follis, A., Kriwacki, R.W., Moldoveanu, T. 2016 FEBS Journal\\npp. 2690-2700\\n116'},\n",
              "  {'abstract': 'The nucleus is the largest and the highly organized organelle of eukaryotic cells. Within nucleus exist a number of pseudo-compartments, which are not separated by any membrane, yet each of them contains only a specific set of proteins. Understanding protein sub-nuclear localization can hence be an important step towards understanding biological functions of the nucleus. Here we have described a method, SubNucPred developed by us for predicting the sub-nuclear localization of proteins. This method predicts protein localization for 10 different sub-nuclear locations sequentially by combining presence or absence of unique Pfam domain and amino acid composition based SVM model. The prediction accuracy during leave-one-out cross-validation for centromeric proteins was 85.05%, for chromosomal proteins 76.85%, for nuclear speckle proteins 81.27%, for nucleolar proteins 81.79%, for nuclear envelope proteins 79.37%, for nuclear matrix proteins 77.78%, for nucleoplasm proteins 76.98%, for nuclear pore complex proteins 88.89%, for PML body proteins 75.40% and for telomeric proteins it was 83.33%. Comparison with other reported methods showed that SubNucPred performs better than existing methods. A web-server for predicting protein sub-nuclear localization named SubNucPred has been established at http://14.139.227.92/mkumar/subnucpred/. Standalone version of SubNucPred can also be downloaded from the webserver. © 2014 Kumar et al.',\n",
              "   'title': '137\\nProtein sub-nuclear localization prediction using SVM and Pfam domain information\\nOpen Access\\nKumar, R., Jain, S., Kumari, B., Kumar, M. 2014 PLoS ONE\\n9(6),e98345\\n23'},\n",
              "  {'abstract': 'The nonstructural NSm protein of tomato spotted wilt tospovirus (TSWV) represents a putative viral movement protein involved in cell-to-cell movement of nonenveloped ribonucleocapsid structures. To study the molecular basis of NSm function, we expressed the protein in Escherichia coli and investigated protein-protein and protein-RNA interactions of NSm protein in vitro. NSm specifically interacts with TSWV N protein and binds single- stranded RNA in a sequence-nonspecific manner. Using NSm as a bait in a yeast two-hybrid screen, we identified two homologous NSm-binding proteins of the DnaJ family from Nicotiana tabacum and Arabidopsis thaliana.',\n",
              "   'title': '138\\nThe movement protein NSm of tomato spotted wilt tospovirus (TSWV): RNA binding, interaction with the TSWV N protein, and identification of interacting plant proteins\\nOpen Access\\nSoellick, T.-R., Uhrig, J.F., Bucher, G.L., Kellmann, J.-W., Schreier, P.H. 2000 Proceedings of the National Academy of Sciences of the United States of America\\n97(5), pp. 2373-2378\\n135'},\n",
              "  {'abstract': 'Protein-protein interactions are observed in various biological processes. They are important for understanding the underlying molecular mechanisms and can be potential targets for developing small-molecule regulators of such processes. Previous studies suggest that certain residues on protein-protein binding interfaces are ″hot spots″. As an extension to this concept, we have developed a residue-based method to identify the characteristic interaction patterns (CIPs) on protein-protein binding interfaces, in which each pattern is a cluster of four contacting residues. Systematic analysis was conducted on a nonredundant set of 1,222 protein-protein binding interfaces selected out of the entire Protein Data Bank. Favored interaction patterns across different protein-protein binding interfaces were retrieved by considering both geometrical and chemical conservations. As demonstrated on two test tests, our method was able to predict hot spot residues on protein-protein binding interfaces with good recall scores and acceptable precision scores. By analyzing the function annotations and the evolutionary tree of the protein-protein complexes in our data set, we also observed that protein-protein interfaces sharing common characteristic interaction patterns are normally associated with identical or similar biological functions. © 2013 American Chemical Society.',\n",
              "   'title': '139\\nMining the characteristic interaction patterns on protein-protein binding interfaces Li, Y., Liu, Z., Han, L., Li, C., Wang, R. 2013 Journal of Chemical Information and Modeling\\n53(9), pp. 2437-2447\\n14'},\n",
              "  {'abstract': 'The RecA protein is a recombinase functioning in recombinational DNA repair in bacteria. RecA is regulated at many levels. The expression of the recA gene is regulated within the SOS response. The activity of the RecA protein itself is autoregulated by its own C-terminus. RecA is also regulated by the action of other proteins. To date, these include the RecF, RecO, RecR, DinI, RecX, RdgC, PsiB, and UvrD proteins. The SSB protein also indirectly affects RecA function by competing for ssDNA binding sites. The RecO and RecR, and possibly the RecF proteins, all facilitate RecA loading onto SSB-coated ssDNA. The RecX protein blocks RecA filament extension, and may have other effects on RecA activity. The DinI protein stabilizes RecA filaments. The RdgC protein binds to dsDNA and blocks RecA access to dsDNA. The PsiB protein, encoded by F plasmids, is uncharacterized, but may inhibit RecA in some manner. The UvrD helicase removes RecA filaments from RecA. All of these proteins function in a network that determines where and how RecA functions. Additional regulatory proteins may remain to be discovered. The elaborate regulatory pattern is likely to be reprised for RecA homologues in archaeans and eukaryotes. Copyright © Informa Healthcare.',\n",
              "   'title': '140\\nRegulation of bacterial RecA protein function Cox, M.M. 2007 Critical Reviews in Biochemistry and Molecular Biology\\n42(1), pp. 41-63\\n257'},\n",
              "  {'abstract': 'We have developed a novel system designated the luciferase assay for protein detection (LAPD) to study protein-protein interactions. This method involves two protein fusions, a soluble reporter fusion and a fusion for immobilizing the target protein. The soluble reporter is an N-terminal Renilla luciferase fusion protein that exhibits high Renilla luciferase activity. Crude cleared lysates from transfected Cos1 cells that express the Renilla luciferase fusion protein can be used in binding assays with immobilized target proteins. Following incubation and washing, target-bound Renilla luciferase fusion proteins produce light from the coelenterazine substrate, indicating an interaction between the two proteins of interest. As proof of the principle, we reproduced known, transient protein-protein interactions between the Cdc42 GTPase and its effector proteins. GTPase Renilla fusion proteins produced in Cos1 cells were tested with immobilized recombinant GST-N-WASP and CEP5 effector proteins. Using this assay, we could detect specific interactions of Cdc42 with these effector proteins in approximately 50 min. The specificity of these interactions was demonstrated by showing that they were GTPase-specific and GTP-dependent and not seen with other unrelated target proteins. These results suggest that the LAPD method, which is both rapid and sensitive, may have research and practical applications.',\n",
              "   'title': '141\\nDetecting protein-protein interactions using Renilla luciferase fusion proteins\\nOpen Access\\nBurbelo, P.D., Kisailus, A.E., Peck, J.W. 2002 BioTechniques\\n33(5), pp. 1044-1050\\n5'},\n",
              "  {'abstract': 'Iron-sulfur (Fe-S) clusters are essential cofactors of proteins with a wide range of biological functions. A dedicated cytosolic Fe-S cluster assembly (CIA) system is required to assemble Fe-S clusters into cytosolic and nuclear proteins. Here, we show that the mammalian nucleotide excision repair protein homolog MMS19 can simultaneously bind probable cytosolic iron-sulfur protein assembly protein CIAO1 and Fe-S proteins, confirming that MMS19 is a central protein of the CIA machinery that brings Fe-S cluster donor proteins and the receiving apoproteins into proximity. In addition, we show that mitotic spindle-associated MMXD complex subunit MIP18 also interacts with both CIAO1 and Fe-S proteins. Specifically, it binds the Fe-S cluster coordinating regions in Fe-S proteins. Furthermore, we show that ADP/ATP translocase 2 (ANT2) interacts with Fe-S apoproteins and MMS19 in the CIA complex but not with the individual proteins. Together, these results elucidate the composition and interactions within the late CIA complex. © 2012 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '142\\nThe mammalian proteins MMS19, MIP18, and ANT2 are involved in cytoplasmic iron-sulfur cluster protein assembly\\nOpen Access\\nVan Wietmarschen, N., Moradian, A., Morin, G.B., Lansdorp, P.M., Uringa, E.-J. 2012 Journal of Biological Chemistry\\n287(52), pp. 43351-43358\\n36'},\n",
              "  {'abstract': '© 2016 Elsevier Inc. All rights reserved. Protein-protein interactions are fundamental for most biological processes, such as the formation of cellular structures and enzymatic complexes or in signaling pathways. The identification and characterization of protein-protein interactions are therefore essential for understanding the mechanisms and regulation of biological systems. The organization and dynamics of the cytoskeleton, as well as its anchorage to specific sites in the plasma membrane and organelles, are regulated by the plakins. These structurally related proteins anchor different cytoskeletal networks to each other and/or to other cellular structures. The association of several plakins with intermediate filaments (IFs) is critical for maintenance of the cytoarchitecture. Pathogenic mutations in the genes encoding different plakins can lead to dramatic manifestations, occurring principally in the skin, striated muscle, and/or nervous system, due to cytoskeletal disorganization resulting in abnormal cell fragility. Nevertheless, it is still unclear how plakins bind to IFs, although some general rules are slowly emerging. We here describe in detail a recently developed protein-protein fluorescence binding assay, based on the production of recombinant proteins tagged with green fluorescent protein (GFP) and their use as fluid-phase fluorescent ligands on immobilized IF proteins. Using this method, we have been able to assess the ability of C-terminal regions of GFP-tagged plakin proteins to bind to distinct IF proteins and IF domains. This simple and sensitive technique, which is expected to facilitate further studies in this area, can also be potentially employed for any kind of protein-protein interaction studies.',\n",
              "   'title': '143\\nDevelopment of a Novel Green Fluorescent Protein-Based Binding Assay to Study the Association of Plakins with Intermediate Filament Proteins Favre, B., Begré, N., Bouameur, J.-E., Borradori, L. 2016 Methods in Enzymology\\n569, pp. 117-137\\n4'},\n",
              "  {'abstract': 'We present a computational procedure for modeling protein-protein association and predicting the structures of protein-protein complexes. The initial sampling stage is based on an efficient Brownian dynamics algorithm that mimics the physical process of diffusional association. Relevant biochemical data can be directly incorporated as distance constraints at this stage. The docked configurations are then grouped with a hierarchical clustering algorithm into ensembles that represent potential protein-protein encounter complexes. Flexible refinement of selected representative structures is done by molecular dynamics simulation. The protein-protein docking procedure was thoroughly tested on 10 structurally and functionally diverse protein-protein complexes. Starting from X-ray crystal structures of the unbound proteins, in 9 out of 10 cases it yields structures of protein-protein complexes close to those determined experimentally with the percentage of correct contacts >30% and interface backbone RMSD <4 A. Detailed examination of all the docking cases gives insights into important determinants of the performance of the computational approach in modeling protein-protein association and predicting of protein-protein complex structures. © 2008 Wiley-Liss, Inc.',\n",
              "   'title': '144\\nProtein-protein docking by simulating the process of association subject to biochemical constraints Motiejunas, D., Gabdoulline, R., Wang, T., (...), Winn, P.J., Wade, R.C. 2008 Proteins: Structure, Function and Genetics\\n71(4), pp. 1955-1969\\n24'},\n",
              "  {'abstract': 'Assigning functions to novel proteins is one of the most important problems in the postgenomic era. Several approaches have been applied to this problem, including the analysis of gene expression patterns, phylogenetic profiles, protein fusions, and protein-protein interactions. In this paper, we develop a novel approach that employs the theory of Markov random fields to infer a protein\\'s functions using protein-protein interaction data and the functional annotations of protein\\'s interaction partners. For each function of interest and protein, we predict the probability that the protein has such function using Bayesian approaches. Unlike other available approaches for protein annotation in which a protein has or does not have a function of interest, we give a probability for having the function. This probability indicates how confident we are about the prediction. We employ our method to predict protein functions based on \"biochemical function,\" \"subcellular location,\" and \"cellular role\" for yeast proteins defined in the Yeast Proteome Database (YPD, www.incyte.com), using the protein-protein interaction data from the Munich Information Center for Protein Sequences (MIPS, mips.gsf.de). We show that our approach outperforms other available methods for function prediction based on protein interaction data. The supplementary data is available at www-hto.usc.edu/~msms/ ProteinFunction.',\n",
              "   'title': '145\\nPrediction of Protein Function Using Protein-Protein Interaction Data Deng, M., Zhang, K., Mehta, S., Chen, T., Sun, F. 2003 Journal of Computational Biology\\n10(6), pp. 947-960\\n215'},\n",
              "  {'abstract': 'For elucidating protein-protein interactions, many methodologies have been developed during the past two decades. For investigation of interactions inside cells under physiological conditions, yeast is an attractive organism with which to quickly screen for hopeful candidates using versatile genetic technologies, and various types of approaches are now available. Among them, a variety of unique systems using the guanine nucleotide-binding protein (G-protein) signaling pathway in yeast have been established to investigate the interactions of proteins for biological study and pharmaceutical research. G-proteins involved in various cellular processes are mainly divided into two groups: Small monomeric G-proteins, and heterotrimeric G-proteins. In this minireview, we summarize the basic principles and applications of yeast-based screening systems, using these two types of G-protein, which are typically used for elucidating biological protein interactions but are differentiated from traditional yeast two-hybrid systems. © 2010 FEBS.',\n",
              "   'title': '146\\nProtein-protein interactions and selection: Yeast-based approaches that exploit guanine nucleotide-binding protein signaling\\nOpen Access\\nIshii, J., Fukuda, N., Tanaka, T., Ogino, C., Kondo, A. 2010 FEBS Journal\\n277(9), pp. 1982-1995\\n25'},\n",
              "  {'abstract': 'Adeno-associated virus (AAV) codes for four related nonstructural Rep proteins. AAV both replicates and assembles in the nucleus and requires coinfection with a helper virus, either adenovirus (Ad) or herpesvirus, for a productive infection. Like other more complex DNA viruses, it is believed that AAV interacts or modifies host cell proteins to carry out its infection cycle. To date, relatively little is known about the host proteins that interact with the viral Rep proteins, which are known to be directly involved in DNA replication, control of viral and cellular transcription, splicing, and protein translation. In this study, we used affinity-tagged Rep protein to purify cellular protein complexes that were associated with Rep in cells that had been infected with Ad and AAV. In all, we identified 188 cellular proteins from 16 functional categories, including 14 transcription factors, 6 translation factors, 15 potential splicing proteins, 5 proteins involved in protein degradation, and 13 proteins involved in DNA replication or repair. This dramatically increases the number of potential interactions over the current number of approximately 26. Twelve of the novel proteins found were further tested by coimmunoprecipitation or colocalization using confocal immunomicroscopy. Of these, 10 were confirmed as proteins that formed complexes with Rep, including proteins of the MCM complex (DNA replication), RCN1 (membrane transport), SMC2 (chromatin dynamics), EDD1 (ubiquitin ligase), IRS4 (signal transduction), and FUS (splicing). Computer analysis suggested that 45 and 28 of the 188 proteins could be placed in a pathway of interacting proteins involved in DNA replication and protein synthesis, respectively. Of the proteins involved in DNA replication, all of the previously identified proteins involved in AAV DNA replication were found, except Ad DBP. The only Ad protein found to interact with Rep was the E1b55K protein. In addition, we confirmed that Rep interacts with Ku70/80 helicase. In vitro DNA synthesis assays demonstrated that although Ku helicase activity could substitute for MCM to promote strand displacement synthesis, its presence was not essential. Our study suggests that the interaction of AAV with cellular proteins is much more complex than previously suspected and provides a resource for further studies of the AAV life cycle. Copyright © 2009, American Society for Microbiology. All Rights Reserved.',\n",
              "   'title': '147\\nIdentification of cellular proteins that interact with the adeno-associated virus Rep protein\\nOpen Access\\nNash, K., Chen, W., Salganik, M., Muzyczka, N. 2009 Journal of Virology\\n83(1), pp. 454-469\\n43'},\n",
              "  {'abstract': 'Clinical and consumer market interest is increasingly directed toward the use of plant-based proteins as dietary components aimed at preserving or increasing skeletal muscle mass. However, recent evidence suggests that the ingestion of the plant-based proteins in soy and wheat results in a lower muscle protein synthetic response when compared with several animal-based proteins. The possible lower anabolic properties of plant-based protein sources may be attributed to the lower digestibility of plant-based sources, in addition to greater splanchnic extraction and subsequent urea synthesis of plant protein-derived amino acids compared with animal-based proteins. The latter may be related to the relative lack of specific essential amino acids in plant- as opposed to animal-based proteins. Furthermore, most plant proteins have a relatively low leucine content, which may further reduce their anabolic properties when compared with animal proteins. However, few studies have actually assessed the postprandial muscle protein synthetic response to the ingestion of plant proteins, with soy and wheat protein being the primary sources studied. Despite the proposed lower anabolic properties of plant vs. animal proteins, various strategies may be applied to augment the anabolic properties of plant proteins. These may include the following: 1) fortification of plant-based protein sources with the amino acids methionine, lysine, and/or leucine; 2) selective breeding of plant sources to improve amino acid profiles; 3) consumption of greater amounts of plant-based protein sources; or 4) ingesting multiple protein sources to provide a more balanced amino acid profile. However, the efficacy of such dietary strategies on postprandial muscle protein synthesis remains to be studied. Future research comparing the anabolic properties of a variety of plant-based proteins should define the preferred protein sources to be used in nutritional interventions to support skeletal muscle mass gain or maintenance in both healthy and clinical populations.',\n",
              "   'title': '148\\nThe skeletal muscle anabolic response to plant- versus animal-based protein consumption\\nOpen Access\\nvan Vliet, S., Burd, N.A., van Loon, L.J.C. 2015 Journal of Nutrition\\n145(9), pp. 1981-1991\\n162'},\n",
              "  {'abstract': '© 2018, Springer International Publishing AG, part of Springer Nature. Modulating protein interaction pathways may lead to the cure of many diseases. Known protein–protein inhibitors bind to large pockets on the protein–protein interface. Such large pockets are detected also in the protein–protein complexes without known inhibitors, making such complexes potentially druggable. The inhibitor-binding site is primary defined by the side chains that form the largest pocket in the protein-bound conformation. Low-resolution ligand docking shows that the success rate for the protein-bound conformation is close to the one for the ligand-bound conformation, and significantly higher than for the apo conformation. The conformational change on the protein interface upon binding to the other protein results in a pocket employed by the ligand when it binds to that interface. This proof-of-concept study suggests that rather than using computational pocket-opening procedures, one can opt for an experimentally determined structure of the target co-crystallized protein–protein complex as a starting point for drug design.',\n",
              "   'title': '149\\nInhibition of protein interactions: co-crystalized protein–protein interfaces are nearly as good as holo proteins in rigid-body ligand docking Belkin, S., Kundrotas, P.J., Vakser, I.A. 2018 Journal of Computer-Aided Molecular Design\\n32(7), pp. 769-779\\n1'},\n",
              "  {'abstract': '© 2016 Wiley Periodicals, Inc. This paper proposes a generalization of the well-known folding funnel concept of proteins. In the funnel model the polypeptide chain is treated as an individual object not interacting with other proteins. Since biological systems are considerably crowded, protein–protein interaction is a fundamental feature during the life cycle of proteins. The folding superfunnel proposed here describes the folding process of interacting proteins in various situations. The first example discussed is the folding of the freshly synthesized protein with the aid of chaperones. Another important aspect of protein–protein interactions is the folding of the recently characterized intrinsically disordered proteins, where binding to target proteins plays a crucial role in the completion of the folding process. The third scenario where the folding superfunnel is used is the formation of aggregates from destabilized proteins, which is an important factor in case of several conformational diseases. The folding superfunnel constructed here with the minimal assumption about the interaction potential explains all three cases mentioned above. Proteins 2016; 84:1009–1016. © 2016 Wiley Periodicals, Inc.',\n",
              "   'title': '150\\nFolding superfunnel to describe cooperative folding of interacting proteins Smeller, L. 2016 Proteins: Structure, Function and Bioinformatics\\n84(7), pp. 1009-1016\\n1'},\n",
              "  {'abstract': 'Interactions of human MCM2-7 proteins with the proteins of TIM, TIPIN, an amino-terminal fragment of Rb, and p27 were examined by co-immuno-precipitation experiment using cell lysates of co-expressed insect cells. TIM and TIPIN, both of which are involved in regulation of DNA replication fork progression, mainly interacted with MCM3-7 proteins. The amino-terminal fragment of Rb, which inhibits DNA replication in Xenopus egg extracts, was able to bind with MCM3 and MCM6 proteins in addition to MCM7 protein. In contrast, p27 was not able to bind any MCM2-7 proteins under the comparable conditions. These results indicate that the proteins, which are known to interact with MCM proteins, bind with MCM2-7 proteins with different affinities and specificities. © The Authors 2010. Published by Oxford University Press on behalf of the Japanese Biochemical Society. All rights reserved.',\n",
              "   'title': '151\\nInteraction of human MCM2-7 proteins with TIM, TIPIN and Rb Numata, Y., Ishihara, S., Hasegawa, N., Nozaki, N., Ishimi, Y. 2010 Journal of Biochemistry\\n147(6), pp. 917-927\\n17'},\n",
              "  {'abstract': '© 2017 The majority of research topics declared that most of the recombinant proteins have been expressed by Escherichia coli in basic investigations. But the majority of high expressed proteins formed as inactive recombinant proteins that are called inclusion body. To overcome this problem, several methods have been used including suitable promoter, environmental factors, ladder tag to secretion of proteins into the periplasm, gene protein optimization, chemical chaperones and molecular chaperones sets. Co-expression of the interest protein with molecular chaperones is one of the common methods The chaperones are a group of proteins, which are involved in making correct folding of recombinant proteins. Chaperones are divided two groups including; cytoplasmic and periplasmic chaperones. Moreover, periplasmic chaperones and proteases can be manipulated to increase the yields of secreted proteins. In this article, we attempted to review cytoplasmic chaperones such as Hsp families and periplasmic chaperones including; generic chaperones, specialized chaperones, PPIases, and proteins involved in disulfide bond formation.',\n",
              "   'title': '152\\nAn overview on molecular chaperones enhancing solubility of expressed recombinant proteins with correct folding Mamipour, M., Yousefi, M., Hasanzadeh, M. 2017 International Journal of Biological Macromolecules\\n102, pp. 367-375\\n5'},\n",
              "  {'abstract': '© 2019 Sha et al. Protein ubiquitination and SUMOylation are required for the maintenance of cellular protein homeostasis, and both increase in proteotoxic conditions (e.g. heat shock or proteasome inhibition). However, we found that when ubiquitination was blocked in several human cell lines by inhibiting the ubiquitin-activating enzyme with TAK243, there was an unexpected, large accumulation of proteins modified by SUMO2/3 chains or SUMO1, but not by several other ubiquitin-like proteins. This buildup of SUMOylated proteins was evident within 3– 4 h. It required the small ubiquitin-like modifier (SUMO)-conjugating enzyme, UBC9, and the promyelocytic leukemia protein (PML) and thus was not due to nonspecific SUMO conjugation by ubiquitination enzymes. The SUMOylated proteins accumulated predominantly bound to chromatin and were localized to PML nuclear bodies. Because blocking protein synthesis with cycloheximide prevented the buildup of SUMOylated proteins, they appeared to be newly-synthesized proteins. The proteins SUMOylated after inhibition of ubiquitination were purified and analyzed by MS. In HeLa and U2OS cells, there was a cycloheximide-sensitive increase in a similar set of SUMOylated proteins (including transcription factors and proteins involved in DNA damage repair). Surprisingly, the inhibition of ubiquitination also caused a cycloheximide-sensitive decrease in a distinct set of SUMOylated proteins (including proteins for chromosome modification and mRNA splicing). More than 80% of the SUMOylated proteins whose levels rose or fell upon inhibiting ubiquitination inhibition underwent similar cycloheximide-sensitive increases or decreases upon proteasome inhibition. Thus, when nuclear substrates of the ubiquitin–proteasome pathway are not efficiently degraded, many become SUMO-modified and accumulate in PML bodies.',\n",
              "   'title': '153\\nInhibiting ubiquitination causes an accumulation of SUMOylated newly synthesized nuclear proteins at PML bodies\\nOpen Access\\nSha, Z., Blyszcz, T., González-Prieto, R., Vertegaal, A.C.O., Goldberg, A.L. 2019 Journal of Biological Chemistry\\n294(42), pp. 15218-15234\\n4'},\n",
              "  {'abstract': 'Polycomb group (PcG) proteins maintain transcriptional repression of hundreds of genes involved in development, signaling or cancer using chromatin-based epigenetic mechanisms. Biochemical studies in Drosophila have revealed that PcG proteins associate in at least two classes of protein complexes known as Polycomb repressive complexes 1 and 2 (PRC1 and PRC2). Drosophila core PRC1 is composed of four subunits, Polycomb (Pc), Sex combs extra (Sce), Polyhomeotic (Ph), and Posterior sex combs (Psc). Each of these proteins has multiple orthologs in vertebrates classified respectively as the CBX, RING1/RNF2, PHC, and BMI1/PCGF families. Mammalian genomes encode five CBX family members (CBX2, CBX4, CBX6, CBX7, and CBX8) that are believed to have distinct biological functions. Here, we applied a tandem affinity purification (TAP) approach coupled with tandem mass spectrometry (MS/MS) methodologies in order to identify interacting partners of CBX family proteins under the same experimental conditions. Our analysis identified with high confidence about 20 proteins co-eluted with CBX2 and CBX7 tagged proteins, about 40 with CBX4, and around 60 with CBX6 and CBX8. We provide evidences that the CBX family proteins are mutually exclusive and define distinct PRC1-like protein complexes. CBX proteins also interact with different efficiencies with the other PRC1 components. Among the novel CBX interacting partners, protein kinase 2 associates with all CBX-PRC1 protein complexes, whereas 14-3-3 proteins specifically bind to CBX4. 14-3-3 protein binding to CBX4 appears to modulate the interaction between CBX4 and the BMI1/PCGF components of PRC1, but has no effect on CBX4-RING1/RNF2 interaction. Finally, we suggest that differences in CBX protein interactions would account, at least in part, for distinct subnuclear localization of the CBX family members. © 2011 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '154\\nInteraction proteomics analysis of polycomb proteins defines distinct PRC1 complexes in mammalian cells\\nOpen Access\\nVandamme, J., Völkel, P., Rosnoblet, C., Le Faou, P., Angrand, P.-O. 2011 Molecular and Cellular Proteomics\\n10(4)\\n86'},\n",
              "  {'abstract': 'Background: In the search for generic expression strategies for mammalian protein families several bacterial expression vectors were examined for their ability to promote high yields of soluble protein. Proteins studied included cell surface receptors (Ephrins and Eph receptors, CD44), kinases (EGFR-cytoplasmic domain, CDK2 and 4), proteases (MMP1, CASP2), signal transduction proteins (GRB2, RAF1, HRAS) and transcription factors (GATA2, Fli1, Trp53, Mdm2, JUN, FOS, MAD, MAX). Over 400 experiments were performed where expression of 30 full-length proteins and protein domains were evaluated with 6 different N-terminal and 8 C-terminal fusion partners. Expression of an additional set of 95 mammalian proteins was also performed to test the conclusions of this study. Results: Several protein features correlated with soluble protein expression yield including molecular weight and the number of contiguous hydrophobic residues and low complexity regions. There was no relationship between successful expression and protein pI, grand average of hydropathicity (GRAVY), or sub-cellular location. Only small globular cytoplasmic proteins with an average molecular weight of 23 kDa did not require a solubility enhancing tag for high level soluble expression. Thioredoxin (Trx) and maltose binding protein (MBP) were the best N-terminal protein fusions to promote soluble expression, but MBP was most effective as a C-terminal fusion. 63 of 95 mammalian proteins expressed at soluble levels of greater than 1 mg/l as N-terminal H10-MBP fusions and those that failed possessed, on average, a higher molecular weight and greater number of contiguous hydrophobic amino acids and low complexity regions. Conclusions: By analysis of the protein features identified here, this study will help predict which mammalian proteins and domains can be successfully expressed in E. coli as soluble product and also which are best targeted for a eukaryotic expression system. In some cases proteins may be truncated to minimise molecular weight and the numbers of contiguous hydrophobic amino acids and low complexity regions to aid soluble expression in E. coli. © 2004 Dyson et al; licensee BioMed Central Ltd.',\n",
              "   'title': '155\\nProduction of soluble mammalian proteins in Escherichia coli: Identification of protein features that correlate with successful expression\\nOpen Access\\nDyson, M.R., Shadbolt, S.P., Vincent, K.J., Perera, R.L., McCafferty, J. 2004 BMC Biotechnology\\n4,32\\n193'},\n",
              "  {'abstract': 'The Protein 4.1 family contains at least two members that function as tumour suppressors, the neurofibromatosis 2 gene product merlin and the recently identified differentially expressed in adenocarcinoma of the lung (DAL-1)/Protein 4.1B molecule. DAL-1/Protein 4.1B loss is observed in a variety of tumours, including breast and lung cancers as well as meningiomas. We have previously demonstrated that DAL-1/Protein 4.1B interacts with some but not all merlin-binding proteins, raising the possibility that DAL-1/Protein 4.1B associates with additional unique proteins specific to its function as a negative growth regulator. Using yeast two-hybrid interaction cloning, we identified three 14-3-3 isoforms, β, γ and η, to be DAL-1/Protein 4.1B-binding proteins. These interactions were verified by using glutathione S-transferase affinity chromatography in vitro and co-immunoprecipitation in vivo. The interaction of 14-3-3 with DAL-1/Protein 4.1B was specific, as 14-3-3 did not bind to the related Protein 4.1 family members merlin, ezrin or radixin. The DAL-1/Protein 4.1B domain that mediates 14-3-3 binding was mapped to residues Pro244 and Leu280 within the 4.1/ezrin/radixin/moesin domain. The identification of this novel DAL-1/Protein 4.1B-interacting protein represents the first step towards elucidating its potentially unique mechanism of action.',\n",
              "   'title': '156\\nThe 4.1/ezrin/radixin/moesin domain of the DAL-1/Protein 4.1B tumour suppressor interacts with 14-3-3 proteins Yu, T., Robb, V.A., Singh, V., Gutmann, D.H., Newsham, I.F. 2002 Biochemical Journal\\n365(3), pp. 783-789\\n35'},\n",
              "  {'abstract': '© 2017 American Chemical Society. Protein regions that are involved in protein-protein interactions (PPIs) very often display a high degree of intrinsic disorder, which is reduced during the recognition process. A prime example is binding of the rigid 14-3-3 adapter proteins to their numerous partner proteins, whose recognition motifs undergo an extensive disorder-to-order transition. In this context, it is highly desirable to control this entropy-costly process using tailored stabilizing agents. This study reveals how the molecular tweezer CLR01 tunes the 14-3-3/Cdc25CpS216 protein-protein interaction. Protein crystallography, biophysical affinity determination and biomolecular simulations unanimously deliver a remarkable finding: a supramolecular \"Janus\" ligand can bind simultaneously to a flexible peptidic PPI recognition motif and to a well-structured adapter protein. This binding fills a gap in the protein-protein interface, \"freezes\" one of the conformational states of the intrinsically disordered Cdc25C protein partner and enhances the apparent affinity of the interaction. This is the first structural and functional proof of a supramolecular ligand targeting a PPI interface and stabilizing the binding of an intrinsically disordered recognition motif to a rigid partner protein.',\n",
              "   'title': '157\\nThe Molecular Tweezer CLR01 Stabilizes a Disordered Protein-Protein Interface\\nOpen Access\\nBier, D., Mittal, S., Bravo-Rodriguez, K., (...), Schrader, T., Ottmann, C. 2017 Journal of the American Chemical Society\\n139(45), pp. 16256-16263\\n32'},\n",
              "  {'abstract': 'Nitration of tyrosine residues in proteins is associated with cell response to oxidative/nitrosative stress. Tyrosine nitration is relatively low abundant post-translational modification that may affect protein functions. Little is known about the extent of protein tyrosine nitration in cells during progression through the cell cycle. Here we report identification of proteins enriched for tyrosine nitration in cells synchronized in G0/G1, S or G2/M phases of the cell cycle. We identified 27 proteins in cells synchronized in G0/G1 phase, 37 proteins in S phase synchronized cells, and 12 proteins related to G2/M phase. Nineteen of the identified proteins were previously described as regulators of cell proliferation. Thus, our data indicate which tyrosine nitrated proteins may affect regulation of the cell cycle. © 2011 Elsevier Inc.',\n",
              "   'title': '158\\nProtein tyrosine nitration in the cell cycle Jia, M., Mateoiu, C., Souchelnytskyi, S. 2011 Biochemical and Biophysical Research Communications\\n413(2), pp. 270-276\\n5'},\n",
              "  {'abstract': '© 2015, Springer-Verlag Berlin Heidelberg. The intracellular transport of membrane proteins is controlled by trafficking signals: Short peptide motifs that mediate the contact with COPI, COPII or various clathrin-associated coat proteins. In addition, many membrane proteins interact with accessory proteins that are involved in the sorting of these proteins to different intracellular compartments. In the K2P channels, TASK-1 and TASK-3, the influence of protein–protein interactions on sorting decisions has been studied in some detail. Both TASK paralogues interact with the adaptor protein 14-3-3; TASK-1 interacts, in addition, with the adaptor protein p11 (S100A10) and the endosomal SNARE protein syntaxin-8. The role of these interacting proteins in controlling the intracellular traffic of the channels and the underlying molecular mechanisms are summarised in this review. In the case of 14-3-3, the interacting protein masks a retention signal in the C-terminus of the channel; in the case of p11, the interacting protein carries a retention signal that localises the channel to the endoplasmic reticulum; and in the case of syntaxin-8, the interacting protein carries an endocytosis signal that complements an endocytosis signal of the channel. These examples illustrate some of the mechanisms by which interacting proteins may determine the itinerary of a membrane protein within a cell and suggest that the intracellular traffic of membrane proteins may be adapted to the specific functions of that protein by multiple protein–protein interactions.',\n",
              "   'title': '159\\nThe role of protein–protein interactions in the intracellular traffic of the potassium channels TASK-1 and TASK-3 Kilisch, M., Lytovchenko, O., Schwappach, B., Renigunta, V., Daut, J. 2015 Pflugers Archiv European Journal of Physiology\\n467(5), pp. 1105-1120\\n15'},\n",
              "  {'abstract': '© 2020, The Author(s). The inner nuclear membrane (INM) selectively accumulates proteins that are essential for nuclear functions; however, overaccumulation of INM proteins results in a range of rare genetic disorders. So far, little is known about how defective, mislocalized, or abnormally accumulated membrane proteins are actively removed from the INM, especially in plants and animals. Here, via analysis of a proximity-labeling proteomic profile of INM-associated proteins in Arabidopsis, we identify critical components for an INM protein degradation pathway. We show that this pathway relies on the CDC48 complex for INM protein extraction and 26S proteasome for subsequent protein degradation. Moreover, we show that CDC48 at the INM may be regulated by a subgroup of PUX proteins, which determine the substrate specificity or affect the ATPase activity of CDC48. These PUX proteins specifically associate with the nucleoskeleton underneath the INM and physically interact with CDC48 proteins to negatively regulate INM protein degradation in plants.',\n",
              "   'title': '160\\nProximity labeling proteomics reveals critical regulators for inner nuclear membrane protein degradation in plants\\nOpen Access\\nHuang, A., Tang, Y., Shi, X., (...), Chen, H., Gu, Y. 2020 Nature Communications\\n11(1),3284\\n0'},\n",
              "  {'abstract': 'Although they are widely distributed across kingdoms and are involved in a myriad of essential processes, until recently, repeat proteins have received little attention in comparison to globular proteins. As the name indicates, repeat proteins contain strings of tandem repeats of a basic structural element. In this respect, their construction is quite different from that of globular proteins, in which sequentially distant elements coalesce to form the protein. The different families of repeat proteins use their diverse scaffolds to present highly specific binding surfaces through which protein-protein interactions are mediated. Recent studies seek to understand the stability, folding and design of this important class of proteins.',\n",
              "   'title': '161\\nThe folding and design of repeat proteins: Reaching a consensus Main, E.R.G., Jackson, S.E., Regan, L. 2003 Current Opinion in Structural Biology\\n13(4), pp. 482-489\\n98'},\n",
              "  {'abstract': '© 2018 The Author(s) Mitochondria possess elaborate machineries for the import of proteins from the cytosol. Cytosolic factors like Hsp70 chaperones and their co-chaperones, the J-proteins, guide proteins to the mitochondrial surface. The translocase of the mitochondrial outer membrane (TOM) forms the entry gate for preproteins. How the proteins are delivered to mitochondrial preprotein receptors is poorly understood. We identify the cytosolic J-protein Xdj1 as a specific interaction partner of the central receptor Tom22. Tom22 recruits Xdj1 to the mitochondrial surface to promote import of preproteins and assembly of the TOM complex. Additionally, we find that the receptor Tom70 binds a different cytosolic J-protein, Djp1. Our findings suggest that cytosolic J-proteins target distinct TOM receptors and promote the biogenesis of mitochondrial proteins. Opaliński et al. report that mitochondrial protein import receptors selectively recognize J-protein co-chaperones of the cytosol. The co-chaperones bind hydrophobic precursor proteins and assist in transferring them to the receptors of the mitochondrial protein entry gate.',\n",
              "   'title': '162\\nRecruitment of Cytosolic J-Proteins by TOM Receptors Promotes Mitochondrial Protein Biogenesis\\nOpen Access\\nOpaliński, Ł., Song, J., Priesnitz, C., (...), Pfanner, N., Becker, T. 2018 Cell Reports\\n25(8), pp. 2036-2043.e5\\n25'},\n",
              "  {'abstract': '© Springer Science+Business Media New York 2014. Physical interactions between proteins are involved in many important cell functions and are key for understanding the mechanisms of biological processes. Protein–protein docking programs provide a means to computationally construct three-dimensional (3D) models of a protein complex structure from its component protein units. A protein docking program takes two or more individual 3D protein structures, which are either experimentally solved or computationally modeled, and outputs a series of probable complex structures. In this chapter we present the LZerD protein docking suite, which includes programs for pairwise docking, LZerD and PI-LZerD, and multiple protein docking, Multi-LZerD, developed by our group. PI-LZerD takes protein docking interface residues as additional input information. The methods use a combination of shape-based protein surface features as well as physics-based scoring terms to generate protein complex models. The programs are provided as stand-alone programs and can be downloaded from http://kiharalab.org/proteindocking .',\n",
              "   'title': '163\\nPairwise and multimeric protein–protein docking using the lzerd program suite Esquivel-Rodriguez, J., Filos-Gonzalez, V., Li, B., Kihara, D. 2014 Methods in Molecular Biology\\n1137, pp. 209-234\\n24'},\n",
              "  {'abstract': 'Inteins are protein elements that covalently reassemble proteins from two precursor fragments in a process known as protein splicing. They are commonly used to reassemble a single target protein by protein splicing, but a second target protein can potentially reassemble by intein dimerization. Here, we use the naturally occurring split DnaE intein from Nostoc punctiforme (NpuDnaE) to demonstrate the simultaneous assembly of two target proteins in several examples studied with live cell imaging: yellow fluorescent protein (YFP) with monomeric red fluorescent protein (mRFP), dominant positive mutant of RhoA GTPase with YFP and GCaMP2 Ca2+ indicator with mRFP. These examples showed the versatility of the strategy along with some interesting attributes: first, the two target proteins are in equal stoichiometry; second, the extent of protein splicing can be reported by a fluorescent protein. In particular, the split GCaMP2 with mRFP could find applications in tissue-specific Ca2+ imaging in transgenic organisms, where mRFP could control for motion-related intensity changes. © 2012 © The Author 2012.',\n",
              "   'title': '164\\nSimultaneous assembly of two target proteins using split inteins for live cell imaging\\nOpen Access\\nWong, S., Mills, E., Truong, K. 2013 Protein Engineering, Design and Selection\\n26(3), pp. 207-213\\n4'},\n",
              "  {'abstract': 'Five proteins of the perilipin (Plin) family such as Plin1 (perilipin) Plin2 (adipose differentiation-related protein), Plin3 (tail-interacting protein of 47 kDa), Plin4 (S3-12), and Plin5 (myocardial lipid droplet protein) are characterized as lipid droplet (LD) proteins in adipocytes. Recent reports have demonstrated that fat-specific protein 27 (FSP27) and hypoxia-inducible protein 2 (HIG2) are also thought to be novel LD proteins in addition to proteins of the Plin family. Growing evidence have shown that LD proteins play a role in the pathophysiology in the fatty liver disease which is characterized by hepatocytes containing LD with excessive neutral lipid. Studies showed LD proteins such as Plin1, Plin2, Plin3, Plin5, FSP27, and HIG2 are expressed in the liver steatosis. Among them, a high fat diet increases expression of Plin2 and/or FSP27 through activation of peroxisome proliferator-activated receptor γ to develop fatty liver. In this article, recent advances on the role of LD proteins in pathophysiology of fatty liver diseases are summarized.',\n",
              "   'title': '165\\nRole of lipid droplet proteins in liver steatosis. Okumura, T. 2011 Journal of physiology and biochemistry\\n67(4), pp. 629-636\\n56'},\n",
              "  {'abstract': 'Background: Identification of protein interacting sites is an important task in computational molecular biology. As more and more protein sequences are deposited without available structural information, it is strongly desirable to predict protein binding regions by their sequences alone. This paper presents a pattern mining approach to tackle this problem. It is observed that a functional region of protein structures usually consists of several peptide segments linked with large wildcard regions. Thus, the proposed mining technology considers large irregular gaps when growing patterns, in order to find the residues that are simultaneously conserved but largely separated on the sequences. A derived pattern is called a cluster-like pattern since the discovered conserved residues are always grouped into several blocks, which each corresponds to a local conserved region on the protein sequence. Results: The experiments conducted in this work demonstrate that the derived long patterns automatically discover the important residues that form one or several hot regions of protein-protein interactions. The methodology is evaluated by conducting experiments on the web server MAGIIC-PRO based on a well known benchmark containing 220 protein chains from 72 distinct complexes. Among the tested 218 proteins, there are 900 sequential blocks discovered, 4.25 blocks per protein chain on average. About 92% of the derived blocks are observed to be clustered in space with at least one of the other blocks, and about 66% of the blocks are found to be near the interface of protein-protein interactions. It is summarized that for about 83% of the tested proteins, at least two interacting blocks can be discovered by this approach. Conclusion: This work aims to demonstrate that the important residues associated with the interface of protein-protein interactions may be automatically discovered by sequential pattern mining. The detected regions possess high conservation and thus are considered as the computational hot regions. This information would be useful to characterizing protein sequences, predicting protein function, finding potential partners, and facilitating protein docking for drug discovery. © 2007 Hsu et al; licensee BioMed Central Ltd.',\n",
              "   'title': '166\\nIdentification of hot regions in protein-protein interactions by sequential pattern mining\\nOpen Access\\nHsu, C.-M., Chen, C.-Y., Liu, B.-J., (...), Lin, C.-C., Wu, T.-L. 2007 BMC Bioinformatics\\n8(SUPPL. 5),S8\\n38'},\n",
              "  {'abstract': 'The bacterial cell division machinery is organized in the so-called divisome composed of highly dynamic but low abundant interacting (membrane-bound) proteins. In order to elucidate the molecular interactions between these proteins, we developed a robust background-insensitive quantitative spectral unmixing method for estimating FRET efficiencies at near endogenous protein levels using fluorescent protein fusions. The assembly of the division machinery of Escherichia coli occurs in two steps that are discrete in time: first the FtsZ-ring and the so-called early localizing proteins that together seem to prepare the division assembly at midcell. Subsequently, the late localizing protein complexes that contain the peptidoglycan-synthesizing proteins PBP1B and FtsI (PBP3) are recruited to the division site, which initiates septation. Physical interactions were observed between members within each group but also between the early and late localizing proteins strongly suggesting that these proteins despite their differential localization in time are linked at the molecular and functional level. Interestingly, we find FtsN, one of the latest proteins in the divisome assembly, interacting with late assembling proteins FtsI and FtsW, but also with early (proto-ring) protein ZapA. This is in line with the recently described role of FtsN in divisome stabilization including the proto-ring elements. © 2010 Blackwell Publishing Ltd.',\n",
              "   'title': '167\\nDirect interactions of early and late assembling division proteins in Escherichia coli cells resolved by FRET Alexeeva, S., Gadella Jr., T.W.J., Verheul, J., Verhoeven, G.S., Den Blaauwen, T. 2010 Molecular Microbiology\\n77(2), pp. 384-398\\n64'},\n",
              "  {'abstract': 'Misfolded proteins are continuously produced in the cell and present an escalating detriment to cellular physiology if not managed effectively. As such, all organisms have evolved mechanisms to address misfolded proteins. One primary way eukaryotic cells handle the complication of misfolded proteins is by destroying them through the ubiquitin-proteasome system. To do this, eukaryotes possess specialized ubiquitin-protein ligases that have the capacity to recognize misfolded proteins over normally folded proteins. The strategies used by these Protein Quality Control (PQC) ligases to target the wide variety of misfolded proteins in the cell will likely be different than those used by ubiquitin-protein ligases that function in regulated degradation to target normally folded proteins. In this review, we highlight what is known about how misfolded proteins are recognized by PQC ubiquitin-protein ligases. © 2012 Elsevier Ltd.',\n",
              "   'title': '168\\nSelective destruction of abnormal proteins by ubiquitin-mediated protein quality control degradation Fredrickson, E.K., Gardner, R.G. 2012 Seminars in Cell and Developmental Biology\\n23(5), pp. 530-537\\n37'},\n",
              "  {'abstract': '© 2016 Bentham Science Publishers. Cancer cells evade apoptosis, which is regulated by proteins of Bcl-2 family in the intrinsic pathways. Numerous experimental three-dimensional (3D) structures of the apoptotic proteins and the proteins bound with small chemical molecules/peptides/proteins have been reported in the literature. In this review article, the 3D structures of the Bcl-2 family proteins from Homo sapiens and as well complex structures of the anti-apoptotic proteins bound with small molecular inhibitors reported in the literature to date have been comprehensively listed out and described in detail. Moreover, the molecular mechanisms by which the Bcl-2 family proteins modulate the apoptotic processes and strategies for designing antagonists to anti-apoptotic proteins have been concisely discussed.',\n",
              "   'title': '169\\nA review on structures and functions of Bcl-2 family proteins from Homo sapiens Sivakumar, D., Sivaraman, T. 2016 Protein and Peptide Letters\\n23(10), pp. 932-941\\n7'},\n",
              "  {'abstract': 'This paper explores the use of support vector machine (SVM) for protein function prediction. Studies are conducted on several groups of proteins with different functions including DNA-binding proteins, RNA-binding proteins, G-protein coupled receptors, drug absorption proteins, drug metabolizing enzymes, drug distribution and excretion proteins. The computed accuracy for the prediction of these proteins is found to be in the range of 82.32% to 99.7%, which illustrates the potential of SVM in facilitating protein function prediction. © 2008 Bentham Science Publishers Ltd.',\n",
              "   'title': '170\\nFunction prediction for DNA-/RNA-binding proteins, GPCRs, and drug ADME-associated proteins by SVM Cai, C., Xiao, H., Yuan, Q., Liu, X., Wen, Y. 2008 Protein and Peptide Letters\\n15(5), pp. 463-468\\n7'},\n",
              "  {'abstract': \"© 2016 The Authors Protein Science published by Wiley Periodicals, Inc. on behalf of The Protein Society Protein:protein interactions play key functional roles in the molecular machinery of the cell. A major challenge for structural biology is to gain high-resolution structural insight into how membrane protein function is regulated by protein:protein interactions. To this end we present a method to express, detect, and purify stable membrane protein complexes that are suitable for further structural characterization. Our approach utilizes bimolecular fluorescence complementation (BiFC), whereby each protein of an interaction pair is fused to nonfluorescent fragments of yellow fluorescent protein (YFP) that combine and mature as the complex is formed. YFP thus facilitates the visualization of protein:protein interactions in vivo, stabilizes the assembled complex, and provides a fluorescent marker during purification. This technique is validated by observing the formation of stable homotetramers of human aquaporin 0 (AQP0). The method's broader applicability is demonstrated by visualizing the interactions of AQP0 and human aquaporin 1 (AQP1) with the cytoplasmic regulatory protein calmodulin (CaM). The dependence of the AQP0-CaM complex on the AQP0 C-terminus is also demonstrated since the C-terminal truncated construct provides a negative control. This screening approach may therefore facilitate the production and purification of membrane protein:protein complexes for later structural studies by X-ray crystallography or single particle electron microscopy.\",\n",
              "   'title': '171\\nApplying bimolecular fluorescence complementation to screen and purify aquaporin protein:protein complexes\\nOpen Access\\nSjöhamn, J., Båth, P., Neutze, R., Hedfalk, K. 2016 Protein Science\\n25(12), pp. 2196-2208\\n5'},\n",
              "  {'abstract': 'Protein complexes mediated by protein-protein interactions are essential for many cellular functions. Transforming growth factor (TGF)-β signaling involves a cascade of protein-protein interactions and malfunctioning of this pathway has been implicated in human diseases. Using an in silico approach, we analyzed the naturally occurring human genetic variations from the proteins involved in the TGF-β signaling (10 TGF-β proteins and 242 other proteins interacting with them) to identify the ones that have potential biological consequences. All proteins were searched in the dbSNP database for the presence of nonsynonymous single nucleotide polymorphisms (nsSNPs). A total of 118 validated nsSNPs from 63 proteins were retrieved and analyzed in terms of 1) evolutionary conservation status, 2) being located in a functional protein domain or motif, and 3) altering putative protein motif or phosphorylation sites. Our results indicated the presence of 31 nsSNPs that occurred at evolutionarily conserved residues, 37 nsSNPs were located in protein domains, motifs, or repeats, and 46 nsSNPs were predicted to either create or abolish putative protein motifs or phosphorylation sites. We undertook this study to analyze the human genetic variations that can affect the protein function and the TGF-β signaling. The nsSNPs reported in here can be characterized by experimental approaches to elucidate their exact biological roles and whether they are related to human disease. Copyright © 2007 the American Physiological Society.',\n",
              "   'title': '172\\nFunctional nonsynonymous single nucleotide polymorphisms from the TGF-β protein interaction network Savas, S., Taylor, I.W., Wrana, J.L., Ozcelik, H. 2007 Physiological Genomics\\n29(2), pp. 109-117\\n10'},\n",
              "  {'abstract': '© 2017 Elsevier B.V. Calmodulin (CaM) is a universal regulator for a huge number of proteins in all eukaryotic cells. Best known is its function as a calcium-dependent modulator of the activity of enzymes, such as protein kinases and phosphatases, as well as other signaling proteins including membrane receptors, channels and structural proteins. However, less well known is the fact that CaM can also function as a Ca2 +-dependent adaptor protein, either by bridging between different domains of the same protein or by linking two identical or different target proteins together. These activities are possible due to the fact that CaM contains two independently-folded Ca2 + binding lobes that are able to interact differentially and to some degree separately with targets proteins. In addition, CaM can interact with and regulates several proteins that function exclusively as adaptors. This review provides an overview over our present knowledge concerning the structural and functional aspects of the role of CaM as an adaptor protein and as a regulator of known adaptor/scaffold proteins.',\n",
              "   'title': '173\\nCalmodulin as a protein linker and a regulator of adaptor/scaffold proteins\\nOpen Access\\nVillalobo, A., Ishida, H., Vogel, H.J., Berchtold, M.W. 2018 Biochimica et Biophysica Acta - Molecular Cell Research\\n1865(3), pp. 507-521\\n24'},\n",
              "  {'abstract': 'During overexpression of recombinant proteins in Escherichia coli, misfolded proteins often aggregate and form inclusion bodies. If an aggregation-prone recombinant protein is fused upstream (as an N-terminal fusion) to GFP, aggregation of the recombinant protein domain also leads to misfolding of the downstream GFP domain, resulting in a decrease or loss of fluorescence. We investigated whether the GFP domain could fold correctly if aggregation of the upstream protein domain was prevented in vivo by a coupled protein folding and binding interaction. Such interaction has been previously shown to occur between the E. coli integration host factors α and β, and between the domains of the general transcriptional coactivator cAMP response element binding protein (CREB)-binding protein and the activator for thyroid hormone and retinoid receptors. In this study, fusion of integration host factor β or the CREB-binding protein domain upstream to GFP resulted in aggregation of the fusion protein. Coexpression of their respective partners, on the other hand, allowed soluble expression of the fusion protein and a dramatic increase in fluorescence. The study demonstrated that coupled protein folding and binding could be correlated to GFP fluorescence. A modified miniintein containing an affinity tag was inserted between the upstream protein domain and GFP to allow rapid purification and identification of the heterodimeric complex. The GFP coexpression fusion system may be used to identify novel protein-protein interactions that involve coupled folding and binding or protein partners that can solubilize aggregation-prone recombinant proteins.',\n",
              "   'title': '174\\nVisualization of coupled protein folding and binding in bacteria and purification of the heterodimeric complex\\nOpen Access\\nWang, H., Chong, S. 2003 Proceedings of the National Academy of Sciences of the United States of America\\n100(2), pp. 478-483\\n36'},\n",
              "  {'abstract': 'Many methods commonly used to identify and characterize interactions between two or more proteins are variations for the immobilized protein-protein interaction assay (for example, glutathione S-transferase (GST) pulldown and coimmunoprecipitation). A potential, and often overlooked, problem with these assays is the possibility that an observed interaction is mediated not by direct contact between proteins, but instead by nucleic acid contaminating the protein preparations. As a negatively charged polymer, nucleic acid (often cellular RNA) can adhere to basic surfaces on proteins, and thereby mediate interactions between an immobilized bait protein and a target protein. The contaminating nucleic acid may cause a false positive result in protein-protein interaction assays or may contribute to general background. Alternatively, in relatively rare cases, the presence of nonspecific nucleic acid can inhibit protein-protein interaction. In general contaminating nucleic acid can be especially problematic with proteins under study that naturally bind RNA or DNA (for example, trascription factors), although nucleic acid can mediate apparent protein-protein interactions in other systems as well. A simple and convenient method for decreasing false positive and background owing to contaminating nucleic acid is to treat the protein preparations with micrococcal nuclease. Micrococcal nuclease (also known as S7 nuclease) cleaves single- and double-stranded DNA and RNA with no sequence specificity. This protocol describes as GST pulldown assay that incorporates micrococcal nuclease treatment of both the immobilized GST-bait protein and the target protein preparations. The strategy can be adapted easily to proteins with other tags and immobilized using other methods. The target protein can be derived from many different sources - for example, a cell extract containing a native protein of interest; a recombinant protein expressed in Escherichia coli, insect cells or mammalian cells; a protein produced in an in vivo translation system; or a highly purified protein. The steps required to treat protein preparations with micrococcal nuclease are straightforwarded and can be incorporated into any immobilized protein-protein interaction assay. © 2006 Nature Publishing Group.',\n",
              "   'title': '175\\nProtein-protein interaction assays: Eliminating false positive interactions Nguyen, T.N., Goodrich, J.A. 2006 Nature Methods\\n3(2), pp. 135-139\\n62'},\n",
              "  {'abstract': 'A signature feature of all living organisms is their utilization of proteins to construct molecular machineries that undertake the complex network of cellular activities. The abundance of a protein element is temporally and spatially regulated in two opposing aspects: de novo synthesis to manufacture the required amount of the protein, and destruction of the protein when it is in excess or no longer needed. One major route of protein destruction is coordinated by a set of conserved molecules, the F-box proteins, which promote ubiquitination in the ubiquitin-proteasome pathway. Here we discuss the functions of F-box proteins in several cellular scenarios including cell cycle progression, synapse formation, plant hormone responses, and the circadian clock. We particularly emphasize the mechanisms whereby F-box proteins recruit specific substrates and regulate their abundance in the context of SCF E3 ligases. For some exceptions, we also review how F-box proteins function through non-SCF mechanisms. © 2008 Birkhaueser.',\n",
              "   'title': '176\\nThe utility F-box for protein destruction Ho, M.S., Ou, C., Chan, Y.-R., Chien, C.-T., Pi, H. 2008 Cellular and Molecular Life Sciences\\n65(13), pp. 1977-2000\\n102'},\n",
              "  {'abstract': '© 2016 Elsevier Ltd Protein–protein interaction is one of the key events in the signal transduction pathway. The interaction changes the conformations, activities, localization and stabilities of the proteins, and transduces the signal to the next step. Frequently, this interaction occurs upon the protein phosphorylation. When upstream signals are stimulated, protein kinase(s) is/are activated and phosphorylate(s) their substrates, and induce the phosphorylation dependent protein–protein interaction. For this interaction, several domains in proteins are known to specifically recognize the phosphorylated residues of target proteins. These specific domains for interaction are important in the progression of the diseases caused by disordered signal transduction such as cancer. Thus small molecules that modulate this interaction are attractive lead compounds for the treatment of such diseases. In this review, we focused on three examples of phosphorylation dependent protein–protein interaction modules (14-3-3, polo box domain of Plk1 and F-box proteins in SCF ubiquitin ligases) and summarize small molecules that modulate their interaction. We also introduce our original screening system to identify such small molecules.',\n",
              "   'title': '177\\nSmall molecules that target phosphorylation dependent protein–protein interaction Watanabe, N., Osada, H. 2016 Bioorganic and Medicinal Chemistry\\n24(15), pp. 3246-3254\\n11'},\n",
              "  {'abstract': '© 2015 Wiley Periodicals, Inc. Significant progress has been made in the determination of the protein structures with their number today passing over a hundred thousand structures. The next challenge is the understanding and prediction of protein-protein and protein-ligand interactions. In this work we address this problem by analyzing curved solenoid proteins. Many of these proteins are considered as \"hub molecules\" for their high potential to interact with many different molecules and to be a scaffold for multisubunit protein machineries. Our analysis of these structures through molecular dynamics simulations reveals that the mobility of the side-chains on the concave surfaces of the solenoids is lower than on the convex ones. This result provides an explanation to the observed preferential binding of the ligands, including small and flexible ligands, to the concave surface of the curved solenoid proteins. The relationship between the landscapes and dynamic properties of the protein surfaces can be further generalized to the other types of protein structures and eventually used in the computer algorithms, allowing prediction of protein-ligand interactions by analysis of protein surfaces.',\n",
              "   'title': '178\\nRestricted mobility of side chains on concave surfaces of solenoid proteins may impart heightened potential for intermolecular interactions Ramya, L., Gautham, N., Chaloin, L., Kajava, A.V. 2015 Proteins: Structure, Function and Bioinformatics\\n83(9), pp. 1654-1664\\n0'},\n",
              "  {'abstract': 'Spinal cord injury reduces the rate of skeletal muscle protein synthesis and increases protein breakdown, resulting in rapid muscle loss. The purpose of this study was to determine whether long-term paraplegia would eventually result in a downregulation of muscle mRNA and protein expression associated with both protein synthesis and breakdown. After 10 weeks of spinal cord transection, soleus muscle from 12 rats (6 sham-control, 6 paraplegic) was studied for mRNAs and proteins associated with protein synthesis and breakdown using real-time polymerase chain reaction and immunoblotting techniques. Protein kinase B (PKB/Akt), ribosomal S6 kinase 1 (S6K1), and myogenin mRNA were downregulated, whereas muscle ring finger 1 (MuRF1) and phospho-forkhead transcription factor 4 (FoxO4) protein were increased in paraplegic rats. We conclude that gene and protein expression of pathways associated with protein synthesis are reduced, whereas some markers of protein breakdown remain elevated following chronic paraplegia. Clinical interventions designed to increase muscle protein synthesis may be helpful in preventing excessive muscle loss during long-term paraplegia. © 2008 Wiley Periodicals, Inc.',\n",
              "   'title': '179\\nGene and protein expression associated with protein synthesis and breakdown in paraplegic skeletal muscle Drummond, M.J., Glynn, E.L., Lujan, H.L., Dicarlo, S.E., Rasmussen, B.B. 2008 Muscle and Nerve\\n37(4), pp. 505-513\\n26'},\n",
              "  {'abstract': 'In order to obtain a polyclonal antibody that recognizes various protein kinases, a peptide corresponding to an amino acid sequence of a highly conserved subdomain (subdomain VIB) of the protein kinase family was synthesized and used for immunization. When the synthetic peptide, CVVHRDLKPENLLLAS, was coupled to keyhole limpet hemocyanin (KLH) and used to immunize rabbits, polyclonal antibodies that detected multiple protein kinases on a Western blot were generated. One of the antibodies obtained, KI98, detected a variety of purified Ser/Thr protein kinases, such as calmodulin-dependent protein kinase II (CaM-kinase II), calmodulin-dependent protein kinase IV (CaM-kinase IV), cAMP-dependent protein kinase, protein kinase C, and Erk2. The antibody detected as low as 0.2 ng of protein kinases blotted onto a nitrocellulose membrane by dot-immunobinding assay. When a rat brain extract was analyzed with this antibody, various protein kinases were simultaneously detected. The present anti-peptide antibody with a broad spectrum of cross-reactivity to multiple protein kinases may be a powerful tool for comprehensive analysis focused on protein kinases. © 2004 Elsevier B.V. All rights reserved.',\n",
              "   'title': '180\\nGeneration of a polyclonal antibody that simultaneously detects multiple Ser/Thr protein kinases Kameshita, I., Kinoshita, S., Shigeri, Y., (...), Yumoto, N., Ishida, A. 2004 Journal of Biochemical and Biophysical Methods\\n60(1), pp. 13-22\\n11'},\n",
              "  {'abstract': 'For decades, protein function was intimately linked to the presence of a unique, aperiodic crystal-like structure in a functional protein. The two only places for conformational ensembles of under-folded (or partially folded) protein forms in this picture were either the end points of the protein denaturation processes or transiently populated folding intermediates. Recent years witnessed dramatic change in this perception and conformational ensembles, which the under-folded proteins are, have moved from the shadow. Accumulated to date data suggest that a protein can exist in at least three global forms-functional and folded, functional and intrinsically disordered (nonfolded), and nonfunctional and misfolded/aggregated. Under-folded protein states are crucial for each of these forms, serving as important folding intermediates of ordered proteins, or as functional states of intrinsically disordered proteins (IDPs) and IDP regions (IDPRs), or as pathology triggers of misfolded proteins. Based on these observations, conformational ensembles of under-folded proteins can be classified as transient (folding and misfolding intermediates) and permanent (IDPs and stable misfolded proteins). Permanently under-folded proteins can further be split into intentionally designed (IDPs and IDPRs) and unintentionally designed (misfolded proteins). Although intrinsic flexibility, dynamics, and pliability are crucial for all under-folded proteins, the different categories of under-foldedness are differently encoded in protein amino acid sequences. © 2013 Wiley Periodicals, Inc. Biopolymers 99: 870-887, 2013. Copyright © 2013 Wiley Periodicals, Inc.',\n",
              "   'title': '181\\nUnder-folded proteins: Conformational ensembles and their roles in protein folding, function, and pathogenesis Uversky, V.N. 2013 Biopolymers\\n99(11), pp. 870-887\\n31'},\n",
              "  {'abstract': 'Regulator of G-protein-signaling (RGS) proteins play a key role in the regulation of G-protein-coupled receptor (GPCR) signaling. The characteristic hallmark of RGS proteins is a conserved ≈120-aa RGS region that confers on these proteins the ability to serve as GTPase-activating proteins (GAPs) for Gα proteins. Most RGS proteins can serve as GAPs for multiple isoforms of Gα and therefore have the potential to influence many cellular signaling pathways. However, RGS proteins can be highly regulated and can demonstrate extreme specificity for a particular signaling pathway. RGS proteins can be regulated by altering their GAP activity or subcellular localization; such regulation is achieved by phosphorylation, palmitoylation, and interaction with protein and lipid-binding partners. Many RGS proteins have GAP-independent functions that influence GPCR and non-GPCR-mediated signaling, such as effector regulation or action as an effector. Hence, RGS proteins should be considered multifunctional signaling regulators. GPCR-mediated signaling is critical for normal function in the cardiovascular system and is currently the primary target for the pharmacological treatment of disease. Alterations in RGS protein levels, in particular RGS2 and RGS4, produce cardiovascular phenotypes. Thus, because of the importance of GPCR-signaling pathways and the profound influence of RGS proteins on these pathways, RGS proteins are regulators of cardiovascular physiology and potentially novel drug targets as well. © 2005 American Heart Association, Inc.',\n",
              "   'title': '182\\nMulti-tasking RGS proteins in the heart: The next therapeutic target?\\nOpen Access\\nRiddle, E.L., Schwartzman, R.A., Bond, M., Insel, P.A. 2005 Circulation Research\\n96(4), pp. 401-411\\n99'},\n",
              "  {'abstract': 'The pathogenesis of neurodegenerative diseases is believed to involve abnormal aggregation of proteins, but the mechanisms initiating protein aggregation are undear. Here we report a novel phenomenon that could be instrumental in triggering protein aggregation in neurodegenerative diseases. We show that the 3′ untranslated region (3′UTR) of a light neurofilament (NF-L) transcript enhances the reactivity of its own translated product and leads to loss of solubility and aggregation of NF-L protein and to coaggregation of mutant superoxide dismutase 1 (SOD1) protein. Full-length mouse NF-L cDNAs, with and without NF-L 3′UTR, were fused to the C terminus of a green fluorescent protein (GFP) reporter gene, and the GFP-tagged NF-L proteins were examined in transfected Neuro2a cells. The GFP-tagged NF-L protein expressed from the transgene containing NF-L 3′UTR, but not from the transgene lacking NF-L 3′UTR, colocalizes with endogenous heavy neurofilament protein and, at high-level expression, leads to loss of solubility and aggregation of GFP-tagged NF-L protein. Aggregation of GFP-tagged NF-L protein triggers coaggregation and loss of solubility of coexpressed DsRed-tagged mutant (G93A) SOD1 protein but not wild-type SOD1 protein. Deletional mutagenesis maps the RNA sequence causing aggregation of GFP-tagged NF-L protein to the proximal 45 nucleotides of NF-L 3′UTR. This is the site of a major destabilizing element in NF-L RNA and binding site for RNA-binding proteins. Our findings support a working model whereby NF-L RNA, or cognate RNA-binding factors, enhances the reactivity of NF-L protein and provides a triggering mechanism leading to aggregation of NF-L and other proteins in neurodegenerative diseases.',\n",
              "   'title': '183\\n3′ Untranslated Region in A Light Neurofilament (NF-L) mRNA Triggers Aggregation of NF-L and Mutant Superoxide Dismutase 1 Proteins in Neuronal Cells\\nOpen Access\\nLin, H., Zhai, J., Cañete-Soler, R., Schlaepfer, W.W. 2004 Journal of Neuroscience\\n24(11), pp. 2716-2726\\n22'},\n",
              "  {'abstract': '© Springer Science+Business Media New York 2015. It has become clear in recent years that the Gβγ subunits of heterotrimeric proteins serve broad roles in the regulation of cellular activity and interact with many proteins in different subcellular locations including the nucleus. Protein affi nity purifi cation is a common method to identify and confi rm protein interactions. When used in conjugation with mass spectrometry it can be used to identify novel protein interactions with a given bait protein. The tandem affi nity purifi cation (TAP) technique identifi es partner proteins bound to tagged protein bait. Combined with protocols to enrich the nuclear fraction of whole cell lysate through sucrose cushions, TAP allows for purifi cation of interacting proteins found specifi cally in the nucleus. Here we describe the use of the TAP technique on cytosolic and nuclear lysates to identify candidate proteins, through mass spectrometry, that bind to Gβ 1 subunits.',\n",
              "   'title': '184\\nTandem affi nity purifi cation to identify cytosolic and nuclear gβγ-interacting proteins Campden, R., Pétrin, D., Robitaille, M., (...), Angers, S., Hébert, T.E. 2015 Methods in Molecular Biology\\n1234, pp. 161-184\\n3'},\n",
              "  {'abstract': 'Small heat shock proteins are ubiquitous proteins found throughout all kingdoms. One of the most notable features is their large oligomeric structures with conserved structural organization. It is well documented that small heat shock proteins can capture unfolding proteins to form stable complexes and prevent their irreversible aggregation. In addition, small heat shock proteins coaggregate with aggregation-prone proteins for subsequent, efficient disaggregation of the protein aggregates. The release of substrate proteins from the transient reservoirs, i.e. complexes and aggregates with small heat shock proteins, and their refolding require cooperation with ATP-dependent chaperone systems. The amphitropic small heat shock proteins were shown to associate with membranes, although they do not contain transmembrane domains or signal sequences. Recent studies indicate that small heat shock proteins play an important role in membrane quality control and thereby potentially contribute to the maintenance of membrane integrity especially under stress conditions. © 2006 Birkhäuser Verlag.',\n",
              "   'title': '185\\nThe small heat shock proteins and their clients Nakamoto, H., Vígh, L. 2007 Cellular and Molecular Life Sciences\\n64(3), pp. 294-306\\n218'},\n",
              "  {'abstract': 'Proteins with expanded polyglutamine (polyQ) tracts have been linked to neurodegenerative diseases. One common characteristic of expanded-polyQ expression is the formation of intracellular aggregates (IAs). IAs purified from polyQ-expressing cells were dissociated and studied by protein blot assay and mass spectrometry to determine the identity, condition, and relative level of several proteins sequestered within aggregates. Most of the sequestered proteins comigrated with bands from control extracts, indicating that the sequestered proteins were intact and not irreversibly bound to the polyQ polymer. Among the proteins found sequestered at relatively high levels in purified IAs were ubiquitin, the cell cycle-regulating proteins p53 and mdm-2, HSP70, the global transcriptional regulator Tata-binding protein/TFIID, cytoskeleton proteins actin and 68-kD neurofilament, and proteins of the nuclear pore complex. These data reveal that IAs are highly complex structures with a multiplicity of contributing proteins.',\n",
              "   'title': '186\\nIdentities of sequestered proteins in aggregates from cells with induced polyglutamine expression\\nOpen Access\\nSuhr, S.T., Senut, M.-C., Whitelegge, J.P., (...), Cuizon, D.B., Gage, F.H. 2001 Journal of Cell Biology\\n153(2), pp. 283-294\\n169'},\n",
              "  {'abstract': '© 2015 by The American Society for Biochemistry and Molecular Biology, Inc. RNA editing in plastids and mitochondria of flowering plants requires pentatricopeptide repeat proteins (PPR proteins) for site recognition and proteins of the multiple organellar RNA editing factor (MORF) family as cofactors. Two MORF proteins, MORF5 and MORF8, are dual-targeted to plastids and mitochondria; two are targeted to plastids, and five are targeted to mitochondria. Pulldown assays from Arabidopsis thaliana tissue culture extracts with the mitochondrial MORF1 and the plastid MORF2 proteins, respectively, both identify the dual-targeted MORF8 protein, showing that these complexes can assemble in the organelles. We have now determined the scope of potential interactions between the various MORF proteins by yeast two-hybrid, in vitro pulldown, and bimolecular fluorescence complementation assays. The resulting MORF-MORF interactome identifies specific heteromeric MORF protein interactions in plastids and in mitochondria. Heteromers are observed for MORF protein combinations affecting a common site, suggesting their functional relevance. Most MORF proteins also undergo homomeric interactions. Submolecular analysis of the MORF1 protein reveals that the MORF-MORF protein connections require the C-terminal region of the central conserved MORF box. This domain has no similarity to known protein modules and may form a novel surface for protein-protein interactions.',\n",
              "   'title': '187\\nSelective homo- and heteromer interactions between the multiple organellar RNA editing factor (MORF) proteins in Arabidopsis thaliana\\nOpen Access\\nZehrmann, A., Härtel, B., Glass, F., (...), Brennicke, A., Takenaka, M. 2015 Journal of Biological Chemistry\\n290(10), pp. 6445-6456\\n29'},\n",
              "  {'abstract': 'We have developed a surface plasmon resonance (SPR)-based protein microarray to study protein-protein interactions in a high-throughput mode. As a model system, triple protein interactions have been explored with human papillomaviral E6 protein, tumor suppressor p53, and ubiquitin ligase E6AP. Human papillomavirus (HPV) is known to be a causative agent of cervical cancer. Upon infection, the viral E6 protein forms a heterotrimeric protein complex with p53 and E6AP. The formation of the complex eventually results in the degradation of p53. In the present study, a GST-fused E6AP protein was layered onto a glutathione (GSH)-modified gold chip surface. The specific binding of GST-E6AP protein onto the gold chip surface was facilitated through the affinity of GST to its specific ligand GSH. The interacting proteins (E6 and/or p53) were then spotted. Detection of the interaction was performed using a SPR imaging (SPRI) technique. The resulting SPRI intensity data showed that the protein-protein interactions of E6AP, E6, and p53 were detected in a concentration-dependent manner, suggesting that the SPRI-based microarray system can be an effective tool to study protein-protein interactions where multiple proteins are involved. © 2006 Wiley-VCH Verlag GmbH & Co. KGaA.',\n",
              "   'title': '188\\nSurface plasmon resonance imaging protein arrays for analysis of triple protein interactions of HPV, E6, E6AP, and p53 Ro, H.-S., Koh, B.H., Jung, S.O., (...), Kim, M.-G., Chung, B.H. 2006 Proteomics\\n6(7), pp. 2108-2111\\n31'},\n",
              "  {'abstract': 'A long-standing goal of computational protein design is to create proteins similar to those found in Nature. One motivation is to harness the exquisite functional capabilities of proteins for our own purposes. The extent of similarity between designed and natural proteins also reports on how faithfully our models represent the selective pressures that determine protein sequences. As the field of protein design shifts emphasis from reproducing native-like protein structure to function, it has become important that these models treat the notion of specificity in molecular interactions. Although specificity may, in some cases, be achieved by optimization of a desired protein in isolation, methods have been developed to address directly the desire for proteins that exhibit specific functions and interactions. © 2010 by The American Society for Biochemistry and Molecular Biology, Inc.',\n",
              "   'title': '189\\nSpecificity in computational protein design\\nOpen Access\\nHavranek, J.J. 2010 Journal of Biological Chemistry\\n285(41), pp. 31095-31099\\n14'},\n",
              "  {'abstract': 'Background: Bacillus subtilis is a favorable host for the production of industrially relevant proteins because of its capacity of secreting proteins into the medium to high levels, its GRAS (Generally Recognized As Safe) status, its genetic accessibility and its capacity to grow in large fermentations. However, production of heterologous proteins still faces limitations.Results: This study aimed at the identification of bottlenecks in secretory protein production by analyzing the response of B. subtilis at the transcriptome level to overproduction of eight secretory proteins of endogenous and heterologous origin and with different subcellular or extracellular destination: secreted proteins (NprE and XynA of B. subtilis, Usp45 of Lactococcus lactis, TEM-1 β-lactamase of Escherichia coli), membrane proteins (LmrA of L. lactis and XylP of Lactobacillus pentosus) and lipoproteins (MntA and YcdH of B. subtilis). Responses specific for proteins with a common localization as well as more general stress responses were observed. The latter include upregulation of genes encoding intracellular stress proteins (groES/EL, CtsR regulated genes). Specific responses include upregulation of the liaIHGFSR operon under Usp45 and TEM-1 β-lactamase overproduction; cssRS, htrA and htrB under all secreted proteins overproduction; sigW and SigW-regulated genes mainly under membrane proteins overproduction; and ykrL (encoding an HtpX homologue) specifically under membrane proteins overproduction.Conclusions: The results give better insights into B. subtilis responses to protein overproduction stress and provide potential targets for genetic engineering in order to further improve B. subtilis as a protein production host. © 2012 Marciniak et al.; licensee BioMed Central Ltd.',\n",
              "   'title': '190\\nComparative transcriptional analysis of Bacillus subtilis cells overproducing either secreted proteins, lipoproteins or membrane proteins\\nOpen Access\\nMarciniak, B.C., Trip, H., van-der Veek, P.J., Kuipers, O.P. 2012 Microbial Cell Factories\\n11,66\\n12'},\n",
              "  {'abstract': 'Coregulator proteins (CoRegs) are part of multi-protein complexes that transiently assemble with transcription factors and chromatin modifiers to regulate gene expression. In this study we analyzed data from 3,290 immuno-precipitations (IP) followed by mass spectrometry (MS) applied to human cell lines aimed at identifying CoRegs complexes. Using the semi-quantitative spectral counts, we scored binary protein-protein and domain-domain associations with several equations. Unlike previous applications, our methods scored prey-prey protein-protein interactions regardless of the baits used. We also predicted domain-domain interactions underlying predicted protein-protein interactions. The quality of predicted protein-protein and domain-domain interactions was evaluated using known binary interactions from the literature, whereas one protein-protein interaction, between STRN and CTTNBP2NL, was validated experimentally; and one domain-domain interaction, between the HEAT domain of PPP2R1A and the Pkinase domain of STK25, was validated using molecular docking simulations. The scoring schemes presented here recovered known, and predicted many new, complexes, protein-protein, and domain-domain interactions. The networks that resulted from the predictions are provided as a web-based interactive application at http://maayanlab.net/HT-IP-MS-2-PPI-DDI/. © 2011 Mazloom et al.',\n",
              "   'title': \"191\\nRecovering protein-protein and domain-domain interactions from aggregation of IP-MS proteomics of coregulator complexes\\nOpen Access\\nMazloom, A.R., Dannenfelser, R., Clark, N.R., (...), Lanz, R.B., Ma'ayan, A. 2011 PLoS Computational Biology\\n7(12),e1002319\\n13\"},\n",
              "  {'abstract': 'Due to the pivotal role of membrane proteins in many cellular processes, their direct link to human disease and their often extracellular accessibility towards drugs, an understanding of membrane protein function is desirable. However, the hydrophobic nature of membrane proteins often results in insoluble proteins which makes protein isolation difficult and therefore hinders the determination of protein complex composition and protein function. Recently, several yeast genetic techniques have made the characterisation of interactions among membrane proteins more feasible. Techniques such as the guanine-nucleotide binding protein fusion assay, the reverse Ras recruitment system and the split-ubiquitin system have been fruitful in monitoring known protein interactions and uncovering novel interactions. Since many disease states have altered membrane protein function, one can use these systems to recreate interactions involving disease causing membrane proteins. Once established, screens for small molecules, peptides and/or single chain antibodies which disrupt such interactions can provide insight into the biology of the interaction and thus help guide therapeutical research. In this review, we speculate on the feasibility of using inhibitors of protein interactions as drugs and the adaptation of these techniques to select for inhibitors of defined protein interactions.',\n",
              "   'title': '192\\nYeast Genetic Methods for the Detection of Membrane Protein Interactions: Potential Use in Drug Discovery Fetchko, M., Auerbach, D., Stagljar, I. 2003 BioDrugs\\n17(6), pp. 413-424\\n15'},\n",
              "  {'abstract': 'OBJECTIVE - Several platelet proteins are palmitoylated, but whether protein palmitoylation functions in platelet activation is unknown. We sought to determine the role of platelet protein palmitoylation in platelet activation and thrombus formation. METHODS AND RESULTS - Platelet proteins were depalmitoylated by infusing acyl-protein thioesterase 1 into permeabilized platelets. In intact platelets, platelet protein palmitoylation was blocked using the protein palmitoylation inhibitor cerulein. The effects of inhibiting platelet protein palmitoylation on platelet function and on thrombus formation in vivo were evaluated. When infused into permeabilized platelets, acyl-protein thioesterase 1 reduced total platelet protein palmitoylation and inhibited protease-activated receptor-1-mediated α-granule secretion with an IC50 of 175 nmol/L and maximal inhibition of ≥90%. Gαq and SNAP-23, membrane-associated proteins that are constitutively palmitoylated, translocated to the cytosol when permeabilized platelets were exposed to recombinant acyl-protein thioesterase 1. The protein palmitoylation inhibitor cerulein also inhibited platelet granule secretion and aggregation. Studies using intravital microscopy showed that incubation with cerulein decreased the rate of platelet accumulation into thrombi formed after laser-induced injury of mouse arterioles and inhibited maximal platelet accumulation by >60%. CONCLUSION - These studies show that platelets possess a protein palmitoylation machinery that is required for both platelet activation and platelet accumulation into thrombi. These studies show that inhibition of platelet protein palmitoylation blocks platelet aggregation and granule secretion. In a murine model of thrombus formation, inhibition of protein palmitoylation markedly inhibits platelet accumulation into thrombi at sites of vascular injury. © 2007 American Heart Association, Inc.',\n",
              "   'title': '193\\nPlatelets possess and require an active protein palmitoylation pathway for agonist-mediated activation and in vivo thrombus formation\\nOpen Access\\nSim, D.S., Dilks, J.R., Flaumenhaft, R. 2007 Arteriosclerosis, Thrombosis, and Vascular Biology\\n27(6), pp. 1478-1485\\n27'},\n",
              "  {'abstract': '© 2017 The Author(s). Background: The Plasmodium genome encodes for a number of 6-Cys proteins that contain a module of six cysteine residues forming three intramolecular disulphide bonds. These proteins have been well characterized at transmission as well as hepatic stages of the parasite life cycle. In the present study, a large complex of 6-Cys proteins: Pfs41, Pfs38 and Pfs12 and three other merozoite surface proteins: Glutamate-rich protein (GLURP), SERA5 and MSP-1 were identified on the Plasmodium falciparum merozoite surface. Methods: Recombinant 6-cys proteins i.e. Pfs38, Pfs12, Pfs41 as well as PfMSP-165 were expressed and purified using Escherichia coli expression system and antibodies were raised against each of these proteins. These antibodies were used to immunoprecipitate the native proteins and their associated partners from parasite lysate. ELISA, Far western, surface plasmon resonance and glycerol density gradient fractionation were carried out to confirm the respective interactions. Furthermore, erythrocyte binding assay with 6-cys proteins were undertaken to find out their possible role in host-parasite infection and seropositivity was assessed using Indian and Liberian sera. Results: Immunoprecipitation of parasite-derived polypeptides, followed by LC-MS/MS analysis, identified a large Pfs38 complex comprising of 6-cys proteins: Pfs41, Pfs38, Pfs12 and other merozoite surface proteins: GLURP, SERA5 and MSP-1. The existence of such a complex was further corroborated by several protein-protein interaction tools, co-localization and co-sedimentation analysis. Pfs38 protein of Pfs38 complex binds to host red blood cells (RBCs) directly via glycophorin A as a receptor. Seroprevalence analysis showed that of the six antigens, prevalence varied from 40 to 99%, being generally highest for MSP-165 and GLURP proteins. Conclusions: Together the data show the presence of a large Pfs38 protein-associated complex on the parasite surface which is involved in RBC binding. These results highlight the complex molecular interactions among the P. falciparum merozoite surface proteins and advocate the development of a multi-sub-unit malaria vaccine based on some of these protein complexes on merozoite surface.',\n",
              "   'title': '194\\nA novel Pfs38 protein complex on the surface of Plasmodium falciparum blood-stage merozoites\\nOpen Access\\nPaul, G., Deshmukh, A., Kaur, I., (...), Theisen, M., Malhotra, P. 2017 Malaria Journal\\n16(1),79\\n5'},\n",
              "  {'abstract': 'Protein molecules interact with each other in protein complexes to perform many vital functions, and different computational techniques have been developed to identify protein complexes in protein-protein interaction (PPI) networks. These techniques are developed to search for subgraphs of high connectivity in PPI networks under the assumption that the proteins in a protein complex are highly interconnected. While these techniques have been shown to be quite effective, it is also possible that the matching rate between the protein complexes they discover and those that are previously determined experimentally be relatively low and the \"false-alarm\" rate can be relatively high. This is especially the case when the assumption of proteins in protein complexes being more highly interconnected be relatively invalid. To increase the matching rate and reduce the false-alarm rate, we have developed a technique that can work effectively without having to make this assumption. The name of the technique called protein complex identification by discovering functional interdependence (PCIFI) searches for protein complexes in PPI networks by taking into consideration both the functional interdependence relationship between protein molecules and the network topology of the network. The PCIFI works in several steps. The first step is to construct a multiple-function protein network graph by labeling each vertex with one or more of the molecular functions it performs. The second step is to filter out protein interactions between protein pairs that are not functionally interdependent of each other in the statistical sense. The third step is to make use of an information-theoretic measure to determine the strength of the functional interdependence between all remaining interacting protein pairs. Finally, the last step is to try to form protein complexes based on the measure of the strength of functional interdependence and the connectivity between proteins. For performance evaluation, PCIFI was used to identify protein complexes in real PPI network data and the protein complexes it found were matched against those that were previously known in MIPS. The results show that PCIFI can be an effective technique for the identification of protein complexes. The protein complexes it found can match more known protein complexes with a smaller false-alarm rate and can provide useful insights into the understanding of the functional interdependence relationships between proteins in protein complexes. © 2006 IEEE.',\n",
              "   'title': '195\\nDiscovering functional interdependence relationship in PPI networks for protein complex identification Lam, W.W.M., Chan, K.C.C. 2012 IEEE Transactions on Biomedical Engineering\\n59(4), pp. 899-908\\n15'},\n",
              "  {'abstract': \"Protein interaction networks are useful resources for the functional annotation of proteins. Recently, we have generated a highly connected protein-protein interaction network for Huntington's disease (HD) by automated yeast two-hybrid (Y2H) screening (Goehler et al., 2004). The network included several novel direct interaction partners for the disease protein huntingtin (htt). Some of these interactions, however, have not been validated by independent methods. Here we describe the verification of the interaction between htt and GASP2 (G protein-coupled receptor associated sorting protein 2), a protein involved in membrane receptor degradation. Using membrane-based and classical coimmunoprecipitation assays we demonstrate that htt and GASP2 form a complex in cotransfected mammalian cells. Moreover, we show that the two proteins colocalize in SH-SY5Y cells, raising the possibility that htt and GASP2 interact in neurons. As the GASP protein family plays a role in G protein-coupled receptor sorting, our data suggest that htt might influence receptor trafficking via the interaction with GASP2. © Springer-Verlag 2006.\",\n",
              "   'title': '196\\nHuntingtin interacts with the receptor sorting family protein GASP2 Horn, S.C., Lalowski, M., Goehler, H., (...), Wanker, E.E., Stelzl, U. 2006 Journal of Neural Transmission\\n113(8), pp. 1081-1090\\n17'},\n",
              "  {'abstract': 'Clathrin-mediated endocytosis is a major cellular pathway for internalization of proteins and lipids and for recycling of synaptic vesicles. The GTPase dynamin plays a key role in this process, and the proline-rich domain of dynamin participates in various protein-protein interactions to ensure a proper coordination of endocytic processes. Although dynamin is not directly associated with actin, several dynamin-binding proteins can interact with actin or with proteins that regulate actin assembly, thereby coordinately regulating actin assembly and trafficking events. This article summarizes dynamin interactions with various Src homology 3-containing proteins, many of which are actin-binding proteins. It also discusses the recently identified two new dynamin binding proteins, SH3 protein interacting with Nck, 90 kDa/Wiskott-Aldrich syndrome protein interacting with SH3 protein (SPIN90/WISH) and sorting nexin 9, and outlines their potential role as a link between endocytosis and actin dynamics. Copyright © 2006 Humana Press Inc. All rights of any nature whatsoever reserved.',\n",
              "   'title': '197\\nEver-expanding network of dynamin-interacting proteins Kim, Y., Chang, S. 2006 Molecular Neurobiology\\n34(2), pp. 129-135\\n21'},\n",
              "  {'abstract': 'Membrane-embedded β-barrel proteins span the membrane via multiple amphipathic β-strands arranged in a cylindrical shape. These proteins are found in the outer membranes of Gram-negative bacteria, mitochondria and chloroplasts. This situation is thought to reflect the evolutionary origin of mitochondria and chloroplasts from Gram-negative bacterial endosymbionts. β-barrel proteins fulfil a variety of functions; among them are pore-forming proteins that allow the flux of metabolites across the membrane by passive diffusion, active transporters of siderophores, enzymes, structural proteins, and proteins that mediate protein translocation across or insertion into membranes. The biogenesis process of these proteins combines evolutionary conservation of the central elements with some noticeable differences in signals and machineries. This review summarizes our current knowledge of the functions and biogenesis of this special family of proteins. © 2009 The Author(s).',\n",
              "   'title': '198\\nBiogenesis of β-barrel membrane proteins in bacteria and eukaryotes: Evolutionary conservation and divergence\\nOpen Access\\nWalther, D.M., Rapaport, D., Tommassen, J. 2009 Cellular and Molecular Life Sciences\\n66(17), pp. 2789-2804\\n116'},\n",
              "  {'abstract': 'Complex protein-protein interaction networks govern biological processes in cells. Protein interfaces are the sites where proteins physically interact. Identification and characterization of protein interfaces will lead to understanding how proteins interact with each other and how they are involved in protein-protein interaction networks. What makes a given interface bind to different proteins; how similar/different the interactions in proteins are some key questions to be answered. Enormous amount of protein structures and experimental protein-protein interactions data necessitate advanced computational methods for analyzing and inferring new knowledge. Interface prediction methods use a wide range of sequence, structural and physico-chemical characteristics that distinguish interface residues from non-interface surface residues. Here, we present a review focusing on the characteristics of interfaces and the current status of interface prediction methods. © 2008 Bentham Science Publishers Ltd.',\n",
              "   'title': '199\\nCharacterization and prediction of protein interfaces to infer protein-protein interaction networks Keskin, O., Tuncbag, N., Gursoy, A. 2008 Current Pharmaceutical Biotechnology\\n9(2), pp. 67-76\\n24'},\n",
              "  {'abstract': '© 2018 The Authors. Biotechnology Journal Published by Wiley-VCHVerlag GmbH & Co. KGaA, Weinheim. Protein conformational disorders are characterized by disruption of protein folding and toxic accumulation of protein aggregates. Here we describe a sensitive and simple method to follow and monitor general protein aggregation in human cells. Heat shock protein 27 (HSP27) is an oligomeric small heat shock protein that binds and keeps unfolded proteins in a folding competent state. This high specificity of HSP27 for aggregated proteins can be explored to monitor aggregation in living cells by fusing it to a fluorescent protein as Green Fluorescent Protein (GFP). We have constructed a HeLa stable cell line expressing a HSP27:GFP chimeric reporter protein and after validation, this stable cell line is exposed to different agents that interfere with proteostasis, namely Arsenite, MG132, and Aβ-peptide. Exposure to proteome destabilizers lead to re-localization of HSP27:GFP fluorescence to foci, confirming that our reporter system is functional and can be used to detect and follow protein aggregation in living cells. This reporter is a valuable tool to setup wide-genetic screens to identify genes and pathways involved in protein misfolding and aggregation.',\n",
              "   'title': '200\\nA Fluorescence-Based Sensor Assay that Monitors General Protein Aggregation in Human Cells\\nOpen Access\\nPereira, M., Tomé, D., Domingues, A.S., (...), Santos, M.A.S., Soares, A.R. 2018 Biotechnology Journal\\n13(4),1700676\\n4'}],\n",
              " 'show_all_abstracts': True,\n",
              " 'value': 'search.page',\n",
              " 'wd': <selenium.webdriver.chrome.webdriver.WebDriver (session=\"50eaead24443124c0da5e767add03d1f\")>}"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "uEX5eLDEAs0p",
        "outputId": "2d31b3c3-27ab-465d-ab9b-e0039fe3f606"
      },
      "source": [
        "import re\n",
        "\n",
        "state[\"search_results\"]\n",
        "\n",
        "\"\"\"\n",
        "The first character is a numeral in the range 1-9, while the last three characters can be either numerals (in the range 0-9) or letters (in the range A-Z in the Latin alphabet). \n",
        "Plans for an expanded identification code system that handle more entries have been announced. \n",
        "\"\"\"\n",
        "for i, s in enumerate(state[\"search_results\"]):\n",
        "  print(i)\n",
        "  # x = re.search(\"[0-9][0-9A-Za-z]{3}\", s['abstract']) \n",
        "  # x = re.search(\"protein[A-Za-z]\", s['abstract']) \n",
        "  matches = re.findall('[()\\[\\]{};,. ][0-9][0-9A-Za-z]{3}[()\\[\\]{};,. ]', s['abstract'], re.DOTALL)\n",
        "  print(matches)\n",
        "  matches_1 = re.findall('[0-9][0-9A-Za-z]{3}', s['abstract'], re.DOTALL)\n",
        "  print(matches_1)\n"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "0\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "1\n",
            "[' 2011 ', ' 2011 ']\n",
            "['2011', '2011']\n",
            "2\n",
            "[' 2006,']\n",
            "['2006']\n",
            "3\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "4\n",
            "[]\n",
            "[]\n",
            "5\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "6\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "7\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "8\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "9\n",
            "[]\n",
            "[]\n",
            "10\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "11\n",
            "[]\n",
            "[]\n",
            "12\n",
            "[]\n",
            "[]\n",
            "13\n",
            "[' 2017 ', ' 2017;', ' 2016 ']\n",
            "['2017', '2017', '2016']\n",
            "14\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "15\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "16\n",
            "[]\n",
            "['4833', '418L', '03Ri', '9D61', '1689', '8WUF', '1046']\n",
            "17\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "18\n",
            "[]\n",
            "[]\n",
            "19\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "20\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "21\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "22\n",
            "[' 2018 ', ' 1012 ']\n",
            "['2018', '1012']\n",
            "23\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "24\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "25\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "26\n",
            "[' 2002 ']\n",
            "['2002']\n",
            "27\n",
            "[]\n",
            "[]\n",
            "28\n",
            "[' 2006 ']\n",
            "['100A', '2006']\n",
            "29\n",
            "[' 2020 ']\n",
            "['2020']\n",
            "30\n",
            "[' 2002 ']\n",
            "['2002']\n",
            "31\n",
            "[' 2013 ']\n",
            "['1750', '1752', '1750', '1752', '2013']\n",
            "32\n",
            "[' 2004 ']\n",
            "['2004']\n",
            "33\n",
            "[' 2018,']\n",
            "['2018']\n",
            "34\n",
            "[' 2012 ']\n",
            "['3K9M', '3K4M', '2012']\n",
            "35\n",
            "[' 2009 ']\n",
            "['2009']\n",
            "36\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "37\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "38\n",
            "[' 2010 ', ' 2010 ']\n",
            "['2010', '2010']\n",
            "39\n",
            "[' 1500 ', ' 2010 ']\n",
            "['227C', '147C', '1500', '2010']\n",
            "40\n",
            "[]\n",
            "[]\n",
            "41\n",
            "[' 2017 ']\n",
            "['2017']\n",
            "42\n",
            "[]\n",
            "[]\n",
            "43\n",
            "[' 2006.']\n",
            "['2006']\n",
            "44\n",
            "[' 2015,']\n",
            "['2015']\n",
            "45\n",
            "[' 2008 ']\n",
            "['1106', '2008']\n",
            "46\n",
            "[' 2009 ']\n",
            "['2009']\n",
            "47\n",
            "[]\n",
            "[]\n",
            "48\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "49\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "50\n",
            "[]\n",
            "[]\n",
            "51\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "52\n",
            "[' 2017 ']\n",
            "['2017']\n",
            "53\n",
            "[' 2017.']\n",
            "['2017']\n",
            "54\n",
            "[' 2018 ']\n",
            "['2018']\n",
            "55\n",
            "[]\n",
            "[]\n",
            "56\n",
            "[]\n",
            "[]\n",
            "57\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "58\n",
            "[' 2018.']\n",
            "['2018']\n",
            "59\n",
            "[' 2018 ']\n",
            "['2018']\n",
            "60\n",
            "[' 2017 ']\n",
            "['2017']\n",
            "61\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "62\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "63\n",
            "[]\n",
            "[]\n",
            "64\n",
            "[' 2015,']\n",
            "['2015']\n",
            "65\n",
            "[' 2019,']\n",
            "['2019']\n",
            "66\n",
            "[' 2012 ']\n",
            "['2012']\n",
            "67\n",
            "[' 2020 ', ' 1027 ']\n",
            "['2020', '1027']\n",
            "68\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "69\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "70\n",
            "['(2003)', ' 2005 ']\n",
            "['2003', '5353', '5360', '2005']\n",
            "71\n",
            "[' 2019 ']\n",
            "['2019']\n",
            "72\n",
            "[]\n",
            "[]\n",
            "73\n",
            "[]\n",
            "['2006']\n",
            "74\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "75\n",
            "[]\n",
            "[]\n",
            "76\n",
            "[]\n",
            "[]\n",
            "77\n",
            "[' 2010.']\n",
            "['2010']\n",
            "78\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "79\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "80\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "81\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "82\n",
            "[' 2005 ']\n",
            "['2005']\n",
            "83\n",
            "[' 2012 ']\n",
            "['2012']\n",
            "84\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "85\n",
            "[]\n",
            "[]\n",
            "86\n",
            "[' 2020 ']\n",
            "['2020']\n",
            "87\n",
            "[' 2018 ', ' 5115 ', ' 2291 ']\n",
            "['2018', '5115', '2291']\n",
            "88\n",
            "[' 1995,']\n",
            "['1995']\n",
            "89\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "90\n",
            "[]\n",
            "[]\n",
            "91\n",
            "[]\n",
            "[]\n",
            "92\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "93\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "94\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "95\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "96\n",
            "[' 2018 ']\n",
            "['2018', '16L1']\n",
            "97\n",
            "[' 2019 ']\n",
            "['2019']\n",
            "98\n",
            "[' 2005 ']\n",
            "['2005']\n",
            "99\n",
            "[' 2005 ']\n",
            "['2005']\n",
            "100\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "101\n",
            "[' 70p5,', ' 70p8,', ' 2005 ']\n",
            "['70p5', '70p8', '2005']\n",
            "102\n",
            "[]\n",
            "[]\n",
            "103\n",
            "[' 2009,']\n",
            "['2009']\n",
            "104\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "105\n",
            "[' 2009)', ' 2009 ']\n",
            "['2009', '2009']\n",
            "106\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "107\n",
            "[' 2004 ']\n",
            "['2004']\n",
            "108\n",
            "[]\n",
            "[]\n",
            "109\n",
            "[' 2004 ']\n",
            "['2004']\n",
            "110\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "111\n",
            "[' 2008.']\n",
            "['2008']\n",
            "112\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "113\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "114\n",
            "[]\n",
            "[]\n",
            "115\n",
            "[' 2000 ']\n",
            "['2000']\n",
            "116\n",
            "[' 1989,']\n",
            "['1989']\n",
            "117\n",
            "[]\n",
            "[]\n",
            "118\n",
            "[' 1732 ', ' 1269 ', ' 2010 ']\n",
            "['1732', '1269', '2010']\n",
            "119\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "120\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "121\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "122\n",
            "[' 2015 ', '.0001,']\n",
            "['2015', '0001']\n",
            "123\n",
            "[' 2018 ']\n",
            "['2018']\n",
            "124\n",
            "[' 2009 ']\n",
            "['2009']\n",
            "125\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "126\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "127\n",
            "[]\n",
            "[]\n",
            "128\n",
            "[' 2018,']\n",
            "['2018']\n",
            "129\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "130\n",
            "[' 2012 ']\n",
            "['2012']\n",
            "131\n",
            "[' 2009 ']\n",
            "['2009']\n",
            "132\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "133\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "134\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "135\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "136\n",
            "[' 2014 ']\n",
            "['2014']\n",
            "137\n",
            "[]\n",
            "[]\n",
            "138\n",
            "[' 2013 ']\n",
            "['2013']\n",
            "139\n",
            "[]\n",
            "[]\n",
            "140\n",
            "[]\n",
            "[]\n",
            "141\n",
            "[' 2012 ']\n",
            "['2012']\n",
            "142\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "143\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "144\n",
            "[]\n",
            "[]\n",
            "145\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "146\n",
            "[' 2009,']\n",
            "['1b55', '2009']\n",
            "147\n",
            "[]\n",
            "[]\n",
            "148\n",
            "[' 2018,']\n",
            "['2018']\n",
            "149\n",
            "[' 2016 ', ' 2016;', ' 2016 ']\n",
            "['2016', '2016', '1009', '1016', '2016']\n",
            "150\n",
            "[' 2010.']\n",
            "['2010']\n",
            "151\n",
            "[' 2017 ']\n",
            "['2017']\n",
            "152\n",
            "[' 2019 ']\n",
            "['2019']\n",
            "153\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "154\n",
            "[' 2004 ']\n",
            "['2004']\n",
            "155\n",
            "[]\n",
            "[]\n",
            "156\n",
            "[' 2017 ']\n",
            "['2017', '25Cp']\n",
            "157\n",
            "[' 2011 ']\n",
            "['2011']\n",
            "158\n",
            "[' 2015,']\n",
            "['2015', '100A']\n",
            "159\n",
            "[' 2020,']\n",
            "['2020']\n",
            "160\n",
            "[]\n",
            "[]\n",
            "161\n",
            "[' 2018 ']\n",
            "['2018']\n",
            "162\n",
            "[' 2014.']\n",
            "['2014']\n",
            "163\n",
            "[' 2012 ', ' 2012.']\n",
            "['2012', '2012']\n",
            "164\n",
            "[]\n",
            "[]\n",
            "165\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "166\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "167\n",
            "[' 2012 ']\n",
            "['2012']\n",
            "168\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "169\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "170\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "171\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "172\n",
            "[' 2017 ']\n",
            "['2017']\n",
            "173\n",
            "[]\n",
            "[]\n",
            "174\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "175\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "176\n",
            "[' 2016 ']\n",
            "['2016']\n",
            "177\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "178\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "179\n",
            "[' 2004 ']\n",
            "['2004']\n",
            "180\n",
            "[' 2013 ', ' 2013.', ' 2013 ']\n",
            "['2013', '2013', '2013']\n",
            "181\n",
            "[' 2005 ']\n",
            "['2005']\n",
            "182\n",
            "[]\n",
            "[]\n",
            "183\n",
            "[' 2015.']\n",
            "['2015']\n",
            "184\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "185\n",
            "[]\n",
            "[]\n",
            "186\n",
            "[' 2015 ']\n",
            "['2015']\n",
            "187\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "188\n",
            "[' 2010 ']\n",
            "['2010']\n",
            "189\n",
            "[' 2012 ']\n",
            "['2012']\n",
            "190\n",
            "[' 2011 ']\n",
            "['2R1A', '2011']\n",
            "191\n",
            "[]\n",
            "[]\n",
            "192\n",
            "[' 2007 ']\n",
            "['2007']\n",
            "193\n",
            "[' 2017 ']\n",
            "['2017']\n",
            "194\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "195\n",
            "[' 2004)', ' 2006.']\n",
            "['2004', '2006']\n",
            "196\n",
            "[' 2006 ']\n",
            "['2006']\n",
            "197\n",
            "[' 2009 ']\n",
            "['2009']\n",
            "198\n",
            "[' 2008 ']\n",
            "['2008']\n",
            "199\n",
            "[' 2018 ']\n",
            "['2018']\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "K4n9sS9Pe2EY"
      },
      "source": [
        "state[\"current_query\"] = \"\"\n",
        "state[\"wd\"] = wd_return_to_search(state[\"wd\"])\n",
        "state[\"value\"] = \"search.page\"\n",
        "display(state)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "EgkvV_LK0-IV"
      },
      "source": [
        "# state[\"current_query\"] = \"functional properties\"\n",
        "# state[\"wd\"] = wd_search_query(state[\"current_query\"], state[\"wd\"])\n",
        "# state[\"value\"] = \"search.results.page\"\n",
        "# state[\"search_results\"] = wd_extract_raw_results(state[\"wd\"])\n",
        "# display(state)\n",
        "\n",
        "# state[\"current_query\"] = \"\"\n",
        "# state[\"wd\"] = wd_return_to_search(state[\"wd\"])\n",
        "# state[\"value\"] = \"search.page\"\n",
        "# display(state)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0rjn0SHL7LwK"
      },
      "source": [
        "# def wd_return_to_search(wd):\n",
        "#   \"\"\"\n",
        "#   transition: search.results.page -> search.page\n",
        "#   \"\"\"\n",
        "#   search_tab = wd.find_element(By.ID ,\"gh-Search\")\n",
        "#   search_tab.click()\n",
        "\n",
        "#   # search_input = wd.find_element(By.ID ,\"searchterm1\")\n",
        "  \n",
        "#   # # erase current input\n",
        "#   # search_input.send_keys(Keys.CONTROL, 'a')\n",
        "#   # search_input.send_keys(Keys.BACKSPACE)\n",
        "#   return wd\n",
        "\n",
        "# def wd_search_query(query, wd):\n",
        "#   \"\"\"\n",
        "#   transition: search.page -> search.results.page\n",
        "#   \"\"\"\n",
        "#   search_input = wd.find_element(By.ID ,\"searchterm1\")\n",
        "  \n",
        "#   # erase current input\n",
        "#   search_input.send_keys(Keys.CONTROL, 'a')\n",
        "#   search_input.send_keys(Keys.BACKSPACE)\n",
        "\n",
        "#   search_input.send_keys(query)\n",
        "\n",
        "#   search_button_row = wd.find_element(By.ID ,\"searchBtnRow\")\n",
        "#   search_button = search_button_row.find_element(By.XPATH, '//button[@title=\"Search\"]')\n",
        "#   search_button.click()\n",
        "\n",
        "#   # print_body(wd)\n",
        "#   return wd\n",
        "\n",
        "# def wd_extract_results_count(wd):\n",
        "#   results_count = (int)(wd.find_element(By.CLASS_NAME ,'resultsCount').text.replace(',', ''))\n",
        "#   return results_count\n",
        "\n",
        "# def wd_extract_raw_results(results_count, current_display, wd):\n",
        "#   \"\"\"\n",
        "#   transition: search.results.page -> search.results.page\n",
        "#   \"\"\"\n",
        "\n",
        "#   # this should be a separated event\n",
        "#   show_abstracts_link = wd.find_element(By.ID ,\"previewAllAbstractsLinkText\")\n",
        "#   show_abstracts_link.click()\n",
        "#   # print(\"show_abstracts_link\", show_abstracts_link.text)\n",
        "#   # print(\"show_abstracts_link\", show_abstracts_link.tag_name)\n",
        "  \n",
        "#   # show_abstracts_button = wd.find_element(By.ID ,\"previewAllAbstractsLinkText\").find_element(By.XPATH, '//a[@class=\"secondaryLink\"]')\n",
        "#   # print(\"show_abstracts_button\", show_abstracts_button.text)\n",
        "#   # print(\"show_abstracts_button\", show_abstracts_button.tag_name)\n",
        "#   # show_abstracts_button.click()\n",
        "\n",
        "#   WebDriverWait(wd, 10).until(EC.presence_of_element_located((By.ID, \"previewAbstract1\")))\n",
        "\n",
        "#   #this does not have support for pages yet\n",
        "#   results = [] \n",
        "#   for i in range(0, min(results_count, current_display)):\n",
        "#     row_title = wd.find_element(By.ID , f'resultDataRow{i}').text\n",
        "#     # print(f'previewAbstract{i + 1}')\n",
        "#     row_abstract = wd.find_element(By.ID , f'previewAbstract{i + 1}').text\n",
        "#     results.append({\n",
        "#         \"title\": row_title,\n",
        "#         \"abstract\": row_abstract\n",
        "#     })\n",
        "\n",
        "#   # results = wd.find_element(By.ID ,\"srchResultsList\")\n",
        "#   # results_list = results.find_elements(By.TAG_NAME ,\"tr\")\n",
        "#   # for r in results_list:\n",
        "#   #   pp.pprint(r)\n",
        "#   #   pp.pprint(r.text)\n",
        "\n",
        "#   return results"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "K4cyfBkahvOR"
      },
      "source": [
        "Failed experiment with post request to get abstracts"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "oHJ_Ps0y-YCK"
      },
      "source": [
        "  # def search(self, query: str, max: int) -> List[SearchResult]:\n",
        "  #   if not self.value == \"search.page\":\n",
        "  #     print(\"Search failed, WebDriver should be on the \\\"search.page\\\" state\")\n",
        "  #     return []\n",
        "\n",
        "  #   self.current_query = self._search_query(query)\n",
        "  #   self.total_results = (int)(wd.find_element(By.CLASS_NAME ,'resultsCount').text.replace(',', ''))\n",
        "    \n",
        "  #   # build abstract request\n",
        "  #   # pp.pprint(self.wd.get_cookies())\n",
        "  #   cookies_string = \"; \".join([f\"{c['name']}={c['value']}\" for c in self.wd.get_cookies()])\n",
        "  #   print(cookies_string)\n",
        "    \n",
        "  #   headers={\n",
        "  #       \"Cookie\": cookies_string,\n",
        "  #       \"Referer\": wd.current_url\n",
        "  #   }\n",
        "    \n",
        "  #   # print(\"https://www.scopus.com/results/showallabstractspreview.uri\")\n",
        "\n",
        "  #   data = {\n",
        "  #     \"count\": \"200\",\n",
        "  #     \"offset\": \"0\",\n",
        "  #     \"searchString\": \"TITLE-ABS-KEY(proteins)\",\n",
        "  #     \"sortType\": \"r-f\",\n",
        "  #     \"origin\": \"resultslist\",\n",
        "  #     \"originationType\": \"b\",\n",
        "  #     \"clusterRowDisplayCount\": \"10\",\n",
        "  #     \"resultsPerPage\": \"200\",\n",
        "  #     \"resultsCount\": \"6,193,247\",\n",
        "  #     \"eid\": \"null\",\n",
        "  #     \"listId\": \"\",\n",
        "  #     \"cite\": \"\",\n",
        "  #     \"citedAuthorId\": \"\",\n",
        "  #     \"citingId\": \"\",\n",
        "  #     \"refeid\": \"\",\n",
        "  #     \"ref\": \"\",\n",
        "  #     \"recent\": \"\",\n",
        "  #     \"recordIndex\": \"null\"\n",
        "  #   }\n",
        "\n",
        "  #   r = requests.post(\"https://www.scopus.com/results/showallabstractspreview.uri\", headers=headers, json=data)\n",
        "  #   print(r.headers)\n",
        "  #   print(r.text)\n",
        "  #   return []"
      ],
      "execution_count": null,
      "outputs": []
    }
  ]
}